# abcrf

<details>

* Version: 1.8.1
* GitHub: NA
* Source code: https://github.com/cran/abcrf
* Date/Publication: 2019-11-05 14:40:02 UTC
* Number of recursive dependencies: 41

Run `revdep_details(, "abcrf")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.1Mb
      sub-directories of 1Mb or more:
        data   2.6Mb
        libs   4.4Mb
    ```

# airpart

<details>

* Version: 1.2.0
* GitHub: NA
* Source code: https://github.com/cran/airpart
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 153

Run `revdep_details(, "airpart")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘airpart-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: makeForest
    > ### Title: Plot allelic ratio result as forest
    > ### Aliases: makeForest
    > 
    > ### ** Examples
    > 
    > sce <- makeSimulatedData()
    ...
    Warning in dt(beta[-p$no.shrink]/p$prior.scale, df = p$prior.df, ncp = p$prior.mean,  :
      full precision may not have been achieved in 'pnt{final}'
    Warning in dt(beta[-p$no.shrink]/p$prior.scale, df = p$prior.df, ncp = p$prior.mean,  :
      full precision may not have been achieved in 'pnt{final}'
    Warning in dt(beta[-p$no.shrink]/p$prior.scale, df = p$prior.df, ncp = p$prior.mean,  :
      full precision may not have been achieved in 'pnt{final}'
    > makeForest(sce_sub, showtext = TRUE)
    svalue shown in columns per cell type
    Error: The package tidyr is needed for this function to work. Please install it.
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
      ...
    --- re-building ‘airpart.Rmd’ using rmarkdown
    Quitting from lines 337-339 (airpart.Rmd) 
    Error: processing vignette 'airpart.Rmd' failed with diagnostics:
    The package tidyr is needed for this function to work. Please install it.
    --- failed re-building ‘airpart.Rmd’
    
    SUMMARY: processing the following file failed:
      ‘airpart.Rmd’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

# AlpsNMR

<details>

* Version: 3.4.0
* GitHub: NA
* Source code: https://github.com/cran/AlpsNMR
<<<<<<< HEAD
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 167
||||||| cc87037
* Date/Publication: 2021-08-29
* Number of recursive dependencies: 163
=======
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 169
>>>>>>> develop

Run `revdep_details(, "AlpsNMR")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘zip’
      All declared Imports should be used.
    ```

# AMARETTO

<details>

* Version: 1.10.0
* GitHub: NA
* Source code: https://github.com/cran/AMARETTO
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 156

Run `revdep_details(, "AMARETTO")` for more info

</details>

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
      ...
    --- re-building ‘amaretto.Rmd’ using rmarkdown
    Quitting from lines 2-16 (amaretto.Rmd) 
    Error: processing vignette 'amaretto.Rmd' failed with diagnostics:
    there is no package called 'BiocStyle'
    --- failed re-building ‘amaretto.Rmd’
    
    SUMMARY: processing the following file failed:
      ‘amaretto.Rmd’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.1Mb
      sub-directories of 1Mb or more:
        data   1.5Mb
        doc    2.3Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    License components with restrictions not permitted:
      Apache License (== 2.0) + file LICENSE
    Package listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘knitr’
    A package should be listed in only one of these fields.
    'LinkingTo' field is unused: package has no 'src' directory
    ```

*   checking R code for possible problems ... NOTE
    ```
    AMARETTO_HTMLreport: no visible binding for global variable ‘ModuleNr’
    AMARETTO_HTMLreport: no visible binding for global variable ‘Weights’
    AMARETTO_HTMLreport: no visible binding for global variable
      ‘RegulatorIDs’
    AMARETTO_HTMLreport: no visible binding for global variable ‘TargetIDs’
    AMARETTO_HTMLreport: no visible binding for global variable ‘moduleNr’
    AMARETTO_HTMLreport: no visible binding for global variable ‘Testset’
    AMARETTO_HTMLreport: no visible binding for global variable ‘padj’
    AMARETTO_HTMLreport: no visible binding for global variable
      ‘n_Overlapping’
    ...
    GmtFromModules: no visible binding for global variable ‘GeneNames’
    HyperGTestGeneEnrichment: no visible binding for global variable ‘i’
    HyperGTestGeneEnrichment: no visible binding for global variable ‘j’
    read_gct: no visible binding for global variable ‘Description’
    Undefined global functions or variables:
      Color Description GeneNames Genes Geneset Geneset_length ModuleNr
      Modules MsigdbMapping NumberGenes Overlapping_genes RegulatorIDs
      TargetIDs Testset Type Weights dt_gensesetsall i j moduleNr
      n_Overlapping overlap_perc p.value p_value padj q.value value
      variable
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 152 marked UTF-8 strings
    ```

# amplican

<details>

* Version: 1.16.0
* GitHub: https://github.com/valenlab/amplican
* Source code: https://github.com/cran/amplican
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 109

Run `revdep_details(, "amplican")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 16.1Mb
      sub-directories of 1Mb or more:
        doc   13.9Mb
        libs   1.3Mb
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘CrispRVariants’
    ```

# aroma.core

<details>

* Version: 3.2.2
* GitHub: https://github.com/HenrikBengtsson/aroma.core
* Source code: https://github.com/cran/aroma.core
* Date/Publication: 2021-01-05 05:10:12 UTC
* Number of recursive dependencies: 48

Run `revdep_details(, "aroma.core")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      'sfit', 'expectile', 'HaarSeg', 'mpcbs'
    ```

# aroma.light

<details>

* Version: 3.24.0
* GitHub: https://github.com/HenrikBengtsson/aroma.light
* Source code: https://github.com/cran/aroma.light
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 6

Run `revdep_details(, "aroma.light")` for more info

</details>

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      inst/rsp/.rspPlugins
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

# autonomics

<details>

* Version: 1.2.0
* GitHub: https://github.com/bhagwataditya/autonomics
* Source code: https://github.com/cran/autonomics
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 206

Run `revdep_details(, "autonomics")` for more info

</details>

## In both

*   checking Rd \usage sections ... WARNING
    ```
    Undocumented arguments in documentation object 'normimpute'
      ‘ref’ ‘pos’
    
    Functions with \usage entries need to have the appropriate \alias
    entries, and all their arguments documented.
    The \usage entries must correspond to syntactically valid R code.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
      ...
    --- re-building ‘using_autonomics.Rmd’ using rmarkdown
    Error: processing vignette 'using_autonomics.Rmd' failed with diagnostics:
    there is no package called ‘BiocStyle’
    --- failed re-building ‘using_autonomics.Rmd’
    
    SUMMARY: processing the following file failed:
      ‘using_autonomics.Rmd’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .BBSoptions
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

# bahc

<details>

* Version: 0.3.0
* GitHub: NA
* Source code: https://github.com/cran/bahc
* Date/Publication: 2020-09-21 16:40:02 UTC
* Number of recursive dependencies: 2

Run `revdep_details(, "bahc")` for more info

</details>

## In both

*   checking LazyData ... NOTE
    ```
      'LazyData' is specified without a 'data' directory
    ```

# BASiCS

<details>

* Version: 2.6.0
* GitHub: https://github.com/catavallejos/BASiCS
* Source code: https://github.com/cran/BASiCS
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 136

Run `revdep_details(, "BASiCS")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 12.1Mb
      sub-directories of 1Mb or more:
        data   2.0Mb
        doc    1.4Mb
        libs   7.8Mb
    ```

# BatchQC

<details>

* Version: 1.22.0
* GitHub: https://github.com/mani2012/BatchQC
* Source code: https://github.com/cran/BatchQC
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 158

Run `revdep_details(, "BatchQC")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    gene_plot: no visible global function definition for ‘boxplot’
    gene_plot: no visible binding for global variable ‘batch’
    gls.series.C: no visible global function definition for ‘lm.fit’
    sample_plot: no visible global function definition for ‘boxplot’
    sample_plot: no visible binding for global variable ‘batch’
    Undefined global functions or variables:
      batch boxplot lm.fit
    Consider adding
      importFrom("graphics", "boxplot")
      importFrom("stats", "lm.fit")
    to your NAMESPACE file.
    ```

# baystability

<details>

* Version: 0.1.0
* GitHub: NA
* Source code: https://github.com/cran/baystability
* Date/Publication: 2018-03-13 15:55:34 UTC
* Number of recursive dependencies: 120

Run `revdep_details(, "baystability")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘ggfortify’ ‘ggplot2’ ‘matrixStats’ ‘reshape2’ ‘scales’
      All declared Imports should be used.
    ```

# bigPint

<details>

* Version: 1.10.0
* GitHub: https://github.com/lindsayrutter/bigPint
* Source code: https://github.com/cran/bigPint
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 167

Run `revdep_details(, "bigPint")` for more info

</details>

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
      ...
    --- re-building ‘bioconductor.Rmd’ using rmarkdown
    Error: processing vignette 'bioconductor.Rmd' failed with diagnostics:
    there is no package called ‘BiocStyle’
    --- failed re-building ‘bioconductor.Rmd’
    
    --- re-building ‘manuscripts.Rmd’ using rmarkdown
    Error: processing vignette 'manuscripts.Rmd' failed with diagnostics:
    there is no package called ‘BiocStyle’
    ...
    --- re-building ‘summarizedExperiment.Rmd’ using rmarkdown
    Error: processing vignette 'summarizedExperiment.Rmd' failed with diagnostics:
    there is no package called ‘BiocStyle’
    --- failed re-building ‘summarizedExperiment.Rmd’
    
    SUMMARY: processing the following files failed:
      ‘bioconductor.Rmd’ ‘manuscripts.Rmd’ ‘summarizedExperiment.Rmd’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.6Mb
      sub-directories of 1Mb or more:
        data             2.0Mb
        doc              2.3Mb
        shiny-examples   3.0Mb
    ```

# bingat

<details>

* Version: 1.3
* GitHub: NA
* Source code: https://github.com/cran/bingat
* Date/Publication: 2017-07-05 18:30:37 UTC
* Number of recursive dependencies: 34

Run `revdep_details(, "bingat")` for more info

</details>

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘genalg’
    ```

# BioNERO

<details>

* Version: 1.2.0
* GitHub: https://github.com/almeidasilvaf/BioNERO
* Source code: https://github.com/cran/BioNERO
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 200

Run `revdep_details(, "BioNERO")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.6Mb
      sub-directories of 1Mb or more:
        data   2.2Mb
        doc    4.0Mb
    ```

# biscuiteer

<details>

* Version: 1.8.0
* GitHub: https://github.com/trichelab/biscuiteer
* Source code: https://github.com/cran/biscuiteer
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 217

Run `revdep_details(, "biscuiteer")` for more info

</details>

## In both

*   checking whether package ‘biscuiteer’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘BiocParallel::bpstart’ by ‘QDNAseq::bpstart’ when loading ‘biscuiteer’
    See ‘/c4/home/henrik/repositories/matrixStats/revdep/checks/biscuiteer/new/biscuiteer.Rcheck/00install.out’ for details.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
      ...
    --- re-building ‘biscuiteer.Rmd’ using rmarkdown
    Error: processing vignette 'biscuiteer.Rmd' failed with diagnostics:
    there is no package called ‘BiocStyle’
    --- failed re-building ‘biscuiteer.Rmd’
    
    SUMMARY: processing the following file failed:
      ‘biscuiteer.Rmd’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.8Mb
      sub-directories of 1Mb or more:
        data      1.1Mb
        extdata   3.4Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported object imported by a ':::' call: ‘QDNAseq:::expectedVariance’
      See the note in ?`:::` about the use of this operator.
    ```

# BloodGen3Module

<details>

* Version: 1.2.0
* GitHub: NA
* Source code: https://github.com/cran/BloodGen3Module
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 163

Run `revdep_details(, "BloodGen3Module")` for more info

</details>

## In both

*   checking whether package ‘BloodGen3Module’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘SummarizedExperiment::rowRanges’ by ‘matrixStats::rowRanges’ when loading ‘BloodGen3Module’
      Warning: replacing previous import ‘SummarizedExperiment::start’ by ‘stats::start’ when loading ‘BloodGen3Module’
      Warning: replacing previous import ‘SummarizedExperiment::end’ by ‘stats::end’ when loading ‘BloodGen3Module’
      Warning: replacing previous import ‘matrixStats::rowRanges’ by ‘SummarizedExperiment::rowRanges’ when loading ‘BloodGen3Module’
    See ‘/c4/home/henrik/repositories/matrixStats/revdep/checks/BloodGen3Module/new/BloodGen3Module.Rcheck/00install.out’ for details.
    ```

*   checking top-level files ... NOTE
    ```
    File
      LICENSE
    is not mentioned in the DESCRIPTION file.
    ```

# bnbc

<details>

* Version: 1.16.0
* GitHub: https://github.com/hansenlab/bnbc
* Source code: https://github.com/cran/bnbc
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 96

Run `revdep_details(, "bnbc")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.2Mb
      sub-directories of 1Mb or more:
        data   1.7Mb
        libs   1.4Mb
    ```

# bnclassify

<details>

* Version: 0.4.6
* GitHub: https://github.com/bmihaljevic/bnclassify
* Source code: https://github.com/cran/bnclassify
<<<<<<< HEAD
* Date/Publication: 2021-10-29 23:10:05 UTC
* Number of recursive dependencies: 97
||||||| cc87037
* Date/Publication: 2020-03-12 17:40:02 UTC
* Number of recursive dependencies: 93
=======
* Date/Publication: 2021-10-29 23:10:05 UTC
* Number of recursive dependencies: 98
>>>>>>> develop

Run `revdep_details(, "bnclassify")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  8.9Mb
      sub-directories of 1Mb or more:
        libs   8.2Mb
    ```

# bnem

<details>

* Version: 1.2.0
* GitHub: https://github.com/MartinFXP/bnem
* Source code: https://github.com/cran/bnem
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 161

Run `revdep_details(, "bnem")` for more info

</details>

## In both

*   checking tests ...
    ```
      Running ‘runTests.R’
     ERROR
    Running the tests in ‘tests/runTests.R’ failed.
    Complete output:
      > BiocGenerics:::testPackage("bnem")
      Error in library("RUnit", quietly = TRUE) : 
        there is no package called 'RUnit'
      Calls: <Anonymous> -> library
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
      ...
    --- re-building ‘bnem.rmd’ using rmarkdown
    Error: processing vignette 'bnem.rmd' failed with diagnostics:
    there is no package called ‘BiocStyle’
    --- failed re-building ‘bnem.rmd’
    
    SUMMARY: processing the following file failed:
      ‘bnem.rmd’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘rmarkdown’
      All declared Imports should be used.
    ```

# brms

<details>

<<<<<<< HEAD
* Version: 2.16.3
||||||| cc87037
* Version: 2.16.1
=======
* Version: 2.17.0
>>>>>>> develop
* GitHub: https://github.com/paul-buerkner/brms
* Source code: https://github.com/cran/brms
<<<<<<< HEAD
* Date/Publication: 2021-11-22 19:50:02 UTC
* Number of recursive dependencies: 214
||||||| cc87037
* Date/Publication: 2021-08-23 14:00:05 UTC
* Number of recursive dependencies: 229
=======
* Date/Publication: 2022-04-13 14:22:29 UTC
* Number of recursive dependencies: 178
>>>>>>> develop

Run `revdep_details(, "brms")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘cmdstanr’
    ```

*   checking installed package size ... NOTE
    ```
<<<<<<< HEAD
      installed size is  7.5Mb
||||||| cc87037
      installed size is  8.3Mb
=======
      installed size is  8.7Mb
>>>>>>> develop
      sub-directories of 1Mb or more:
<<<<<<< HEAD
        R     4.1Mb
        doc   2.4Mb
||||||| cc87037
        R      4.4Mb
        doc    2.4Mb
        help   1.0Mb
=======
        R     4.2Mb
        doc   3.6Mb
>>>>>>> develop
    ```

# BSgenome

<details>

* Version: 1.62.0
* GitHub: https://github.com/Bioconductor/BSgenome
* Source code: https://github.com/cran/BSgenome
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 101

Run `revdep_details(, "BSgenome")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘BSgenome-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: SNPlocs-class
    > ### Title: SNPlocs objects
    > ### Aliases: class:SNPlocs SNPlocs-class SNPlocs class:ODLT_SNPlocs
    > ###   ODLT_SNPlocs-class ODLT_SNPlocs class:OldFashionSNPlocs
    > ###   OldFashionSNPlocs-class OldFashionSNPlocs provider,SNPlocs-method
    > ###   providerVersion,SNPlocs-method releaseDate,SNPlocs-method
    > ###   releaseName,SNPlocs-method referenceGenome
    ...
    > seqlevels(my_cds, pruning.mode="coarse") <- c("chr22", "chrM")
    > seqlevelsStyle(my_cds)  # UCSC
    [1] "UCSC"
    > seqlevelsStyle(snps)    # NCBI
    [1] "NCBI"
    > seqlevelsStyle(my_cds) <- seqlevelsStyle(snps)
    Error in .form_assembly_report_url(assembly_accession) : 
      don't know where to find assembly report for GCF_000001405.39
    Calls: seqlevelsStyle<- ... .get_NCBI_chrom_info_from_accession -> fetch_assembly_report -> .form_assembly_report_url
    Execution halted
    ```

*   checking Rd cross-references ... WARNING
    ```
    Missing link or links in documentation object 'export-methods.Rd':
      ‘[rtracklayer]{export}’
    
    See section 'Cross-references' in the 'Writing R Extensions' manual.
    ```

*   checking for missing documentation entries ... WARNING
    ```
    Undocumented code objects:
      ‘MaskedBSgenome’ ‘OnDiskLongTable’ ‘OnDiskLongTable_old’
      ‘as.data.frame.BSgenomeViews’ ‘batchsizes’ ‘blocksizes’ ‘breakpoints’
      ‘forgeMaskedBSgenomeDataPkg’
      ‘getBatchesByOverlapsFromOnDiskLongTable’
      ‘getBatchesBySeqnameFromOnDiskLongTable’
      ‘getBatchesFromOnDiskLongTable’ ‘getBatchesFromOnDiskLongTable_old’
      ‘getRowsByIdFromOnDiskLongTable’ ‘getRowsByIdFromOnDiskLongTable_old’
      ‘getRowsByIndexFromOnDiskLongTable_old’ ‘getRowsFromOnDiskLongTable’
      ‘rowids’ ‘saveAsOnDiskLongTable_old’
    ...
      generic 'seqinfo' and siglist 'TwobitNamedSequences'
      generic 'seqnames' and siglist 'OnDiskNamedSequences'
      generic 'show' and siglist 'OnDiskLongTable'
      generic 'show' and siglist 'OnDiskLongTable_old'
      generic 'show' and siglist 'OnDiskNamedSequences'
      generic 'spatialIndex' and siglist 'OnDiskLongTable'
    All user-level objects in a package (including S4 classes and methods)
    should have documentation entries.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      'BiocGenerics', 'S4Vectors', 'IRanges', 'GenomeInfoDb',
      'GenomicRanges', 'Biostrings', 'rtracklayer'
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking for portable file names ... NOTE
    ```
    Found the following non-portable file path:
      BSgenome/inst/extdata/GentlemanLab/BSgenome.Gmellonella.NCBI.ASM364042v2-tools/fasta_to_sorted_2bit.R
    
    Tarballs are only required to store paths of up to 100 bytes and cannot
    store those of more than 256 bytes, with restrictions including to 100
    bytes for the final component.
    See section ‘Package structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.8Mb
      sub-directories of 1Mb or more:
        R         1.7Mb
        extdata   5.5Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Packages listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘methods’ ‘BiocGenerics’ ‘S4Vectors’ ‘IRanges’ ‘GenomeInfoDb’ ‘GenomicRanges’ ‘Biostrings’ ‘rtracklayer’
    A package should be listed in only one of these fields.
    ```

*   checking package subdirectories ... NOTE
    ```
    Found the following CITATION file in a non-standard place:
      inst/extdata/GentlemanLab/BSgenome.Creinhardtii.JGI.v5.6-tools/CITATION
    Most likely ‘inst/CITATION’ should be used instead.
    ```

*   checking dependencies in R code ... NOTE
    ```
    ':::' calls which should be '::':
      ‘S4Vectors:::makeClassinfoRowForCompactPrinting’
      ‘S4Vectors:::makePrettyMatrixForCompactPrinting’
      See the note in ?`:::` about the use of this operator.
    Unexported objects imported by ':::' calls:
      ‘GenomeInfoDb:::compactPrintNamedAtomicVector’
      ‘GenomicRanges:::get_out_of_bound_index’ ‘IRanges:::.normargSEW’
      ‘IRanges:::unlist_as_integer’ ‘S4Vectors:::anyMissing’
      ‘S4Vectors:::anyMissingOrOutside’ ‘S4Vectors:::decodeRle’
      ‘S4Vectors:::diffWithInitialZero’
      ‘S4Vectors:::make_zero_col_DataFrame’ ‘S4Vectors:::quick_unlist’
      ‘S4Vectors:::quick_unsplit’ ‘S4Vectors:::recycleVector’
      ‘rtracklayer:::.DNAString_to_twoBit’ ‘rtracklayer:::.TwoBits_export’
      ‘rtracklayer:::checkArgFormat’ ‘rtracklayer:::twoBitPath’
      See the note in ?`:::` about the use of this operator.
    ```

# BSW

<details>

* Version: 0.1.1
* GitHub: https://github.com/adam-bec/BSW
* Source code: https://github.com/cran/BSW
* Date/Publication: 2021-03-22 16:20:09 UTC
* Number of recursive dependencies: 51

Run `revdep_details(, "BSW")` for more info

</details>

## In both

*   checking LazyData ... NOTE
    ```
      'LazyData' is specified without a 'data' directory
    ```

<<<<<<< HEAD
# bulkAnalyseR

<details>

* Version: 0.1.1
* GitHub: https://github.com/Core-Bioinformatics/bulkAnalyseR
* Source code: https://github.com/cran/bulkAnalyseR
* Date/Publication: 2022-01-15 09:02:49 UTC
* Number of recursive dependencies: 166

Run `revdep_details(, "bulkAnalyseR")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘shinythemes’
      All declared Imports should be used.
    ```

||||||| cc87037
=======
# bulkAnalyseR

<details>

* Version: 1.0.0
* GitHub: https://github.com/Core-Bioinformatics/bulkAnalyseR
* Source code: https://github.com/cran/bulkAnalyseR
* Date/Publication: 2022-04-07 10:12:29 UTC
* Number of recursive dependencies: 187

Run `revdep_details(, "bulkAnalyseR")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.0Mb
      sub-directories of 1Mb or more:
        doc       2.3Mb
        extdata   3.4Mb
    ```

>>>>>>> develop
# bumphunter

<details>

* Version: 1.36.0
* GitHub: https://github.com/rafalab/bumphunter
* Source code: https://github.com/cran/bumphunter
* Date/Publication: 2021-10-27
* Number of recursive dependencies: 112

Run `revdep_details(, "bumphunter")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      'S4Vectors', 'IRanges', 'GenomeInfoDb', 'GenomicRanges', 'foreach',
      'iterators', 'parallel', 'locfit'
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported object imported by a ':::' call: ‘doParallel:::.options’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    bumphunterEngine: no visible binding for global variable ‘bootstraps’
    Undefined global functions or variables:
      bootstraps
    ```

# Canek

<details>

* Version: 0.2.0
* GitHub: NA
* Source code: https://github.com/cran/Canek
* Date/Publication: 2022-03-23 12:10:02 UTC
* Number of recursive dependencies: 187

Run `revdep_details(, "Canek")` for more info

</details>

## In both

*   checking tests ...
    ```
      Running ‘testthat.R’
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Complete output:
      > library(testthat)
      > library(Canek)
      > 
      > test_check("Canek")
      [ FAIL 2 | WARN 0 | SKIP 0 | PASS 88 ]
      
    ...
      as.integer(table(dataLouvain$cluster)) not equal to c(178, 146, 162, 145).
      Lengths differ: 5 is not 4
      ── Failure (test-Clustering.R:20:3): Clustering with louvain works ─────────────
      dim(dataLouvain$centers) not equal to c(4, 10).
      1/2 mismatches
      [1] 5 - 4 == 1
      
      [ FAIL 2 | WARN 0 | SKIP 0 | PASS 88 ]
      Error: Test failures
      Execution halted
    ```

# CARBayesST

<details>

* Version: 3.2.2
* GitHub: https://github.com/duncanplee/CARBayesST
* Source code: https://github.com/cran/CARBayesST
* Date/Publication: 2022-03-16 16:00:05 UTC
* Number of recursive dependencies: 107

Run `revdep_details(, "CARBayesST")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.5Mb
      sub-directories of 1Mb or more:
        libs   6.1Mb
    ```

# CATALYST

<details>

* Version: 1.18.1
* GitHub: https://github.com/HelenaLC/CATALYST
* Source code: https://github.com/cran/CATALYST
* Date/Publication: 2022-01-16
* Number of recursive dependencies: 246

Run `revdep_details(, "CATALYST")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 14.3Mb
      sub-directories of 1Mb or more:
        data   3.1Mb
        doc    9.9Mb
    ```

# causalOT

<details>

* Version: 0.1
* GitHub: NA
* Source code: https://github.com/cran/causalOT
* Date/Publication: 2022-03-14 08:50:08 UTC
* Number of recursive dependencies: 102

Run `revdep_details(, "causalOT")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 31.6Mb
      sub-directories of 1Mb or more:
        R      1.6Mb
        libs  29.9Mb
    ```

# ccdf

<details>

* Version: 1.1.4
* GitHub: https://github.com/mgauth/ccdf
* Source code: https://github.com/cran/ccdf
* Date/Publication: 2021-09-24 08:00:05 UTC
* Number of recursive dependencies: 51

Run `revdep_details(, "ccdf")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘statmod’
      All declared Imports should be used.
    ```

# CDSeq

<details>

* Version: 1.0.8
* GitHub: https://github.com/kkang7/CDSeq_R_Package
* Source code: https://github.com/cran/CDSeq
* Date/Publication: 2021-02-10 16:10:02 UTC
* Number of recursive dependencies: 187

Run `revdep_details(, "CDSeq")` for more info

</details>

## In both

*   checking whether package ‘CDSeq’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘Rcpp::LdFlags’ by ‘RcppThread::LdFlags’ when loading ‘CDSeq’
      Warning: replacing previous import ‘RcppThread::detectCores’ by ‘parallel::detectCores’ when loading ‘CDSeq’
    See ‘/c4/home/henrik/repositories/matrixStats/revdep/checks/CDSeq/new/CDSeq.Rcheck/00install.out’ for details.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.8Mb
      sub-directories of 1Mb or more:
        doc    2.4Mb
        libs   2.2Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘ggpubr’ ‘matrixStats’ ‘qlcMatrix’
      All declared Imports should be used.
    ```

# celda

<details>

* Version: 1.10.0
* GitHub: https://github.com/campbio/celda
* Source code: https://github.com/cran/celda
<<<<<<< HEAD
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 364
||||||| cc87037
* Date/Publication: 2021-05-30
* Number of recursive dependencies: 357
=======
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 365
>>>>>>> develop

Run `revdep_details(, "celda")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
<<<<<<< HEAD
      installed size is 10.3Mb
||||||| cc87037
      installed size is  8.6Mb
=======
      installed size is 10.4Mb
>>>>>>> develop
      sub-directories of 1Mb or more:
<<<<<<< HEAD
        doc    3.0Mb
        libs   5.9Mb
    ```

||||||| cc87037
        libs   5.9Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    featureModuleTable : <anonymous>: no visible global function definition
      for ‘rowData’
    Undefined global functions or variables:
      rowData
    ```

=======
        doc    3.0Mb
        libs   6.0Mb
    ```

>>>>>>> develop
# CelliD

<details>

* Version: 1.2.1
* GitHub: NA
* Source code: https://github.com/cran/CelliD
* Date/Publication: 2022-01-09
* Number of recursive dependencies: 266

Run `revdep_details(, "CelliD")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.1Mb
      sub-directories of 1Mb or more:
        doc    3.2Mb
        libs   3.0Mb
    ```

# cellWise

<details>

* Version: 2.2.6
* GitHub: NA
* Source code: https://github.com/cran/cellWise
* Date/Publication: 2022-03-16 14:30:02 UTC
* Number of recursive dependencies: 72

Run `revdep_details(, "cellWise")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘GSE’
    ```

*   checking installed package size ... NOTE
    ```
<<<<<<< HEAD
      installed size is 14.7Mb
||||||| cc87037
      installed size is 14.9Mb
=======
      installed size is 14.8Mb
>>>>>>> develop
      sub-directories of 1Mb or more:
        data   3.1Mb
        doc    2.0Mb
        libs   9.3Mb
    ```

# CEMiTool

<details>

* Version: 1.18.1
* GitHub: NA
* Source code: https://github.com/cran/CEMiTool
* Date/Publication: 2021-10-28
* Number of recursive dependencies: 189

Run `revdep_details(, "CEMiTool")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  8.8Mb
      sub-directories of 1Mb or more:
        data      3.1Mb
        doc       2.3Mb
        extdata   2.8Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    flip_vector: no visible global function definition for ‘setNames’
    select_genes: no visible global function definition for ‘var’
    get_hubs,CEMiTool : <anonymous>: no visible global function definition
      for ‘head’
    get_merged_mods,CEMiTool: no visible global function definition for
      ‘as.dist’
    get_mods,CEMiTool: no visible global function definition for ‘as.dist’
    get_phi,CEMiTool: no visible global function definition for ‘tail’
    get_phi,CEMiTool: no visible global function definition for ‘head’
    mod_gene_num,CEMiTool: no visible binding for global variable ‘modules’
    ...
    save_plots,CEMiTool: no visible global function definition for
      ‘dev.off’
    Undefined global functions or variables:
      ..eq.label.. ..rr.label.. := Mean Variance as.dist data dev.off dist
      head modules num_genes setNames tail var
    Consider adding
      importFrom("grDevices", "dev.off")
      importFrom("stats", "as.dist", "dist", "setNames", "var")
      importFrom("utils", "data", "head", "tail")
    to your NAMESPACE file.
    ```

# ChAMP

<details>

* Version: 2.24.0
* GitHub: NA
* Source code: https://github.com/cran/ChAMP
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 242

Run `revdep_details(, "ChAMP")` for more info

</details>

## In both

*   checking whether package ‘ChAMP’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Note: possible error in 'dmrcate(myannotation, ': unused argument (mc.cores = cores) 
      Warning: replacing previous import 'plyr::summarise' by 'plotly::summarise' when loading 'ChAMP'
      Warning: replacing previous import 'plyr::rename' by 'plotly::rename' when loading 'ChAMP'
      Warning: replacing previous import 'plyr::arrange' by 'plotly::arrange' when loading 'ChAMP'
      Warning: replacing previous import 'plyr::mutate' by 'plotly::mutate' when loading 'ChAMP'
      Warning: replacing previous import 'plyr::is.discrete' by 'Hmisc::is.discrete' when loading 'ChAMP'
      Warning: replacing previous import 'plyr::summarize' by 'Hmisc::summarize' when loading 'ChAMP'
      Warning: replacing previous import 'plotly::subplot' by 'Hmisc::subplot' when loading 'ChAMP'
      Warning: replacing previous import 'GenomicRanges::sort' by 'globaltest::sort' when loading 'ChAMP'
      Warning: replacing previous import 'plotly::last_plot' by 'ggplot2::last_plot' when loading 'ChAMP'
      Warning: replacing previous import 'globaltest::model.matrix' by 'stats::model.matrix' when loading 'ChAMP'
      Warning: replacing previous import 'globaltest::p.adjust' by 'stats::p.adjust' when loading 'ChAMP'
    See ‘/c4/home/henrik/repositories/matrixStats/revdep/checks/ChAMP/new/ChAMP.Rcheck/00install.out’ for details.
    Information on the location(s) of code generating the ‘Note’s can be
    obtained by re-running with environment variable R_KEEP_PKG_SOURCE set
    to ‘yes’.
    ```

*   checking Rd \usage sections ... WARNING
    ```
    Duplicated \argument entries in documentation object 'champ.SVD':
      ‘Rplot’
    
    Functions with \usage entries need to have the appropriate \alias
    entries, and all their arguments documented.
    The \usage entries must correspond to syntactically valid R code.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      'minfi', 'ChAMPdata', 'DMRcate', 'Illumina450ProbeVariants.db',
      'IlluminaHumanMethylationEPICmanifest', 'DT', 'RPMM'
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.9Mb
      sub-directories of 1Mb or more:
        doc       3.6Mb
        extdata   1.9Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Package listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘rmarkdown’
    A package should be listed in only one of these fields.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: 'prettydoc'
      All declared Imports should be used.
    Package in Depends field not imported from: 'DT'
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    champ.DMR: possible error in dmrcate(myannotation, min.cpgs =
      minProbes, lambda = lambda, C = C, mc.cores = cores): unused argument
      (mc.cores = cores)
    champ.ebGSEA : gseaWTfn: no visible global function definition for
      'wilcox.test'
    champ.ebGSEA: no visible global function definition for 'stopCluster'
    champ.norm: no visible global function definition for 'stopCluster'
    champ.process: no visible global function definition for 'champ.EpiMod'
    Undefined global functions or variables:
      champ.EpiMod stopCluster wilcox.test
    Consider adding
      importFrom("stats", "wilcox.test")
    to your NAMESPACE file.
    ```

<<<<<<< HEAD
# ChIPpeakAnno

<details>

* Version: 3.28.0
* GitHub: NA
* Source code: https://github.com/cran/ChIPpeakAnno
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 207

Run `revdep_details(, "ChIPpeakAnno")` for more info

</details>

## In both

*   checking examples ... ERROR
||||||| cc87037
# Chicago

<details>

* Version: 1.20.0
* GitHub: NA
* Source code: https://github.com/cran/Chicago
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 171

Run `revdep_details(, "Chicago")` for more info

</details>

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
      ...
    --- re-building ‘Chicago.Rmd’ using rmarkdown
    Loading required package: data.table
    
    Welcome to CHiCAGO - version 1.20.0
    If you are new to CHiCAGO, please consider reading the vignette through the command: vignette("Chicago").
    NOTE: Default values of tlb.minProxOEPerBin and tlb.minProxB2BPerBin changed as of Version 1.1.5. No action is required unless you specified non-default values, or wish to re-run the pipeline on old chicagoData objects. See news(package="Chicago")
    Quitting from lines 44-46 (Chicago.Rmd) 
    Error: processing vignette 'Chicago.Rmd' failed with diagnostics:
    there is no package called 'PCHiCdata'
    --- failed re-building ‘Chicago.Rmd’
    
    SUMMARY: processing the following file failed:
      ‘Chicago.Rmd’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘PCHiCdata’
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘PCHiCdata’
    ```

# ChIPpeakAnno

<details>

* Version: 3.26.4
* GitHub: NA
* Source code: https://github.com/cran/ChIPpeakAnno
* Date/Publication: 2021-09-12
* Number of recursive dependencies: 206

Run `revdep_details(, "ChIPpeakAnno")` for more info

</details>

## In both

*   checking examples ... ERROR
=======
# ChIPpeakAnno

<details>

* Version: 3.28.1
* GitHub: NA
* Source code: https://github.com/cran/ChIPpeakAnno
* Date/Publication: 2022-02-06
* Number of recursive dependencies: 207

Run `revdep_details(, "ChIPpeakAnno")` for more info

</details>

## In both

*   checking examples ... ERROR
>>>>>>> develop
    ```
    Running examples in ‘ChIPpeakAnno-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: findEnhancers
    > ### Title: Find possible enhancers depend on DNA interaction data
    > ### Aliases: findEnhancers
    > ### Keywords: misc
    > 
    > ### ** Examples
    > 
    ...
    
     Try to keep the seqname style consistent.
    
     Try to keep the seqname style consistent.
    >   data("myPeakList")
    >   findEnhancers(myPeakList[500:1000], annoData, DNAinteractiveData)
    Error in findEnhancers(myPeakList[500:1000], annoData, DNAinteractiveData) : 
      length(intersect(seqlevelsStyle(peaks), seqlevelsStyle(annoData))) >  .... is not TRUE
    Calls: findEnhancers -> stopifnot
    Execution halted
    ```

*   checking tests ...
    ```
      Running ‘runTests.R’
     ERROR
    Running the tests in ‘tests/runTests.R’ failed.
    Last 50 lines of output:
          filter
      
      [1] TRUE
      > require("TxDb.Hsapiens.UCSC.hg38.knownGene") || stop("unable to load TxDb.Hsapiens.UCSC.hg38.knownGene")
      Loading required package: TxDb.Hsapiens.UCSC.hg38.knownGene
      [1] TRUE
    ...
        6.     └─BSgenome .local(x, ...)
        7.       └─BSgenome:::.extractFromBSgenomeMultipleSequences(...)
        8.         ├─BiocGenerics::lapply(...)
        9.         └─base::lapply(...)
       10.           └─BSgenome FUN(X[[i]], ...)
       11.             └─BSgenome:::.getOneSeqFromBSgenomeMultipleSequences(...)
      
      [ FAIL 1 | WARN 1 | SKIP 6 | PASS 309 ]
      Error: Test failures
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
    --- re-building ‘ChIPpeakAnno.Rmd’ using rmarkdown
    duplicated or NA names found. Rename all the names by numbers.
    If you are importing files downloaded from ensembl, 
              it will be better to import the files into a TxDb object,
              and then convert to GRanges by toGRanges. Here is the sample code:
              library(GenomicFeatures)
              txdb <- makeTxDbFromGFF('/c4/home/henrik/repositories/matrixStats/revdep/checks/ChIPpeakAnno/new/ChIPpeakAnno.Rcheck/ChIPpeakAnno/extdata/GFF_peaks.gff')
              anno <- toGRanges(txdb, format='gene')
    duplicated or NA names found. 
    ...
    Quitting from lines 65-72 (quickStart.Rmd) 
    Error: processing vignette 'quickStart.Rmd' failed with diagnostics:
    'length.out' must be a non-negative number
    --- failed re-building ‘quickStart.Rmd’
    
    SUMMARY: processing the following files failed:
      ‘pipeline.Rmd’ ‘quickStart.Rmd’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 25.0Mb
      sub-directories of 1Mb or more:
        data     12.6Mb
        doc       6.0Mb
        extdata   5.2Mb
    ```

# cifti

<details>

* Version: 0.4.5
* GitHub: NA
* Source code: https://github.com/cran/cifti
* Date/Publication: 2018-02-01 23:25:24 UTC
* Number of recursive dependencies: 47

Run `revdep_details(, "cifti")` for more info

</details>

## In both

*   checking LazyData ... NOTE
    ```
      'LazyData' is specified without a 'data' directory
    ```

# cliqueMS

<details>

* Version: 1.8.0
* GitHub: https://github.com/osenan/cliqueMS
* Source code: https://github.com/cran/cliqueMS
<<<<<<< HEAD
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 141
||||||| cc87037
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 136
=======
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 141
>>>>>>> develop

Run `revdep_details(, "cliqueMS")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘cliqueMS-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: computeCliques
    > ### Title: Computes clique groups from a similarity network
    > ### Aliases: computeCliques
    > 
    > ### ** Examples
    > 
    > mzfile <- system.file("standards.mzXML", package = "cliqueMS")
    ...
    > ex.anClique <- createanClique(msSet)
    > show(ex.anClique)
    anClique S4 object with 126 features 
    No computed clique groups
    No isotope annotation
    No adduct annotation
    > netlist <- createNetwork(msSet, peaks(msSet), filter = TRUE)
    Error in peaks(msSet) : could not find function "peaks"
    Calls: createNetwork ... createNetwork -> <Anonymous> -> <Anonymous> -> nrow
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.4Mb
      sub-directories of 1Mb or more:
        libs   5.8Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘MSnbase’
      All declared Imports should be used.
    ```

# clusterExperiment

<details>

* Version: 2.14.0
* GitHub: https://github.com/epurdom/clusterExperiment
* Source code: https://github.com/cran/clusterExperiment
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 185

Run `revdep_details(, "clusterExperiment")` for more info

</details>

## In both

*   checking contents of ‘data’ directory ... WARNING
    ```
    Output for data("rsecFluidigm", package = "clusterExperiment"):
      Search path was changed
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 17.7Mb
      sub-directories of 1Mb or more:
        R      1.4Mb
        data   3.6Mb
        doc   10.4Mb
        libs   1.9Mb
    ```

*   checking foreign function calls ... NOTE
    ```
    Foreign function calls to a different package:
      .C(ape::node_depth, ...)
      .C(ape::node_depth_edgelength, ...)
      .C(ape::node_height, ...)
      .C(ape::node_height_clado, ...)
    See chapter ‘System and foreign language interfaces’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘scRNAseq’, ‘ConsensusClusterPlus’
    ```

# clustifyr

<details>

* Version: 1.6.0
* GitHub: https://github.com/rnabioco/clustifyr
* Source code: https://github.com/cran/clustifyr
* Date/Publication: 2021-11-07
* Number of recursive dependencies: 148

Run `revdep_details(, "clustifyr")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.4Mb
      sub-directories of 1Mb or more:
        data   2.2Mb
        doc    1.5Mb
        help   1.1Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    assess_rank_bias : <anonymous>: no visible binding for '<<-' assignment
      to ‘res3’
    assess_rank_bias: no visible binding for global variable ‘res3’
    average_clusters : <anonymous>: no visible binding for global variable
      ‘pbmc_matrix_small’
    Undefined global functions or variables:
      pbmc_matrix_small res3
    ```

# cmapR

<details>

* Version: 1.6.0
* GitHub: https://github.com/cmap/cmapR
* Source code: https://github.com/cran/cmapR
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 78

Run `revdep_details(, "cmapR")` for more info

</details>

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
    --- re-building ‘tutorial.Rmd’ using rmarkdown
    trying URL 'https://bioconductor.org/packages/3.14/bioc/src/contrib/BiocVersion_3.14.0.tar.gz'
    Content type 'application/x-gzip' length 969 bytes
    ==================================================
    downloaded 969 bytes
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .dockerignore
      docs/build/html/.buildinfo
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.8Mb
      sub-directories of 1Mb or more:
        data      2.3Mb
        extdata   2.1Mb
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    '::' or ':::' import not declared from: ‘BiocManager’
    'library' or 'require' calls not declared from:
      ‘BiocManager’ ‘ggplot2’
    ```

# cna

<details>

* Version: 3.3.0
* GitHub: NA
* Source code: https://github.com/cran/cna
* Date/Publication: 2021-12-17 14:30:02 UTC
* Number of recursive dependencies: 79

Run `revdep_details(, "cna")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.0Mb
      sub-directories of 1Mb or more:
        libs   5.8Mb
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error(s) in re-building vignettes:
    --- re-building ‘cna.Rnw’ using Sweave
    Registered S3 method overwritten by 'cna':
      method          from
      some.data.frame car 
    Error: processing vignette 'cna.Rnw' failed with diagnostics:
    Running 'texi2dvi' on 'cna.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `nicefrac.sty' not found.
    
    ...
    l.12 \usepackage
                    {float}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    --- failed re-building 'cna.Rnw'
    
    SUMMARY: processing the following file failed:
      'cna.Rnw'
    
    Error: Vignette re-building failed.
    Execution halted
    ```

# CNVScope

<details>

* Version: 3.7.2
* GitHub: https://github.com/jamesdalg/CNVScope
* Source code: https://github.com/cran/CNVScope
* Date/Publication: 2022-03-30 23:40:08 UTC
* Number of recursive dependencies: 197

Run `revdep_details(, "CNVScope")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.1Mb
      sub-directories of 1Mb or more:
        doc       3.2Mb
        extdata   1.7Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: 'Hmisc'
      All declared Imports should be used.
    ```

# coin

<details>

* Version: 1.4-2
* GitHub: NA
* Source code: https://github.com/cran/coin
* Date/Publication: 2021-10-08 08:40:15 UTC
* Number of recursive dependencies: 20

Run `revdep_details(, "coin")` for more info

</details>

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    Error(s) in re-building vignettes:
    --- re-building ‘Implementation.Rnw’ using Sweave
    Loading required package: survival
    Loading required package: grid
    --- finished re-building ‘Implementation.Rnw’
    
    --- re-building ‘LegoCondInf.Rnw’ using Sweave
    Loading required package: coin
    Loading required package: survival
    Error: processing vignette 'LegoCondInf.Rnw' failed with diagnostics:
    ...
    Warning in .local(object, ...) :
      p-values may be incorrect due to violation
      of the subset pivotality condition
    --- finished re-building ‘coin.Rnw’
    
    SUMMARY: processing the following file failed:
      ‘LegoCondInf.Rnw’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

# cola

<details>

* Version: 2.0.0
* GitHub: https://github.com/jokergoo/cola
* Source code: https://github.com/cran/cola
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 238

Run `revdep_details(, "cola")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.7Mb
      sub-directories of 1Mb or more:
        data      3.5Mb
        extdata   1.0Mb
        libs      1.9Mb
    ```

# ComplexHeatmap

<details>

* Version: 2.10.0
* GitHub: https://github.com/jokergoo/ComplexHeatmap
* Source code: https://github.com/cran/ComplexHeatmap
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 109

Run `revdep_details(, "ComplexHeatmap")` for more info

</details>

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘magick’
    ```

# conclus

<details>

<<<<<<< HEAD
* Version: 1.2.0
||||||| cc87037
* Version: 1.0.0
=======
* Version: 1.2.4
>>>>>>> develop
* GitHub: NA
* Source code: https://github.com/cran/conclus
<<<<<<< HEAD
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 250
||||||| cc87037
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 253
=======
* Date/Publication: 2022-04-10
* Number of recursive dependencies: 250
>>>>>>> develop

Run `revdep_details(, "conclus")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘conclus-Ex.R’ failed
    The error most likely occurred in:
    
<<<<<<< HEAD
    > ### Name: normaliseCountMatrix,scRNAseq-method
    > ### Title: normaliseCountMatrix
    > ### Aliases: normaliseCountMatrix,scRNAseq-method normaliseCountMatrix
||||||| cc87037
    > ### Name: retrieveFromGEO
    > ### Title: retrieveFromGEO
    > ### Aliases: retrieveFromGEO
=======
    > ### Name: addClustering,scRNAseq-method
    > ### Title: addClustering
    > ### Aliases: addClustering,scRNAseq-method addClustering
>>>>>>> develop
    > 
    > ### ** Examples
    > 
<<<<<<< HEAD
    > ## Load the count matrix
||||||| cc87037
    > outputDirectory <- "./YourOutputDirectory"
=======
    > ## Object scr containing the results of previous steps
>>>>>>> develop
    ...
<<<<<<< HEAD
    Peer's Certificate issuer is not recognized.
    # Attempt 5/5 # Connection to Ensembl ... 
    Ensembl site unresponsive, trying uswest mirror
    Error in curl::curl_fetch_memory(url, handle = handle) : 
      Peer's Certificate issuer is not recognized.
    Peer's Certificate issuer is not recognized.
    Error in .tryUseMart(biomart = "ensembl", dataset) : 
      Peer's Certificate issuer is not recognized.
    Calls: normaliseCountMatrix ... normaliseCountMatrix -> .annotateGenes -> .defineMartVar -> .tryUseMart
||||||| cc87037
    GSE96982-GPL19057_series_matrix.txt.gz
    trying URL 'https://ftp.ncbi.nlm.nih.gov/geo/series/GSE96nnn/GSE96982/matrix/GSE96982-GPL19057_series_matrix.txt.gz'
    Content type 'application/x-gzip' length 27102 bytes (26 KB)
    ==================================================
    downloaded 26 KB
    
    Error: The size of the connection buffer (131072) was not large enough
    to fit a complete line:
      * Increase it by setting `Sys.setenv("VROOM_CONNECTION_SIZE")`
=======
    # Attempt 5/5 # Retrieving information about genes from biomaRt ...
    Error in .readFromCache(bfc, hash) : Multiple cache results found.
    Please clear your cache by running biomartCacheClear()
    Multiple cache results found.
    Please clear your cache by running biomartCacheClear()
    Error in .tryGetBM(attributes = attributes, ensembl = ensembl, values = values,  : 
      Multiple cache results found.
    Please clear your cache by running biomartCacheClear()
    Calls: addClustering ... .defineDatabase -> .queryBiomart -> merge -> .returnDB1 -> .tryGetBM
>>>>>>> develop
    Execution halted
    ```

<<<<<<< HEAD
*   checking tests ...
    ```
      Running ‘test-all.R’
     ERROR
    Running the tests in ‘tests/test-all.R’ failed.
    Complete output:
      > library("testthat")
      > library("conclus")
      Setting options('download.file.method.GEOquery'='auto')
      Setting options('GEOquery.inmemory.gpl'=FALSE)
      
      > test_check("conclus")
    ...
          ▆
       1. ├─conclus::normaliseCountMatrix(...) at test_scRNAseq-methods.R:55:0
       2. └─conclus::normaliseCountMatrix(...)
       3.   └─conclus:::.annotateGenes(...)
       4.     └─conclus:::.defineMartVar(species)
       5.       └─conclus:::.tryUseMart(biomart = "ensembl", dataset)
      
      [ FAIL 1 | WARN 0 | SKIP 0 | PASS 81 ]
      Error: Test failures
      Execution halted
    ```

||||||| cc87037
=======
*   checking tests ...
    ```
      Running ‘test-all.R’
     ERROR
    Running the tests in ‘tests/test-all.R’ failed.
    Complete output:
      > library("testthat")
      > library("conclus")
      
      
      Setting options('download.file.method.GEOquery'='auto')
      Setting options('GEOquery.inmemory.gpl'=FALSE)
    ...
       2. └─conclus::retrieveGenesInfo(scrFinal, cores = 2)
       3.   └─conclus:::.defineDatabase(species, genes, speDbID, speDBdescription)
       4.     └─conclus:::.queryBiomart(genes, ensembl)
       5.       ├─base::merge(...)
       6.       └─conclus:::.returnDB1(genes, ensembl)
       7.         └─conclus:::.tryGetBM(...)
      
      [ FAIL 1 | WARN 0 | SKIP 0 | PASS 81 ]
      Error: Test failures
      Execution halted
    ```

>>>>>>> develop
*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
      ...
    --- re-building ‘conclus_vignette.Rmd’ using rmarkdown
    Quitting from lines 177-214 (conclus_vignette.Rmd) 
    Error: processing vignette 'conclus_vignette.Rmd' failed with diagnostics:
    not all 'rnames' found or unique.
    --- failed re-building ‘conclus_vignette.Rmd’
    
    SUMMARY: processing the following file failed:
      ‘conclus_vignette.Rmd’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

*   checking R code for possible problems ... NOTE
    ```
    .retrieveClustersNumberK: no visible global function definition for
      ‘median’
    Undefined global functions or variables:
      median
    Consider adding
      importFrom("stats", "median")
    to your NAMESPACE file.
    ```

# condiments

<details>

* Version: 1.2.0
* GitHub: https://github.com/HectorRDB/condiments
* Source code: https://github.com/cran/condiments
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 175

Run `revdep_details(, "condiments")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘condiments-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: differentiationTest
    > ### Title: Differential Differentiation Test
    > ### Aliases: differentiationTest differentiationTest,matrix-method
    > ###   differentiationTest,SlingshotDataSet-method
    > ###   differentiationTest,SingleCellExperiment-method
    > ###   differentiationTest,PseudotimeOrdering-method
    > 
    ...
    > data('slingshotExample', package = "slingshot")
    > rd <- slingshotExample$rd
    > cl <- slingshotExample$cl
    > condition <- factor(rep(c('A','B'), length.out = nrow(rd)))
    > condition[110:139] <- 'A'
    > sds <- slingshot::slingshot(rd, cl)
    Error in loadNamespace(x) : 
      there is no package called ‘DelayedMatrixStats’
    Calls: <Anonymous> ... loadNamespace -> withRestarts -> withOneRestart -> doWithOneRestart
    Execution halted
    ```

*   checking tests ...
    ```
      Running ‘testthat.R’
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 50 lines of output:
      ── Error (test_topology.R:12:1): (code run outside of `test_that()`) ───────────
      <packageNotFoundError/error/condition>
      Error in `loadNamespace(x)`: there is no package called 'DelayedMatrixStats'
      Backtrace:
           ▆
        1. ├─slingshot::slingshot(rd, cl) at test_topology.R:12:0
    ...
       14. │               └─TrajectoryUtils FUN(x, clusters)
       15. │                 └─TrajectoryUtils:::.rowstats_w(...)
       16. └─base::loadNamespace(x)
       17.   └─base::withRestarts(stop(cond), retry_loadNamespace = function() NULL)
       18.     └─base withOneRestart(expr, restarts[[1L]])
       19.       └─base doWithOneRestart(return(expr), restart)
      
      [ FAIL 7 | WARN 0 | SKIP 0 | PASS 10 ]
      Error: Test failures
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
    --- re-building ‘condiments.Rmd’ using rmarkdown
    Loading required package: princurve
    Loading required package: TrajectoryUtils
    Loading required package: SingleCellExperiment
    Loading required package: SummarizedExperiment
    Loading required package: MatrixGenerics
    Loading required package: matrixStats
    
    Attaching package: 'MatrixGenerics'
    ...
    --- failed re-building ‘controls.Rmd’
    
    --- re-building ‘examples.Rmd’ using rmarkdown
    --- finished re-building ‘examples.Rmd’
    
    SUMMARY: processing the following files failed:
      ‘condiments.Rmd’ ‘controls.Rmd’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

*   checking R code for possible problems ... NOTE
    ```
    .differentiationTest: no visible binding for global variable ‘pair’
    .differentiationTest: no visible binding for global variable
      ‘statistic’
    .differentiationTest: no visible binding for global variable ‘p.value’
    .progressionTest: no visible binding for global variable ‘lineage’
    .progressionTest: no visible binding for global variable ‘statistic’
    .progressionTest: no visible binding for global variable ‘p.value’
    Undefined global functions or variables:
      lineage p.value pair statistic
    ```

# conquer

<details>

* Version: 1.3.0
* GitHub: https://github.com/XiaoouPan/conquer
* Source code: https://github.com/cran/conquer
* Date/Publication: 2022-03-21 08:20:05 UTC
* Number of recursive dependencies: 5

Run `revdep_details(, "conquer")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
<<<<<<< HEAD
      installed size is 17.5Mb
||||||| cc87037
      installed size is  8.1Mb
=======
      installed size is 17.6Mb
>>>>>>> develop
      sub-directories of 1Mb or more:
<<<<<<< HEAD
        libs  17.3Mb
||||||| cc87037
        R   6.4Mb
=======
        libs  17.4Mb
>>>>>>> develop
    ```

# CopywriteR

<details>

* Version: 2.26.0
* GitHub: https://github.com/PeeperLab/CopywriteR
* Source code: https://github.com/cran/CopywriteR
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 67

Run `revdep_details(, "CopywriteR")` for more info

</details>

## In both

*   checking running R code from vignettes ...
    ```
      ‘CopywriteR.Rnw’... failed
     ERROR
    Errors in running code in vignettes:
    when running code in ‘CopywriteR.Rnw’
      ...
     --- function search by body ---
    Function .tng in namespace CopywriteR has this body.
     ----------- END OF FAILURE REPORT -------------- 
    
      When sourcing ‘CopywriteR.R’:
    Error: The GC-content and mappability normalization did not work due to a
      failure to calculate loesses. This can generally be solved by using
      larger bin sizes. Stopping execution of the remaining part of the
      script...
    Execution halted
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error(s) in re-building vignettes:
    --- re-building ‘CopywriteR.Rnw’ using Sweave
    Loading required package: BiocParallel
    Warning in simpleLoess(y, x, w, span, degree = degree, parametric = parametric,  :
      pseudoinverse used at 0.979
    Warning in simpleLoess(y, x, w, span, degree = degree, parametric = parametric,  :
      neighborhood radius 0.001
    Warning in simpleLoess(y, x, w, span, degree = degree, parametric = parametric,  :
      reciprocal condition number  0
    Warning in simpleLoess(y, x, w, span, degree = degree, parametric = parametric,  :
    ...
      larger bin sizes. Stopping execution of the remaining part of the
      script...
    
    --- failed re-building ‘CopywriteR.Rnw’
    
    SUMMARY: processing the following file failed:
      ‘CopywriteR.Rnw’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

# corrcoverage

<details>

* Version: 1.2.1
* GitHub: https://github.com/annahutch/corrcoverage
* Source code: https://github.com/cran/corrcoverage
* Date/Publication: 2019-12-06 23:20:12 UTC
* Number of recursive dependencies: 75

Run `revdep_details(, "corrcoverage")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.3Mb
      sub-directories of 1Mb or more:
        extdata   3.8Mb
        libs      1.2Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘data.table’
      All declared Imports should be used.
    ```

*   checking LazyData ... NOTE
    ```
      'LazyData' is specified without a 'data' directory
    ```

# crlmm

<details>

* Version: 1.52.0
* GitHub: NA
* Source code: https://github.com/cran/crlmm
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 58

Run `revdep_details(, "crlmm")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.7Mb
      sub-directories of 1Mb or more:
        R      1.2Mb
        data   2.7Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Versioned 'LinkingTo' value for ‘preprocessCore’ is only usable in R >= 3.0.2
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' call to ‘RUnit’ in package code.
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    Namespace in Imports field not imported from: ‘splines’
      All declared Imports should be used.
    Unexported object imported by a ':::' call: ‘Biobase:::assayDataEnvLock’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    .test: no visible global function definition for ‘defineTestSuite’
    .test: no visible global function definition for ‘runTestSuite’
    .test: no visible global function definition for ‘printTextProtocol’
    calculateRBafCNSet : processByChromosome: no visible global function
      definition for ‘position’
    genotypeInf: no visible binding for global variable ‘anno’
    krlmm: no visible binding for global variable ‘VGLMparameters’
    Undefined global functions or variables:
      VGLMparameters anno defineTestSuite position printTextProtocol
      runTestSuite
    ```

# crossmeta

<details>

* Version: 1.20.0
* GitHub: NA
* Source code: https://github.com/cran/crossmeta
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 174

Run `revdep_details(, "crossmeta")` for more info

</details>

## Newly fixed

*   checking examples ... ERROR
    ```
    Running examples in ‘crossmeta-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: es_meta
    > ### Title: Effect size combination meta analysis.
    > ### Aliases: es_meta
    > 
    > ### ** Examples
    > 
    > 
    ...
    > 
    > # add tissue sources to perform seperate meta-analyses for each source (optional)
    > # anals <- add_sources(anals, data_dir)
    > 
    > # perform meta-analysis
    > es <- es_meta(anals, by_source = TRUE)
    Error in `[.data.frame`(top, , c("SYMBOL", "dprime", "vardprime")) : 
      undefined columns selected
    Calls: es_meta ... FUN -> get_es -> lapply -> FUN -> [ -> [.data.frame
    Execution halted
    ```

## In both

*   checking Rd \usage sections ... WARNING
    ```
    Documented arguments not in \usage in documentation object 'add_vsd':
      ‘pbulk’ ‘vsd_path’
    
    Functions with \usage entries need to have the appropriate \alias
    entries, and all their arguments documented.
    The \usage entries must correspond to syntactically valid R code.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘RColorBrewer’ ‘statmod’
      All declared Imports should be used.
    Unexported object imported by a ':::' call: ‘GEOquery:::parseGSEMatrix’
      See the note in ?`:::` about the use of this operator.
    There are ::: calls to the package's namespace in its code. A package
      almost never needs to use ::: for its own objects:
      ‘getDirListing’ ‘getGEO’ ‘getGEOSuppFiles’
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘GeneMeta’
    ```

# cSEM

<details>

* Version: 0.4.0
* GitHub: https://github.com/M-E-Rademaker/cSEM
* Source code: https://github.com/cran/cSEM
* Date/Publication: 2021-04-19 22:00:18 UTC
* Number of recursive dependencies: 122

Run `revdep_details(, "cSEM")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘Rdpack’
      All declared Imports should be used.
    ```

# cytomapper

<details>

* Version: 1.6.0
* GitHub: https://github.com/BodenmillerGroup/cytomapper
* Source code: https://github.com/cran/cytomapper
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 139

Run `revdep_details(, "cytomapper")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.4Mb
      sub-directories of 1Mb or more:
        R     1.2Mb
        doc   4.9Mb
    ```

# CytoTree

<details>

* Version: 1.4.0
* GitHub: https://github.com/JhuangLab/CytoTree
* Source code: https://github.com/cran/CytoTree
* Date/Publication: 2021-12-14
* Number of recursive dependencies: 253

Run `revdep_details(, "CytoTree")` for more info

</details>

## In both

<<<<<<< HEAD
*   checking installed package size ... NOTE
||||||| cc87037
*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘destiny’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# dearseq

<details>

* Version: 1.4.0
* GitHub: https://github.com/borishejblum/dearseq
* Source code: https://github.com/cran/dearseq
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 141

Run `revdep_details(, "dearseq")` for more info

</details>

## In both

*   checking re-building of vignette outputs ... WARNING
=======
*   checking installed package size ... NOTE
    ```
      installed size is  6.6Mb
      sub-directories of 1Mb or more:
        doc       3.0Mb
        figures   1.2Mb
        libs      1.2Mb
    ```

# DCEtool

<details>

* Version: 1.0.0
* GitHub: NA
* Source code: https://github.com/cran/DCEtool
* Date/Publication: 2022-04-14 22:22:34 UTC
* Number of recursive dependencies: 141

Run `revdep_details(, "DCEtool")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘bslib’ ‘knitr’ ‘matrixStats’ ‘rmarkdown’ ‘shinythemes’ ‘tidyverse’
      All declared Imports should be used.
    ```

# dearseq

<details>

* Version: 1.6.0
* GitHub: https://github.com/borishejblum/dearseq
* Source code: https://github.com/cran/dearseq
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 153

Run `revdep_details(, "dearseq")` for more info

</details>

## In both

*   checking re-building of vignette outputs ... WARNING
>>>>>>> develop
    ```
<<<<<<< HEAD
      installed size is  6.6Mb
      sub-directories of 1Mb or more:
        doc       3.0Mb
        figures   1.2Mb
        libs      1.2Mb
||||||| cc87037
    Error(s) in re-building vignettes:
    --- re-building ‘dearseqUserguide.Rmd’ using rmarkdown
    Loading required package: Biobase
    Loading required package: BiocGenerics
    Loading required package: parallel
    
    Attaching package: 'BiocGenerics'
    
    The following objects are masked from 'package:parallel':
    
    ...
    Quitting from lines 152-164 (dearseqUserguide.Rmd) 
    Error: processing vignette 'dearseqUserguide.Rmd' failed with diagnostics:
    package 'MatrixGenerics' required by 'SummarizedExperiment' could not be found
    --- failed re-building ‘dearseqUserguide.Rmd’
    
    SUMMARY: processing the following file failed:
      ‘dearseqUserguide.Rmd’
    
    Error: Vignette re-building failed.
    Execution halted
=======
    Error(s) in re-building vignettes:
    --- re-building ‘dearseqUserguide.Rmd’ using rmarkdown
    Loading required package: Biobase
    Loading required package: BiocGenerics
    
    Attaching package: 'BiocGenerics'
    
    The following objects are masked from 'package:stats':
    
        IQR, mad, sd, var, xtabs
    ...
    Quitting from lines 186-211 (dearseqUserguide.Rmd) 
    Error: processing vignette 'dearseqUserguide.Rmd' failed with diagnostics:
    'x' must be an array of at least two dimensions
    --- failed re-building ‘dearseqUserguide.Rmd’
    
    SUMMARY: processing the following file failed:
      ‘dearseqUserguide.Rmd’
    
    Error: Vignette re-building failed.
    Execution halted
>>>>>>> develop
    ```

# decompr

<details>

* Version: 6.2.0
* GitHub: https://github.com/bquast/decompr
* Source code: https://github.com/cran/decompr
* Date/Publication: 2022-01-10 17:22:43 UTC
* Number of recursive dependencies: 50

Run `revdep_details(, "decompr")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘gvc’
    ```

# DeepBlueR

<details>

* Version: 1.20.0
* GitHub: NA
* Source code: https://github.com/cran/DeepBlueR
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 156

Run `revdep_details(, "DeepBlueR")` for more info

</details>

## In both

*   checking Rd files ... NOTE
    ```
    prepare_Rd: deepblue_enrich_regions_fast.Rd:35-38: Dropping empty section \examples
    ```

# DelayedMatrixStats

<details>

* Version: 1.16.0
* GitHub: https://github.com/PeteHaitch/DelayedMatrixStats
* Source code: https://github.com/cran/DelayedMatrixStats
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 75

Run `revdep_details(, "DelayedMatrixStats")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Missing object imported by a ':::' call: ‘DelayedArray:::.reduce_array_dimensions’
    Unexported objects imported by ':::' calls:
      ‘DelayedArray:::.get_ans_type’ ‘DelayedArray:::RleArraySeed’
      ‘DelayedArray:::get_Nindex_lengths’ ‘DelayedArray:::set_dim’
      ‘DelayedArray:::set_dimnames’ ‘DelayedArray:::subset_by_Nindex’
      ‘DelayedArray:::to_linear_index’
      See the note in ?`:::` about the use of this operator.
    ```

# DeMixT

<details>

* Version: 1.10.0
* GitHub: NA
* Source code: https://github.com/cran/DeMixT
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 69

Run `revdep_details(, "DeMixT")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      'parallel', 'Rcpp', 'SummarizedExperiment', 'knitr', 'KernSmooth',
      'matrixcalc', 'rmarkdown'
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.2Mb
      sub-directories of 1Mb or more:
        data   3.8Mb
        libs   2.7Mb
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    'library' or 'require' call not declared from: ‘calibrate’
    ```

# DepecheR

<details>

* Version: 1.10.0
* GitHub: NA
* Source code: https://github.com/cran/DepecheR
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 114

Run `revdep_details(, "DepecheR")` for more info

</details>

## In both

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Package listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘reshape2’
    A package should be listed in only one of these fields.
    ```

<<<<<<< HEAD
*   checking compiled code ... NOTE
    ```
    File ‘DepecheR/libs/DepecheR.so’:
      Found ‘rand’, possibly from ‘rand’ (C)
        Object: ‘Clusterer.o’
      Found ‘srand’, possibly from ‘srand’ (C)
        Objects: ‘Clusterer.o’, ‘InterfaceUtils.o’
    
    Compiled code should not call entry points which might terminate R nor
    write to stdout/stderr instead of to the console, nor use Fortran I/O
    nor system RNGs.
    
    See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
    ```

||||||| cc87037
*   checking R code for possible problems ... NOTE
    ```
    depecheCoFunction : <anonymous>: possible error in
      dAllocate(inDataFrame = selectionDataSet, clusterCenters = x, log2Off
      = TRUE, noZeroNum = FALSE): unused arguments (clusterCenters = x,
      log2Off = TRUE, noZeroNum = FALSE)
    depecheCoFunction: possible error in dAllocate(inDataFrameScaled,
      reducedClusterCenters, log2Off = TRUE, noZeroNum = FALSE): unused
      arguments (log2Off = TRUE, noZeroNum = FALSE)
    ```

*   checking compiled code ... NOTE
    ```
    File ‘DepecheR/libs/DepecheR.so’:
      Found ‘rand’, possibly from ‘rand’ (C)
        Object: ‘Clusterer.o’
      Found ‘srand’, possibly from ‘srand’ (C)
        Objects: ‘Clusterer.o’, ‘InterfaceUtils.o’
    
    Compiled code should not call entry points which might terminate R nor
    write to stdout/stderr instead of to the console, nor use Fortran I/O
    nor system RNGs.
    
    See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
    ```

=======
>>>>>>> develop
# DEqMS

<details>

<<<<<<< HEAD
* Version: 1.11.1
||||||| cc87037
* Version: 1.10.0
=======
* Version: 1.12.1
>>>>>>> develop
* GitHub: https://github.com/yafeng/DEqMS
* Source code: https://github.com/cran/DEqMS
<<<<<<< HEAD
* Date/Publication: 2021-09-29
* Number of recursive dependencies: 116
||||||| cc87037
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 114
=======
* Date/Publication: 2022-01-20
* Number of recursive dependencies: 116
>>>>>>> develop

Run `revdep_details(, "DEqMS")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
<<<<<<< HEAD
    Running examples in ‘DEqMS-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: Residualplot
    > ### Title: plot the residuals against the number of quantified
    > ###   peptides/PSMs.
    > ### Aliases: Residualplot
    > 
    > ### ** Examples
    > 
    ...
    > eh = ExperimentHub(localHub=TRUE)
    Testing for internet connectivity via https_proxy... success!
    snapshotDate(): 2022-03-16
    > query(eh, "DEqMS")
    ExperimentHub with 0 records
    # snapshotDate(): 2022-03-16
    > dat.psm = eh[["EH1663"]]
    Error: File not previously downloaded.
      Run with 'localHub=FALSE'
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
      ...
    --- re-building ‘DEqMS-package-vignette.Rmd’ using rmarkdown
    trying URL 'ftp://ftp.pride.ebi.ac.uk/pride/data/archive/2016/06/PXD004163/Yan_miR_Protein_table.flatprottable.txt'
    Quitting from lines 64-69 (DEqMS-package-vignette.Rmd) 
    Error: processing vignette 'DEqMS-package-vignette.Rmd' failed with diagnostics:
    cannot open URL 'ftp://ftp.pride.ebi.ac.uk/pride/data/archive/2016/06/PXD004163/Yan_miR_Protein_table.flatprottable.txt'
    --- failed re-building ‘DEqMS-package-vignette.Rmd’
    
    SUMMARY: processing the following file failed:
      ‘DEqMS-package-vignette.Rmd’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Title field: should not end in a period.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Package in Depends field not imported from: ‘matrixStats’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

# detrendr

||||||| cc87037

# detrendr

=======

*   checking dependencies in R code ... NOTE
    ```
    Package in Depends field not imported from: ‘matrixStats’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

# detrendr

>>>>>>> develop
<details>

* Version: 0.6.14
* GitHub: https://github.com/rorynolan/detrendr
* Source code: https://github.com/cran/detrendr
* Date/Publication: 2021-05-16 05:00:05 UTC
* Number of recursive dependencies: 114

Run `revdep_details(, "detrendr")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  8.8Mb
      sub-directories of 1Mb or more:
        libs   7.7Mb
    ```

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

# DHS.rates

<details>

* Version: 0.9.1
* GitHub: NA
* Source code: https://github.com/cran/DHS.rates
* Date/Publication: 2021-12-09 19:00:02 UTC
* Number of recursive dependencies: 62

Run `revdep_details(, "DHS.rates")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘rlang’
      All declared Imports should be used.
    ```

# diffloop

<details>

* Version: 1.22.0
* GitHub: https://github.com/aryeelab/diffloop
* Source code: https://github.com/cran/diffloop
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 151

Run `revdep_details(, "diffloop")` for more info

</details>

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

# diffUTR

<details>

* Version: 1.2.0
* GitHub: https://github.com/ETHZ-INS/diffUTR
* Source code: https://github.com/cran/diffUTR
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 145

Run `revdep_details(, "diffUTR")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    deuBinPlot: no visible binding for global variable 'x_start'
    deuBinPlot: no visible binding for global variable 'x_end'
    deuBinPlot: no visible binding for global variable 'y_start'
    deuBinPlot: no visible binding for global variable 'y_end'
    plotTopGenes: no visible binding for global variable 'sizeScore'
    plotTopGenes: no visible binding for global variable 'q.value'
    plotTopGenes: no visible binding for global variable 'density.ratio'
    plotTopGenes: no visible binding for global variable 'geneMeanDensity'
    plotTopGenes: no visible binding for global variable 'w.abs.coef'
    plotTopGenes: no visible binding for global variable 'tmp'
    plotTopGenes: no visible binding for global variable 'name'
    Undefined global functions or variables:
      density.ratio geneMeanDensity name q.value sizeScore tmp w.abs.coef
      x_end x_start y_end y_start
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    '::' or ':::' import not declared from: ‘ggbio’
    ```

# DiscoRhythm

<details>

<<<<<<< HEAD
* Version: 1.10.0
||||||| cc87037
* Version: 1.8.0
=======
* Version: 1.10.1
>>>>>>> develop
* GitHub: https://github.com/matthewcarlucci/DiscoRhythm
* Source code: https://github.com/cran/DiscoRhythm
<<<<<<< HEAD
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 155
||||||| cc87037
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 154
=======
* Date/Publication: 2022-03-08
* Number of recursive dependencies: 155
>>>>>>> develop

Run `revdep_details(, "DiscoRhythm")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘data.table’
      All declared Imports should be used.
    ```

*   checking R code for possible problems ... NOTE
    ```
    discoApp: no visible binding for global variable ‘.discorhythm_ncores’
    discoApp: no visible binding for global variable ‘.discorhythm_local’
    discoBatch: no visible binding for global variable ‘discoODAres’
    Undefined global functions or variables:
      .discorhythm_local .discorhythm_ncores discoODAres
    ```

# DMCFB

<details>

* Version: 1.8.0
* GitHub: https://github.com/shokoohi/DMCFB
* Source code: https://github.com/cran/DMCFB
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 110

Run `revdep_details(, "DMCFB")` for more info

</details>

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
      ...
    --- re-building ‘DMCFB.Rmd’ using rmarkdown
    Error: processing vignette 'DMCFB.Rmd' failed with diagnostics:
    there is no package called ‘BiocStyle’
    --- failed re-building ‘DMCFB.Rmd’
    
    SUMMARY: processing the following file failed:
      ‘DMCFB.Rmd’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

# dmrseq

<details>

* Version: 1.14.0
* GitHub: NA
* Source code: https://github.com/cran/dmrseq
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 163

Run `revdep_details(, "dmrseq")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘dmrseq-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: getAnnot
    > ### Title: Retrieve annotation information
    > ### Aliases: getAnnot
    > 
    > ### ** Examples
    > 
    > 
    ...
    Trying again (2 attempts remaining)
    Error in get(txdb_name) : 
      object 'TxDb.Hsapiens.UCSC.hg19.knownGene' not found
    
    Trying again (1 attempts remaining)
    Error in get(txdb_name) : 
      object 'TxDb.Hsapiens.UCSC.hg19.knownGene' not found
    
    Error in getAnnot("hg19") : Annotation could not be retrieved.
    Execution halted
    ```

*   checking LazyData ... WARNING
    ```
      LazyData DB of 8.8 MB without LazyDataCompression set
      See §1.1.6 of 'Writing R Extensions'
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
    --- re-building ‘dmrseq.Rmd’ using rmarkdown
    Loading required package: bsseq
    Loading required package: BiocGenerics
    
    Attaching package: 'BiocGenerics'
    
    The following objects are masked from 'package:stats':
    
        IQR, mad, sd, var, xtabs
    ...
    Quitting from lines 543-548 (dmrseq.Rmd) 
    Error: processing vignette 'dmrseq.Rmd' failed with diagnostics:
    Annotation could not be retrieved.
    --- failed re-building ‘dmrseq.Rmd’
    
    SUMMARY: processing the following file failed:
      ‘dmrseq.Rmd’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 10.3Mb
      sub-directories of 1Mb or more:
        data   8.8Mb
        doc    1.3Mb
    ```

# Doscheda

<details>

* Version: 1.16.0
* GitHub: NA
* Source code: https://github.com/cran/Doscheda
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 151

Run `revdep_details(, "Doscheda")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.9Mb
      sub-directories of 1Mb or more:
        data             2.1Mb
        doc              1.6Mb
        shiny-examples   1.9Mb
    ```

# DSWE

<details>

* Version: 1.5.1
* GitHub: https://github.com/TAMU-AML/DSWE-Package
* Source code: https://github.com/cran/DSWE
* Date/Publication: 2021-01-11 10:30:07 UTC
* Number of recursive dependencies: 18

Run `revdep_details(, "DSWE")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.1Mb
      sub-directories of 1Mb or more:
        data   2.8Mb
        libs   4.1Mb
    ```

# dtree

<details>

* Version: 0.4.2
* GitHub: NA
* Source code: https://github.com/cran/dtree
* Date/Publication: 2018-01-18 23:57:29 UTC
* Number of recursive dependencies: 91

Run `revdep_details(, "dtree")` for more info

</details>

## In both

*   checking LazyData ... NOTE
    ```
      'LazyData' is specified without a 'data' directory
    ```

<<<<<<< HEAD
# dwp

<details>

* Version: 1.0
* GitHub: NA
* Source code: https://github.com/cran/dwp
* Date/Publication: 2022-01-27 19:40:08 UTC
* Number of recursive dependencies: 69

Run `revdep_details(, "dwp")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘GenEst’ ‘expint’ ‘numDeriv’
      All declared Imports should be used.
    ```

# easier

<details>

* Version: 1.0.0
* GitHub: NA
* Source code: https://github.com/cran/easier
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 196

Run `revdep_details(, "easier")` for more info

</details>

## Newly fixed

*   checking examples ... ERROR
    ```
    Running examples in ‘easier-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: assess_immune_response
    > ### Title: Assess easier score as predictor of patients' immune response
    > ### Aliases: assess_immune_response
    > 
    > ### ** Examples
    > 
    > # using a SummarizedExperiment object
    ...
    Testing for internet connectivity via https_proxy... success!
    snapshotDate(): 2021-10-19
    Retrieving Mariathasan2018_PDL1_treatment dataset...
    see ?easierData and browseVignettes('easierData') for documentation
    loading from cache
    Error: failed to load resource
      name: EH6677
      title: Mariathasan2018_PDL1_treatment
      reason: error in evaluating the argument 'x' in selecting a method for function 'get': error reading from connection
    Execution halted
    ```

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘coin’
      All declared Imports should be used.
    ```

||||||| cc87037
=======
# dwp

<details>

* Version: 1.0
* GitHub: NA
* Source code: https://github.com/cran/dwp
* Date/Publication: 2022-01-27 19:40:08 UTC
* Number of recursive dependencies: 69

Run `revdep_details(, "dwp")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘GenEst’ ‘expint’ ‘numDeriv’
      All declared Imports should be used.
    ```

# easier

<details>

* Version: 1.0.0
* GitHub: NA
* Source code: https://github.com/cran/easier
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 196

Run `revdep_details(, "easier")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘coin’
      All declared Imports should be used.
    ```

>>>>>>> develop
# ecospat

<details>

* Version: 3.2.1
* GitHub: https://github.com/ecospat/ecospat
* Source code: https://github.com/cran/ecospat
* Date/Publication: 2022-01-07 14:02:45 UTC
* Number of recursive dependencies: 244

Run `revdep_details(, "ecospat")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘maps’
      All declared Imports should be used.
    ```

# EMDomics

<details>

* Version: 2.24.0
* GitHub: NA
* Source code: https://github.com/cran/EMDomics
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 49

Run `revdep_details(, "EMDomics")` for more info

</details>

## In both

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Authors@R field gives more than one person with maintainer role:
      Sadhika Malladi <contact@sadhikamalladi.com> [aut, cre]
      Daniel Schmolze <emd@schmolze.com> [aut, cre]
    ```

# EnrichedHeatmap

<details>

* Version: 1.24.0
* GitHub: https://github.com/jokergoo/EnrichedHeatmap
* Source code: https://github.com/cran/EnrichedHeatmap
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 101

Run `revdep_details(, "EnrichedHeatmap")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  9.7Mb
      sub-directories of 1Mb or more:
        doc       6.0Mb
        extdata   2.5Mb
        libs      1.0Mb
    ```

# EpiDISH

<details>

* Version: 2.10.0
* GitHub: https://github.com/sjczheng/EpiDISH
* Source code: https://github.com/cran/EpiDISH
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 92

Run `revdep_details(, "EpiDISH")` for more info

</details>

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

# eseis

<details>

* Version: 0.6.0
* GitHub: NA
* Source code: https://github.com/cran/eseis
* Date/Publication: 2021-11-26 14:00:02 UTC
* Number of recursive dependencies: 82

Run `revdep_details(, "eseis")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘EMMAgeo’ ‘limSolve’ ‘shinyFiles’
      All declared Imports should be used.
    ```

# EstimationTools

<details>

* Version: 2.1.0
* GitHub: https://github.com/Jaimemosg/EstimationTools
* Source code: https://github.com/cran/EstimationTools
* Date/Publication: 2021-03-10 18:40:05 UTC
* Number of recursive dependencies: 112

Run `revdep_details(, "EstimationTools")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘matrixStats’
      All declared Imports should be used.
    ```

# estudy2

<details>

* Version: 0.10.0
* GitHub: https://github.com/irudnyts/estudy2
* Source code: https://github.com/cran/estudy2
* Date/Publication: 2021-11-15 12:50:07 UTC
* Number of recursive dependencies: 63

Run `revdep_details(, "estudy2")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘shinyWidgets’
    ```

# evaluomeR

<details>

* Version: 1.10.0
* GitHub: https://github.com/neobernad/evaluomeR
* Source code: https://github.com/cran/evaluomeR
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 112

Run `revdep_details(, "evaluomeR")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      'SummarizedExperiment', 'MultiAssayExperiment', 'cluster', 'fpc',
      'randomForest', 'flexmix'
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking top-level files ... NOTE
    ```
    File
      LICENSE
    is not mentioned in the DESCRIPTION file.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘kableExtra’
      All declared Imports should be used.
    Packages in Depends field not imported from:
      ‘flexmix’ ‘randomForest’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

# EventPointer

<details>

* Version: 3.2.0
* GitHub: https://github.com/jpromeror/EventPointer
* Source code: https://github.com/cran/EventPointer
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 169

Run `revdep_details(, "EventPointer")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘EventPointer-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: CDFfromGTF
    > ### Title: CDF file creation for EventPointer
    > ### Aliases: CDFfromGTF
    > 
    > ### ** Examples
    > 
    >    PathFiles<-system.file('extdata',package='EventPointer')
    ...
    Indexing Genes and Probes...Done
    Finding Events...
    
      |                                                                            
      |                                                                      |   0%
      |                                                                            
      |======================================================================| 100%Error in which(Adj > 0, arr.ind = 1)[, 2] : 
      incorrect number of dimensions
    Calls: CDFfromGTF ... SG_creation_RNASeq -> cbind -> standardGeneric -> eval -> eval -> eval
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
    --- re-building ‘EventPointer.Rmd’ using rmarkdown
    Loading required package: SGSeq
    Loading required package: IRanges
    Loading required package: BiocGenerics
    
    Attaching package: 'BiocGenerics'
    
    The following objects are masked from 'package:stats':
    
    ...
    Quitting from lines 131-146 (EventPointer.Rmd) 
    Error: processing vignette 'EventPointer.Rmd' failed with diagnostics:
    incorrect number of dimensions
    --- failed re-building ‘EventPointer.Rmd’
    
    SUMMARY: processing the following file failed:
      ‘EventPointer.Rmd’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Authors@R field gives persons with no role:
      Pablo Sacristan
      Ander Muniategui
      Fernando Carazo
      Ander Aramburu
      Angel Rubio
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      ‘SGSeq:::addFeatureID’ ‘SGSeq:::addGeneID’ ‘SGSeq:::edges’
      ‘SGSeq:::exonGraph’ ‘SGSeq:::feature2name’ ‘SGSeq:::matchSGFeatures’
      ‘SGSeq:::nodes’ ‘SGSeq:::propagateAnnotation’
      ‘SGSeq:::splitCharacterList’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    EventDetection_transcriptome: no visible binding for global variable
      ‘jj’
    annotate2: no visible global function definition for
      ‘addDummySpliceSites’
    annotate2: no visible global function definition for ‘annotatePaths’
    annotateFeatures2: no visible global function definition for
      ‘matchTxFeatures’
    callGRseq_parallel: no visible global function definition for
      ‘makeCluster’
    callGRseq_parallel: no visible binding for global variable ‘i’
    ...
    call_get_table_Bootstrap: no visible binding for global variable ‘ii’
    call_get_table_Bootstrap: no visible global function definition for
      ‘stopCluster’
    fitgl: no visible binding for global variable ‘hist’
    Undefined global functions or variables:
      addDummySpliceSites annotatePaths hist i ii jj makeCluster
      matchTxFeatures stopCluster
    Consider adding
      importFrom("graphics", "hist")
    to your NAMESPACE file.
    ```

# ExCluster

<details>

* Version: 1.12.0
* GitHub: NA
* Source code: https://github.com/cran/ExCluster
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 36

Run `revdep_details(, "ExCluster")` for more info

</details>

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .BBSoptions
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

# ExpHunterSuite

<details>

* Version: 1.2.0
* GitHub: NA
* Source code: https://github.com/cran/ExpHunterSuite
* Date/Publication: 2021-10-29
* Number of recursive dependencies: 219

Run `revdep_details(, "ExpHunterSuite")` for more info

</details>

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
      ...
    --- re-building ‘hunter.Rmd’ using rmarkdown
    Quitting from lines 2-51 (hunter.Rmd) 
    Error: processing vignette 'hunter.Rmd' failed with diagnostics:
    there is no package called 'BiocManager'
    --- failed re-building ‘hunter.Rmd’
    
    SUMMARY: processing the following file failed:
      ‘hunter.Rmd’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

*   checking R code for possible problems ... NOTE
    ```
    functional_hunter: no visible binding for global variable ‘genes_tag’
    get_stats_by_group: no visible binding for global variable
      ‘known_miRNA’
    get_stats_by_group: no visible binding for global variable
      ‘possible_positives’
    main_targets_functional: no visible binding for global variable
      ‘main_path_script’
    Undefined global functions or variables:
      genes_tag known_miRNA main_path_script possible_positives
    ```

# ExpressionNormalizationWorkflow

<details>

* Version: 1.20.0
* GitHub: https://github.com/NA/NA
* Source code: https://github.com/cran/ExpressionNormalizationWorkflow
* Date/Publication: 2021-10-29
* Number of recursive dependencies: 123

Run `revdep_details(, "ExpressionNormalizationWorkflow")` for more info

</details>

## In both

*   checking whether package ‘ExpressionNormalizationWorkflow’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: /c4/home/henrik/repositories/matrixStats/revdep/checks/ExpressionNormalizationWorkflow/new/ExpressionNormalizationWorkflow.Rcheck/00_pkg_src/ExpressionNormalizationWorkflow/man/ExpressionNormalizationWorkflow-package.Rd:29: All text must be in a section
      Warning: /c4/home/henrik/repositories/matrixStats/revdep/checks/ExpressionNormalizationWorkflow/new/ExpressionNormalizationWorkflow.Rcheck/00_pkg_src/ExpressionNormalizationWorkflow/man/ExpressionNormalizationWorkflow-package.Rd:30: All text must be in a section
      Warning: replacing previous import ‘Biobase::anyMissing’ by ‘matrixStats::anyMissing’ when loading ‘ExpressionNormalizationWorkflow’
      Warning: replacing previous import ‘Biobase::rowMedians’ by ‘matrixStats::rowMedians’ when loading ‘ExpressionNormalizationWorkflow’
    See ‘/c4/home/henrik/repositories/matrixStats/revdep/checks/ExpressionNormalizationWorkflow/new/ExpressionNormalizationWorkflow.Rcheck/00install.out’ for details.
    ```

*   checking Rd files ... WARNING
    ```
    prepare_Rd: ExpressionNormalizationWorkflow-package.Rd:29: All text must be in a section
    prepare_Rd: ExpressionNormalizationWorkflow-package.Rd:30: All text must be in a section
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.2Mb
      sub-directories of 1Mb or more:
        doc       2.5Mb
        extdata   4.7Mb
    ```

*   checking Rd cross-references ... NOTE
    ```
    Unknown package ‘<pkg>’ in Rd xrefs
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    '::' or ':::' import not declared from: ‘BiocManager’
    ```

# FastHCS

<details>

* Version: 0.0.7
* GitHub: NA
* Source code: https://github.com/cran/FastHCS
* Date/Publication: 2020-05-10 12:50:03 UTC
* Number of recursive dependencies: 5

Run `revdep_details(, "FastHCS")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 19.8Mb
      sub-directories of 1Mb or more:
        data   1.3Mb
        libs  18.5Mb
    ```

# FastPCS

<details>

* Version: 0.1.3
* GitHub: NA
* Source code: https://github.com/cran/FastPCS
* Date/Publication: 2018-05-22 16:49:12 UTC
* Number of recursive dependencies: 6

Run `revdep_details(, "FastPCS")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 10.1Mb
      sub-directories of 1Mb or more:
        libs  10.1Mb
    ```

# FastRCS

<details>

* Version: 0.0.8
* GitHub: NA
* Source code: https://github.com/cran/FastRCS
* Date/Publication: 2018-05-13 19:59:51 UTC
* Number of recursive dependencies: 6

Run `revdep_details(, "FastRCS")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 21.8Mb
      sub-directories of 1Mb or more:
        libs  21.8Mb
    ```

# fastverse

<details>

* Version: 0.2.3
* GitHub: https://github.com/SebKrantz/fastverse
* Source code: https://github.com/cran/fastverse
* Date/Publication: 2022-02-17 13:02:06 UTC
* Number of recursive dependencies: 54

Run `revdep_details(, "fastverse")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘collapse’ ‘data.table’ ‘fst’ ‘kit’ ‘magrittr’ ‘matrixStats’
      All declared Imports should be used.
    ```

# FDb.FANTOM4.promoters.hg19

<details>

* Version: 1.0.0
* GitHub: NA
* Source code: https://github.com/cran/FDb.FANTOM4.promoters.hg19
* Number of recursive dependencies: 89

Run `revdep_details(, "FDb.FANTOM4.promoters.hg19")` for more info

</details>

## In both

*   checking package subdirectories ... WARNING
    ```
    Found the following non-empty subdirectories of ‘inst’ also used by R:
      inst/build
    It is recommended not to interfere with package subdirectories used by
    R.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    Package listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘GenomicFeatures’
    A package should be listed in only one of these fields.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls in package code:
      ‘BSgenome.Hsapiens.UCSC.hg19’ ‘matrixStats’
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    Package in Depends field not imported from: ‘Biostrings’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ':::' call which should be '::': ‘matrixStats:::rowProds’
      See the note in ?`:::` about the use of this operator.
    ```

# FEAST

<details>

* Version: 1.2.0
* GitHub: https://github.com/suke18/FEAST
* Source code: https://github.com/cran/FEAST
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 219

Run `revdep_details(, "FEAST")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.3Mb
      sub-directories of 1Mb or more:
        data   3.9Mb
        doc    1.3Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    FEAST: multiple local function definitions for ‘bp_fun’ with different
      formal arguments
    SC3_Clust: no visible global function definition for ‘metadata’
    Visual_Rslt: no visible global function definition for ‘ggbarplot’
    Visual_Rslt: no visible global function definition for ‘theme’
    Visual_Rslt: no visible global function definition for ‘element_text’
    Visual_Rslt: no visible global function definition for ‘ggline’
    Visual_Rslt: no visible global function definition for ‘get_palette’
    Visual_Rslt: no visible global function definition for
      ‘scale_y_continuous’
    Visual_Rslt: no visible global function definition for ‘guides’
    Visual_Rslt: no visible global function definition for ‘guide_legend’
    Visual_Rslt: no visible global function definition for ‘ggarrange’
    aricode_NMI : entropy: no visible global function definition for
      ‘sortPairs’
    Undefined global functions or variables:
<<<<<<< HEAD
      element_text get_palette ggarrange ggbarplot ggline guide_legend
      guides metadata scale_y_continuous sortPairs theme
    ```

# fergm

<details>

* Version: 1.1.4
* GitHub: https://github.com/benjamin-w-campbell/fergm
* Source code: https://github.com/cran/fergm
* Date/Publication: 2018-10-17 22:20:11 UTC
* Number of recursive dependencies: 93

Run `revdep_details(, "fergm")` for more info

</details>

## In both

*   checking contents of ‘data’ directory ... WARNING
    ```
    Output for data("fergm.fit", package = "fergm"):
      Warning: namespace ‘DBI’ is not available and has been replaced
      by .GlobalEnv when processing object ‘fergm.fit’
      Warning: namespace ‘DBI’ is not available and has been replaced
      by .GlobalEnv when processing object ‘fergm.fit’
      Warning: namespace ‘DBI’ is not available and has been replaced
      by .GlobalEnv when processing object ‘fergm.fit’
      Warning: namespace ‘DBI’ is not available and has been replaced
      by .GlobalEnv when processing object ‘fergm.fit’
||||||| cc87037
      CreateSeuratObject FindClusters FindNeighbors FindVariableFeatures
      Idents NormalizeData RunPCA ScaleData VariableFeatures element_text
      get_palette ggarrange ggbarplot ggline guide_legend guides metadata
      scale_y_continuous sortPairs theme
    ```

# fergm

<details>

* Version: 1.1.4
* GitHub: https://github.com/benjamin-w-campbell/fergm
* Source code: https://github.com/cran/fergm
* Date/Publication: 2018-10-17 22:20:11 UTC
* Number of recursive dependencies: 91

Run `revdep_details(, "fergm")` for more info

</details>

## In both

*   checking contents of ‘data’ directory ... WARNING
    ```
    Output for data("fergm.fit", package = "fergm"):
      Warning: namespace ‘DBI’ is not available and has been replaced
      by .GlobalEnv when processing object ‘fergm.fit’
      Warning: namespace ‘DBI’ is not available and has been replaced
      by .GlobalEnv when processing object ‘fergm.fit’
      Warning: namespace ‘DBI’ is not available and has been replaced
      by .GlobalEnv when processing object ‘fergm.fit’
      Warning: namespace ‘DBI’ is not available and has been replaced
      by .GlobalEnv when processing object ‘fergm.fit’
=======
      element_text get_palette ggarrange ggbarplot ggline guide_legend
      guides metadata scale_y_continuous sortPairs theme
>>>>>>> develop
    ```

# fipp

<details>

* Version: 1.0.0
* GitHub: NA
* Source code: https://github.com/cran/fipp
* Date/Publication: 2021-02-11 10:30:02 UTC
* Number of recursive dependencies: 26

Run `revdep_details(, "fipp")` for more info

</details>

## In both

*   checking LazyData ... NOTE
    ```
      'LazyData' is specified without a 'data' directory
    ```

# fishpond

<details>

* Version: 2.0.1
* GitHub: https://github.com/mikelove/fishpond
* Source code: https://github.com/cran/fishpond
* Date/Publication: 2021-12-02
* Number of recursive dependencies: 179

Run `revdep_details(, "fishpond")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      'samr:::localfdr' 'samr:::predictlocalfdr' 'samr:::qvalue.func'
      'samr:::samr.compute.delta.table.seq'
      See the note in ?`:::` about the use of this operator.
    ```

# flowCore

<details>

* Version: 2.6.0
* GitHub: NA
* Source code: https://github.com/cran/flowCore
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 134

Run `revdep_details(, "flowCore")` for more info

</details>

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
    --- re-building ‘HowTo-flowCore.Rnw’ using knitr
    Warning: 'getData' is deprecated.
    Use 'gs_pop_get_data' instead.
    See help("Deprecated")
    Warning: 'flowJo_biexp_trans' is deprecated.
    Use 'flowjo_biexp_trans' instead.
    See help("Deprecated")
    Warning: 'getData' is deprecated.
    Use 'gs_pop_get_data' instead.
    ...
    l.76 \usepackage
                    {graphicx}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    --- failed re-building ‘HowTo-flowCore.Rnw’
    
    SUMMARY: processing the following file failed:
      ‘HowTo-flowCore.Rnw’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 18.3Mb
      sub-directories of 1Mb or more:
        R         1.1Mb
        data      5.0Mb
        extdata   1.1Mb
        libs      9.9Mb
    ```

# flowGraph

<details>

* Version: 1.2.0
* GitHub: https://github.com/aya49/flowGraph
* Source code: https://github.com/cran/flowGraph
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 87

Run `revdep_details(, "flowGraph")` for more info

</details>

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
    --- re-building ‘flowGraph.Rmd’ using rmarkdown
    - 8 pops @ layer 1
<<<<<<< HEAD
    15:24:49-15:24:49 > 16:00:00
||||||| cc87037
    14:50:42-14:50:42 > 16:00:00
=======
    19:23:37-19:23:37 > 16:00:00
>>>>>>> develop
    - 24 pops @ layer 2
<<<<<<< HEAD
    15:24:49-15:24:49 > 16:00:00
||||||| cc87037
    14:50:42-14:50:42 > 16:00:00
=======
    19:23:37-19:23:37 > 16:00:00
>>>>>>> develop
    - 32 pops @ layer 3
<<<<<<< HEAD
    15:24:49-15:24:49 > 16:00:00
||||||| cc87037
    14:50:42-14:50:42 > 16:00:00
=======
    19:23:37-19:23:37 > 16:00:00
>>>>>>> develop
    - 16 pops @ layer 4
<<<<<<< HEAD
    15:24:49-15:24:49 > 16:00:00
||||||| cc87037
    14:50:42-14:50:42 > 16:00:00
=======
    19:23:37-19:23:37 > 16:00:00
>>>>>>> develop
    ...
    Quitting from lines 557-561 (flowGraph.Rmd) 
    Error: processing vignette 'flowGraph.Rmd' failed with diagnostics:
    `map_df()` requires dplyr
    --- failed re-building ‘flowGraph.Rmd’
    
    SUMMARY: processing the following file failed:
      ‘flowGraph.Rmd’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

*   checking R code for possible problems ... NOTE
    ```
    get_child: no visible binding for global variable ‘no_cores’
    get_paren: no visible binding for global variable ‘no_cores’
    ms_psig: no visible binding for global variable ‘meta’
    Undefined global functions or variables:
      meta no_cores
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘doParallel’
    ```

# flowWorkspace

<details>

* Version: 4.6.0
* GitHub: NA
* Source code: https://github.com/cran/flowWorkspace
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 145

Run `revdep_details(, "flowWorkspace")` for more info

</details>

## In both

*   checking for missing documentation entries ... WARNING
    ```
    Undocumented code objects:
      ‘gs_clone’ ‘gs_copy_tree_only’
    Undocumented S4 methods:
      generic 'dimnames' and siglist 'cytoframe'
      generic 'rownames' and siglist 'cytoframe'
      generic 'rownames<-' and siglist 'cytoframe'
      generic 'show' and siglist 'cytoframe'
      generic 'transform' and siglist 'cytoframe'
    All user-level objects in a package (including S4 classes and methods)
    should have documentation entries.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking Rd \usage sections ... WARNING
    ```
    Documented arguments not in \usage in documentation object 'GatingSet-methods':
      ‘...’
    
    Undocumented arguments in documentation object 'cf_append_cols'
      ‘ctx’
    
    Undocumented arguments in documentation object 'convert'
      ‘backend’
    
    Undocumented arguments in documentation object 'cs_get_uri'
    ...
    The \usage entries must correspond to syntactically valid R code.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    S3 methods shown with full name in documentation object 'cytoctx':
      ‘print.cytoctx’
    
    The \usage entries for S3 methods should use the \method markup and not
    their full name.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking for portable use of $(BLAS_LIBS) and $(LAPACK_LIBS) ... WARNING
    ```
      apparently using $(BLAS_LIBS) without following $(FLIBS) in ‘src/Makevars’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 69.5Mb
      sub-directories of 1Mb or more:
        doc    3.4Mb
        libs  64.8Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Title field: should not end in a period.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘RBGL’ ‘RProtoBufLib’ ‘ggplot2’ ‘grDevices’ ‘graphics’ ‘utils’
      All declared Imports should be used.
    Unexported objects imported by ':::' calls:
      ‘DelayedArray:::simplify_NULL_dimnames’ ‘flowCore:::.estimateLogicle’
      ‘flowCore:::.spillover_pattern’ ‘flowCore:::checkClass’
      ‘flowCore:::guid’ ‘flowCore:::logicle_transform’
      ‘flowCore:::parse_pd_for_read_fs’ ‘flowCore:::txt2spillmatrix’
      ‘flowCore:::updateTransformKeywords’
      ‘flowCore:::validFilterResultList’ ‘graph:::.makeEdgeKeys’
      ‘ncdfFlow:::.isValidSamples’ ‘stats:::.splinefun’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking S3 generic/method consistency ... NOTE
    ```
    Found the following apparent S3 methods exported but not registered:
      print.cytoctx
    See section ‘Registering S3 methods’ in the ‘Writing R Extensions’
    manual.
    ```

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

<<<<<<< HEAD
# footBayes
||||||| cc87037
# FRASER
=======
# footBayes

<details>

* Version: 0.1.0
* GitHub: https://github.com/leoegidi/footbayes
* Source code: https://github.com/cran/footBayes
* Date/Publication: 2022-02-21 08:20:05 UTC
* Number of recursive dependencies: 140

Run `revdep_details(, "footBayes")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘tidyverse’
      All declared Imports should be used.
    ```

# FRASER
>>>>>>> develop

<details>

<<<<<<< HEAD
* Version: 0.1.0
* GitHub: https://github.com/leoegidi/footbayes
* Source code: https://github.com/cran/footBayes
* Date/Publication: 2022-02-21 08:20:05 UTC
* Number of recursive dependencies: 140
||||||| cc87037
* Version: 1.4.0
* GitHub: https://github.com/gagneurlab/FRASER
* Source code: https://github.com/cran/FRASER
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 185
=======
* Version: 1.6.1
* GitHub: https://github.com/gagneurlab/FRASER
* Source code: https://github.com/cran/FRASER
* Date/Publication: 2022-04-10
* Number of recursive dependencies: 188
>>>>>>> develop

Run `revdep_details(, "footBayes")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
<<<<<<< HEAD
    Namespace in Imports field not imported from: ‘tidyverse’
      All declared Imports should be used.
    ```

# FRASER

<details>

* Version: 1.6.0
* GitHub: https://github.com/gagneurlab/FRASER
* Source code: https://github.com/cran/FRASER
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 188

Run `revdep_details(, "FRASER")` for more info

</details>

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
    --- re-building ‘FRASER.Rnw’ using knitr
    Mon Mar 28 15:49:56 2022: Start counting the split reads ...
    Mon Mar 28 15:49:57 2022: Count split reads for sample: sample1
    Mon Mar 28 15:49:59 2022: Count split reads for sample: sample2
    Mon Mar 28 15:50:00 2022: Count split reads for sample: sample3
    Mon Mar 28 15:50:01 2022 : count ranges need to be merged ...
    Mon Mar 28 15:50:02 2022: Create splice site indices ...
    Mon Mar 28 15:50:02 2022: Writing split counts to folder: /scratch/henrik/457086/Rtmp3OWl0a/savedObjects/Data_Analysis/splitCounts
    Mon Mar 28 15:50:02 2022: Identifying introns with read count <= 20 in all samples...
    ...
    l.189 \RequirePackage
                         {parnotes}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    --- failed re-building ‘FRASER.Rnw’
||||||| cc87037
    Error(s) in re-building vignettes:
    --- re-building ‘FRASER.Rnw’ using knitr
    Wed Sep 15 05:54:45 2021: Start counting the split reads ...
    Wed Sep 15 05:54:45 2021: Count split reads for sample: sample1
    Wed Sep 15 05:54:50 2021: Count split reads for sample: sample2
    Wed Sep 15 05:54:51 2021: Count split reads for sample: sample3
    Wed Sep 15 05:54:53 2021 : count ranges need to be merged ...
    Wed Sep 15 05:54:54 2021: Create splice site indices ...
    Wed Sep 15 05:54:54 2021: Writing split counts to folder: /scratch/henrik/RtmpqPdiN5/savedObjects/Data_Analysis/splitCounts
    Wed Sep 15 05:54:54 2021: Identifying introns with read count <= 20 in all samples...
    ...
    l.189 \RequirePackage
                         {parnotes}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    --- failed re-building ‘FRASER.Rnw’
=======
    Error(s) in re-building vignettes:
    --- re-building ‘FRASER.Rnw’ using knitr
    Sat Apr 16 20:01:54 2022: Start counting the split reads ...
    Sat Apr 16 20:01:55 2022: Count split reads for sample: sample1
    Sat Apr 16 20:01:56 2022: Count split reads for sample: sample2
    Sat Apr 16 20:01:57 2022: Count split reads for sample: sample3
    Sat Apr 16 20:02:06 2022 : count ranges need to be merged ...
    Sat Apr 16 20:02:06 2022: Create splice site indices ...
    Sat Apr 16 20:02:06 2022: Writing split counts to folder: FRASER_output/savedObjects/Data_Analysis/splitCounts
    Sat Apr 16 20:02:07 2022: Identifying introns with read count <= 20 in all samples...
    ...
    l.189 \RequirePackage
                         {parnotes}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    --- failed re-building ‘FRASER.Rnw’
>>>>>>> develop
    
    SUMMARY: processing the following file failed:
      ‘FRASER.Rnw’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  8.4Mb
      sub-directories of 1Mb or more:
<<<<<<< HEAD
        R      1.3Mb
        doc    1.5Mb
||||||| cc87037
        R      1.6Mb
        doc    1.6Mb
=======
        R      1.3Mb
        doc    1.6Mb
>>>>>>> develop
        libs   4.8Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported object imported by a ':::' call: ‘S4Vectors:::selectSome’
      See the note in ?`:::` about the use of this operator.
    ```

# funtooNorm

<details>

* Version: 1.18.0
* GitHub: NA
* Source code: https://github.com/cran/funtooNorm
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 149

Run `revdep_details(, "funtooNorm")` for more info

</details>

## In both

*   checking Rd \usage sections ... WARNING
    ```
    Documented arguments not in \usage in documentation object 'show,SampleSet-method':
      ‘...’
    
    Functions with \usage entries need to have the appropriate \alias
    entries, and all their arguments documented.
    The \usage entries must correspond to syntactically valid R code.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

# GenEst

<details>

* Version: 1.4.6
* GitHub: NA
* Source code: https://github.com/cran/GenEst
* Date/Publication: 2021-06-17 07:10:08 UTC
* Number of recursive dependencies: 90

Run `revdep_details(, "GenEst")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘htmlwidgets’
      All declared Imports should be used.
    ```

# geneticae

<details>

* Version: 0.3.0
* GitHub: https://github.com/jangelini/geneticae
* Source code: https://github.com/cran/geneticae
* Date/Publication: 2022-02-09 22:30:13 UTC
* Number of recursive dependencies: 130

Run `revdep_details(, "geneticae")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘calibrate’ ‘graphics’ ‘matrixStats’ ‘prettydoc’ ‘reshape2’
      All declared Imports should be used.
    ```

# GeneTonic

<details>

<<<<<<< HEAD
* Version: 1.6.1
||||||| cc87037
* Version: 1.4.1
=======
* Version: 1.6.4
>>>>>>> develop
* GitHub: https://github.com/federicomarini/GeneTonic
* Source code: https://github.com/cran/GeneTonic
<<<<<<< HEAD
* Date/Publication: 2021-12-02
* Number of recursive dependencies: 203
||||||| cc87037
* Date/Publication: 2021-06-06
* Number of recursive dependencies: 202
=======
* Date/Publication: 2022-03-27
* Number of recursive dependencies: 203
>>>>>>> develop

Run `revdep_details(, "GeneTonic")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘GeneTonic-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: ggs_backbone
    > ### Title: Extract the backbone for the gene-geneset graph
    > ### Aliases: ggs_backbone
    > 
    > ### ** Examples
    > 
    > library("macrophage")
    ...
    +   n_gs = 50,
    +   bb_on = "genesets",
    +   color_graph = TRUE,
    +   color_by_geneset = "z_score"
    + )
    This matrix object is being treated as an unweighted bipartite network of 50 agents and 319 artifacts.
    Error in backbone::backbone.extract(bbobj, alpha = bb_extract_alpha, fwer = bb_extract_fwer) : 
      unused argument (fwer = bb_extract_fwer)
    Calls: ggs_backbone
    Execution halted
    ```

*   checking tests ...
    ```
      Running ‘testthat.R’
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Complete output:
      > library(testthat)
      > library(GeneTonic)
      
      > 
      > test_check("GeneTonic")
      --- Loading packages...
    ...
      ══ Failed tests ════════════════════════════════════════════════════════════════
      ── Error (test-ggs_graph.R:78:3): Backbone functionality up and running ────────
      Error in `backbone::backbone.extract(bbobj, alpha = bb_extract_alpha, fwer = bb_extract_fwer)`: unused argument (fwer = bb_extract_fwer)
      Backtrace:
          ▆
       1. └─GeneTonic::ggs_backbone(...) at test-ggs_graph.R:78:2
      
      [ FAIL 1 | WARN 1 | SKIP 0 | PASS 232 ]
      Error: Test failures
      Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 13.8Mb
      sub-directories of 1Mb or more:
        data      1.2Mb
        doc      10.6Mb
        extdata   1.4Mb
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘pcaExplorer’
    ```

# genomation

<details>

* Version: 1.26.0
* GitHub: https://github.com/BIMSBbioinfo/genomation
* Source code: https://github.com/cran/genomation
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 104

Run `revdep_details(, "genomation")` for more info

</details>

## In both

*   checking whether package ‘genomation’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘Biostrings::pattern’ by ‘grid::pattern’ when loading ‘genomation’
    See ‘/c4/home/henrik/repositories/matrixStats/revdep/checks/genomation/new/genomation.Rcheck/00install.out’ for details.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.5Mb
      sub-directories of 1Mb or more:
        doc       3.5Mb
        extdata   1.2Mb
        libs      1.7Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported object imported by a ':::' call: ‘BiocGenerics:::testPackage’
      See the note in ?`:::` about the use of this operator.
    ```

# geocmeans

<details>

* Version: 0.2.0
* GitHub: https://github.com/JeremyGelb/geocmeans
* Source code: https://github.com/cran/geocmeans
* Date/Publication: 2021-08-23 07:11:35 UTC
<<<<<<< HEAD
* Number of recursive dependencies: 203
||||||| cc87037
* Number of recursive dependencies: 216
=======
* Number of recursive dependencies: 204
>>>>>>> develop

Run `revdep_details(, "geocmeans")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 11.4Mb
      sub-directories of 1Mb or more:
        data   2.3Mb
        doc    1.9Mb
        libs   6.1Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘rgdal’
      All declared Imports should be used.
    ```

# ggdmc

<details>

* Version: 0.2.6.0
* GitHub: https://github.com/yxlin/ggdmc
* Source code: https://github.com/cran/ggdmc
* Date/Publication: 2019-04-29 05:10:03 UTC
* Number of recursive dependencies: 51

Run `revdep_details(, "ggdmc")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.4Mb
      sub-directories of 1Mb or more:
        libs   7.1Mb
    ```

*   checking LazyData ... NOTE
    ```
      'LazyData' is specified without a 'data' directory
    ```

# GGPA

<details>

* Version: 1.6.0
* GitHub: https://github.com/dongjunchung/GGPA
* Source code: https://github.com/cran/GGPA
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 72

Run `revdep_details(, "GGPA")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.6Mb
      sub-directories of 1Mb or more:
        doc    1.6Mb
        libs   3.8Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Packages in Depends field not imported from:
      ‘network’ ‘scales’ ‘sna’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking compiled code ... NOTE
    ```
    File ‘GGPA/libs/GGPA.so’:
      Found ‘_ZSt4cout’, possibly from ‘std::cout’ (C++)
        Object: ‘3_Param.o’
    
    Compiled code should not call entry points which might terminate R nor
    write to stdout/stderr instead of to the console, nor use Fortran I/O
    nor system RNGs.
    
    See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
    ```

# GJRM

<details>

<<<<<<< HEAD
* Version: 0.2-5.1
||||||| cc87037
* Version: 0.2-4
=======
* Version: 0.2-6
>>>>>>> develop
* GitHub: NA
* Source code: https://github.com/cran/GJRM
<<<<<<< HEAD
* Date/Publication: 2021-10-12 09:30:02 UTC
* Number of recursive dependencies: 64
||||||| cc87037
* Date/Publication: 2021-04-12 18:00:11 UTC
* Number of recursive dependencies: 67
=======
* Date/Publication: 2022-04-01 10:30:02 UTC
* Number of recursive dependencies: 64
>>>>>>> develop

Run `revdep_details(, "GJRM")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package which this enhances but not available for checking: ‘sp’
    ```

# glmGamPoi

<details>

* Version: 1.6.0
* GitHub: https://github.com/const-ae/glmGamPoi
* Source code: https://github.com/cran/glmGamPoi
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 168

Run `revdep_details(, "glmGamPoi")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  9.8Mb
      sub-directories of 1Mb or more:
        doc    1.1Mb
        libs   8.3Mb
    ```

# graper

<details>

* Version: 1.10.0
* GitHub: NA
* Source code: https://github.com/cran/graper
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 70

Run `revdep_details(, "graper")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.5Mb
      sub-directories of 1Mb or more:
        doc    1.8Mb
        libs   5.6Mb
    ```

# GUIDEseq

<details>

* Version: 1.24.0
* GitHub: NA
* Source code: https://github.com/cran/GUIDEseq
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 160

Run `revdep_details(, "GUIDEseq")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 11.0Mb
      sub-directories of 1Mb or more:
        extdata  10.5Mb
    ```

*   checking Rd files ... NOTE
    ```
    prepare_Rd: annotateOffTargets.Rd:33-35: Dropping empty section \details
    prepare_Rd: annotateOffTargets.Rd:43-45: Dropping empty section \references
    prepare_Rd: createBarcodeFasta.Rd:42-43: Dropping empty section \value
    prepare_Rd: createBarcodeFasta.Rd:44-46: Dropping empty section \references
    prepare_Rd: getUsedBarcodes.Rd:39-41: Dropping empty section \references
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error(s) in re-building vignettes:
    --- re-building ‘GUIDEseq.Rnw’ using Sweave
    Loading required package: GenomicRanges
    Loading required package: stats4
    Loading required package: BiocGenerics
    
    Attaching package: ‘BiocGenerics’
    
    The following objects are masked from ‘package:stats’:
    
    ...
    l.189 \RequirePackage
                         {parnotes}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    --- failed re-building 'GUIDEseq.Rnw'
    
    SUMMARY: processing the following file failed:
      'GUIDEseq.Rnw'
    
    Error: Vignette re-building failed.
    Execution halted
    ```

# Gviz

<details>

<<<<<<< HEAD
* Version: 1.38.1
||||||| cc87037
* Version: 1.36.2
=======
* Version: 1.38.4
>>>>>>> develop
* GitHub: https://github.com/ivanek/Gviz
* Source code: https://github.com/cran/Gviz
<<<<<<< HEAD
* Date/Publication: 2022-01-06
* Number of recursive dependencies: 153
||||||| cc87037
* Date/Publication: 2021-07-04
* Number of recursive dependencies: 150
=======
* Date/Publication: 2022-04-10
* Number of recursive dependencies: 153
>>>>>>> develop

Run `revdep_details(, "Gviz")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 10.3Mb
      sub-directories of 1Mb or more:
        R         1.7Mb
        doc       4.7Mb
        extdata   2.5Mb
    ```

# GWENA

<details>

* Version: 1.4.0
* GitHub: https://github.com/Kumquatum/GWENA
* Source code: https://github.com/cran/GWENA
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 147

Run `revdep_details(, "GWENA")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.9Mb
      sub-directories of 1Mb or more:
        data   2.3Mb
        doc    5.3Mb
    ```

# hipathia

<details>

* Version: 2.10.0
* GitHub: NA
* Source code: https://github.com/cran/hipathia
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 127

Run `revdep_details(, "hipathia")` for more info

</details>

## In both

<<<<<<< HEAD
||||||| cc87037
*   checking examples ... ERROR
    ```
    Running examples in ‘hipathia-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: create_report
    > ### Title: Create visualization HTML
    > ### Aliases: create_report
    > 
    > ### ** Examples
    > 
    > data(comp)
    ...
    > pathways <- load_pathways(species = "hsa", pathways_list = c("hsa03320",
    + "hsa04012"))
    snapshotDate(): 2021-05-18
    Error in h(simpleError(msg, call)) : 
      error in evaluating the argument 'x' in selecting a method for function 'query': Corrupt Cache: index file
      See AnnotationHub's TroubleshootingTheCache vignette section on corrupt cache
      cache: /scratch/henrik/revdepcheck.extras/matrixStats/revdep/cache/R/AnnotationHub
      filename: annotationhub.index.rds
    Calls: load_pathways ... hub -> query -> AnnotationHub -> .Hub -> .db_create_index
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 50 lines of output:
        cache: /scratch/henrik/revdepcheck.extras/matrixStats/revdep/cache/R/AnnotationHub
        filename: annotationhub.index.rds
      Backtrace:
           █
        1. ├─hipathia::load_pathways("hsa", pathways_list = pathways_list) test-path-summary.R:9:0
        2. │ └─hipathia:::load_mgi(species)
        3. │   └─hipathia:::hub()
    ...
        5. │     └─AnnotationHub::AnnotationHub()
        6. │       └─AnnotationHub::.Hub(...)
        7. │         └─AnnotationHub:::.db_create_index(hub)
        8. │           └─base::stop(...)
        9. └─base::.handleSimpleError(...)
       10.   └─base:::h(simpleError(msg, call))
      
      [ FAIL 10 | WARN 0 | SKIP 0 | PASS 46 ]
      Error: Test failures
      Execution halted
    ```

=======
*   checking examples ... ERROR
    ```
    Running examples in ‘hipathia-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: create_report
    > ### Title: Create visualization HTML
    > ### Aliases: create_report
    > 
    > ### ** Examples
    > 
    > data(comp)
    > pathways <- load_pathways(species = "hsa", pathways_list = c("hsa03320",
    + "hsa04012"))
    Error in h(simpleError(msg, call)) : 
      error in evaluating the argument 'x' in selecting a method for function 'query': Corrupt Cache: sqlite file
      See AnnotationHub's TroubleshootingTheCache vignette section on corrupt cache
      cache: /c4/home/henrik/.cache/R/AnnotationHub
      filename: annotationhub.sqlite3
    Calls: load_pathways ... AnnotationHub -> .Hub -> .create_cache -> .updateHubDB
    Execution halted
    ```

*   checking tests ...
    ```
      Running ‘testthat.R’
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 50 lines of output:
           ▆
        1. ├─hipathia::load_pathways("hsa", pathways_list = pathways_list) at test-path-summary.R:9:0
        2. │ └─hipathia:::load_mgi(species)
        3. │   └─hipathia:::hub()
        4. │     ├─AnnotationHub::query(AnnotationHub(), "hpAnnot")
        5. │     └─AnnotationHub::AnnotationHub()
    ...
        6. │       └─AnnotationHub::.Hub(...)
        7. │         └─AnnotationHub:::.create_cache(...)
        8. │           └─AnnotationHub:::.updateHubDB(hub_bfc, .class, url, proxy, localHub)
        9. │             └─base::stop(...)
       10. └─base::.handleSimpleError(...)
       11.   └─base h(simpleError(msg, call))
      
      [ FAIL 10 | WARN 0 | SKIP 0 | PASS 46 ]
      Error: Test failures
      Execution halted
    ```

>>>>>>> develop
*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
    --- re-building ‘hipathia-vignette.Rmd’ using rmarkdown
    Loading required package: igraph
    
    Attaching package: 'igraph'
    
    The following objects are masked from 'package:stats':
    
        decompose, spectrum
    
    ...
<<<<<<< HEAD
    Quitting from lines 774-775 (hipathia-vignette.Rmd) 
    Error: processing vignette 'hipathia-vignette.Rmd' failed with diagnostics:
    Failed to create server
||||||| cc87037
      See AnnotationHub's TroubleshootingTheCache vignette section on corrupt cache
      cache: /scratch/henrik/revdepcheck.extras/matrixStats/revdep/cache/R/AnnotationHub
      filename: annotationhub.index.rds
=======
      See AnnotationHub's TroubleshootingTheCache vignette section on corrupt cache
      cache: /c4/home/henrik/.cache/R/AnnotationHub
      filename: annotationhub.sqlite3
>>>>>>> develop
    --- failed re-building ‘hipathia-vignette.Rmd’
    
    SUMMARY: processing the following file failed:
      ‘hipathia-vignette.Rmd’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 11.2Mb
      sub-directories of 1Mb or more:
        data      4.9Mb
        doc       1.7Mb
        extdata   4.3Mb
    ```

# InfiniumPurify

<details>

* Version: 1.3.1
* GitHub: NA
* Source code: https://github.com/cran/InfiniumPurify
* Date/Publication: 2017-01-14 12:12:25
* Number of recursive dependencies: 1

Run `revdep_details(, "InfiniumPurify")` for more info

</details>

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘minfi’
    ```

# JMbayes2

<details>

* Version: 0.2-0
* GitHub: https://github.com/drizopoulos/JMbayes2
* Source code: https://github.com/cran/JMbayes2
* Date/Publication: 2022-02-10 10:00:06 UTC
* Number of recursive dependencies: 79

Run `revdep_details(, "JMbayes2")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  9.6Mb
      sub-directories of 1Mb or more:
        libs   9.0Mb
    ```

# kgschart

<details>

* Version: 1.3.5
* GitHub: https://github.com/kota7/kgschart
* Source code: https://github.com/cran/kgschart
* Date/Publication: 2017-07-02 22:16:15 UTC
* Number of recursive dependencies: 74

Run `revdep_details(, "kgschart")` for more info

</details>

## In both

*   checking LazyData ... NOTE
    ```
      'LazyData' is specified without a 'data' directory
    ```

# kissDE

<details>

* Version: 1.14.0
* GitHub: NA
* Source code: https://github.com/cran/kissDE
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 147

Run `revdep_details(, "kissDE")` for more info

</details>

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    Error(s) in re-building vignettes:
    --- re-building ‘kissDE.rnw’ using Sweave
    Pre-processing the data...
    Trying to fit models on data...
    This can be a time-consuming step, so do not hesitate to have 
        a look at the very well-written vignette !
    Computing pvalues...
    Computing size of the effect and last cutoffs...
    Pre-processing the data...
    Trying to fit models on data...
    ...
    l.189 \RequirePackage
                         {parnotes}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    --- failed re-building ‘kissDE.rnw’
    
    SUMMARY: processing the following file failed:
      ‘kissDE.rnw’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

# landmap

<details>

* Version: 0.0.13
* GitHub: https://github.com/envirometrix/landmap
* Source code: https://github.com/cran/landmap
* Date/Publication: 2021-10-14 12:00:02 UTC
* Number of recursive dependencies: 132

Run `revdep_details(, "landmap")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘ParamHelpers’ ‘geoR’ ‘kernlab’ ‘maptools’ ‘nnet’ ‘rpart’ ‘spdep’
      ‘xgboost’
      All declared Imports should be used.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 23 marked UTF-8 strings
    ```

# ldsep

<details>

* Version: 2.1.4
* GitHub: https://github.com/dcgerard/ldsep
* Source code: https://github.com/cran/ldsep
* Date/Publication: 2022-02-11 15:00:02 UTC
* Number of recursive dependencies: 169

Run `revdep_details(, "ldsep")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  8.0Mb
      sub-directories of 1Mb or more:
        libs   7.3Mb
    ```

# LFDREmpiricalBayes

<details>

* Version: 1.0
* GitHub: NA
* Source code: https://github.com/cran/LFDREmpiricalBayes
* Date/Publication: 2017-09-27 09:08:46 UTC
* Number of recursive dependencies: 33

Run `revdep_details(, "LFDREmpiricalBayes")` for more info

</details>

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    Error(s) in re-building vignettes:
      ...
    --- re-building ‘LFDREmpiricalBayes.Rnw’ using Sweave
    Error: processing vignette 'LFDREmpiricalBayes.Rnw' failed with diagnostics:
    Running 'texi2dvi' on 'LFDREmpiricalBayes.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `changepage.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    ...
    l.10 \usepackage
                    {url}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    --- failed re-building ‘LFDREmpiricalBayes.Rnw’
    
    SUMMARY: processing the following file failed:
      ‘LFDREmpiricalBayes.Rnw’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

# liger

<details>

* Version: 2.0.1
* GitHub: https://github.com/JEFworks/liger
* Source code: https://github.com/cran/liger
* Date/Publication: 2021-01-25 05:50:09 UTC
* Number of recursive dependencies: 50

Run `revdep_details(, "liger")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.3Mb
      sub-directories of 1Mb or more:
        data   3.3Mb
        doc    2.1Mb
        libs   1.9Mb
    ```

# LSAmitR

<details>

* Version: 1.0-2
* GitHub: NA
* Source code: https://github.com/cran/LSAmitR
* Date/Publication: 2018-06-08 13:23:05 UTC
* Number of recursive dependencies: 160

Run `revdep_details(, "LSAmitR")` for more info

</details>

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 55 marked UTF-8 strings
    ```

# lspartition

<details>

* Version: 0.4
* GitHub: NA
* Source code: https://github.com/cran/lspartition
* Date/Publication: 2019-08-08 22:40:06 UTC
* Number of recursive dependencies: 38

Run `revdep_details(, "lspartition")` for more info

</details>

## In both

*   checking LazyData ... NOTE
    ```
      'LazyData' is specified without a 'data' directory
    ```

# Luminescence

<details>

* Version: 0.9.19
* GitHub: https://github.com/R-Lum/Luminescence
* Source code: https://github.com/cran/Luminescence
* Date/Publication: 2022-03-10 22:00:02 UTC
* Number of recursive dependencies: 136

Run `revdep_details(, "Luminescence")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.8Mb
      sub-directories of 1Mb or more:
        R      1.6Mb
        help   1.2Mb
        libs   2.9Mb
    ```

# M3Drop

<details>

* Version: 1.20.0
* GitHub: https://github.com/tallulandrews/M3Drop
* Source code: https://github.com/cran/M3Drop
<<<<<<< HEAD
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 221
||||||| cc87037
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 206
=======
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 222
>>>>>>> develop

Run `revdep_details(, "M3Drop")` for more info

</details>

## In both

*   checking dependencies in R code ... WARNING
    ```
    Package in Depends field not imported from: ‘numDeriv’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    Missing or unexported object: ‘scater::exprs’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 12.9Mb
      sub-directories of 1Mb or more:
        doc  12.6Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    M3DropConvertData: no visible global function definition for ‘assays’
    NBumiConvertData: no visible global function definition for ‘assays’
    bg__filter_cells: no visible binding for global variable ‘x’
    bg__get_extreme_residuals: no visible binding for global variable ‘x’
    Undefined global functions or variables:
      assays x
    ```

*   checking Rd files ... NOTE
    ```
    prepare_Rd: NBumi_FSOther.Rd:8-10: Dropping empty section \usage
    prepare_Rd: NBumi_TradDE.Rd:7-8: Dropping empty section \usage
    ```

# maEndToEnd

<details>

* Version: 2.14.0
* GitHub: NA
* Source code: https://github.com/cran/maEndToEnd
* Date/Publication: 2021-10-29
* Number of recursive dependencies: 231

Run `revdep_details(, "maEndToEnd")` for more info

</details>

## In both

*   checking DESCRIPTION meta-information ... WARNING
    ```
    Invalid license file pointers: LICENSE
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
    --- re-building ‘MA-Workflow.Rmd’ using rmarkdown
    trying URL 'https://www.ebi.ac.uk/arrayexpress/files/A-AFFY-141/A-AFFY-141.adf.txt'
    Content type 'text/plain' length 94175098 bytes (89.8 MB)
    ==================================================
    downloaded 89.8 MB
    
    trying URL 'https://www.ebi.ac.uk/arrayexpress/files/E-MTAB-2967/E-MTAB-2967.sdrf.txt'
    Content type 'text/plain' length 22516 bytes (21 KB)
    ==================================================
    ...
    Quitting from lines 1964-1966 (MA-Workflow.Rmd) 
    Error: processing vignette 'MA-Workflow.Rmd' failed with diagnostics:
    there is no package called 'ggnewscale'
    --- failed re-building ‘MA-Workflow.Rmd’
    
    SUMMARY: processing the following file failed:
      ‘MA-Workflow.Rmd’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      'Biobase', 'oligoClasses', 'ArrayExpress', 'pd.hugene.1.0.st.v1',
      'hugene10sttranscriptcluster.db', 'oligo', 'arrayQualityMetrics',
      'limma', 'topGO', 'ReactomePA', 'clusterProfiler', 'gplots',
      'ggplot2', 'geneplotter', 'pheatmap', 'RColorBrewer', 'dplyr',
      'tidyr', 'stringr', 'matrixStats', 'genefilter', 'openxlsx',
      'Rgraphviz', 'enrichplot'
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.1Mb
      sub-directories of 1Mb or more:
        doc   6.1Mb
    ```

*   checking top-level files ... NOTE
    ```
    File
      LICENCE
    is not mentioned in the DESCRIPTION file.
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    '::' or ':::' import not declared from: ‘AnnotationDbi’
    ```

# MatrixGenerics

<details>

* Version: 1.6.0
* GitHub: https://github.com/Bioconductor/MatrixGenerics
* Source code: https://github.com/cran/MatrixGenerics
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 50

Run `revdep_details(, "MatrixGenerics")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      ‘methods:::fromNextMethod’ ‘methods:::getGenericFromCall’
      See the note in ?`:::` about the use of this operator.
    There are ::: calls to the package's namespace in its code. A package
      almost never needs to use ::: for its own objects:
      ‘.load_next_suggested_package_to_search’
    ```

# mcmcsae

<details>

* Version: 0.7.0
* GitHub: NA
* Source code: https://github.com/cran/mcmcsae
* Date/Publication: 2021-10-10 10:40:01 UTC
* Number of recursive dependencies: 113

Run `revdep_details(, "mcmcsae")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 16.7Mb
      sub-directories of 1Mb or more:
        libs  15.4Mb
    ```

# MEAL

<details>

* Version: 1.24.0
* GitHub: NA
* Source code: https://github.com/cran/MEAL
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 226

Run `revdep_details(, "MEAL")` for more info

</details>

## In both

*   checking dependencies in R code ... WARNING
    ```
    '::' or ':::' import not declared from: 'sva'
    ```

*   checking Rd \usage sections ... WARNING
    ```
    Undocumented arguments in documentation object 'getProbeResults'
      ‘robust’
    
    Functions with \usage entries need to have the appropriate \alias
    entries, and all their arguments documented.
    The \usage entries must correspond to syntactically valid R code.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking examples ... WARNING
    ```
    Found the following significant warnings:
    
      Warning: 'runRegionAnalysis' is deprecated.
    Deprecated functions may be defunct as soon as of the next release of
    R.
    See ?Deprecated.
    ```

*   checking top-level files ... NOTE
    ```
    File
      LICENSE
    is not mentioned in the DESCRIPTION file.
    ```

*   checking R code for possible problems ... NOTE
    ```
    plotRegion: no visible global function definition for
      ‘subsetByOverlaps’
    plotRegion: no visible global function definition for ‘mcols<-’
    runRDA: no visible global function definition for ‘rowData’
    runSVA: no visible global function definition for ‘resid’
    runSVA: no visible global function definition for ‘quantile’
    Undefined global functions or variables:
      mcols<- quantile resid rowData subsetByOverlaps
    Consider adding
      importFrom("stats", "quantile", "resid")
    to your NAMESPACE file.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘DMRcate’
    ```

# memes

<details>

* Version: 1.2.4
* GitHub: https://github.com/snystrom/memes
* Source code: https://github.com/cran/memes
* Date/Publication: 2021-12-26
* Number of recursive dependencies: 155

Run `revdep_details(, "memes")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘memes-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: ame_compare_heatmap_methods
    > ### Title: Compare AME heatmap methods
    > ### Aliases: ame_compare_heatmap_methods
    > 
    > ### ** Examples
    > 
    > data("example_ame", package = "memes")
    > ame_compare_heatmap_methods(example_ame$Decreasing)
    Error in is_call(uq_expr, c("after_stat", "after_scale")) : 
      argument "uq_expr" is missing, with no default
    Calls: ame_compare_heatmap_methods ... Map -> mapply -> <Anonymous> -> strip_stage -> is_call
    Execution halted
    ```

# metagene

<details>

* Version: 2.26.0
* GitHub: https://github.com/CharlesJB/metagene
* Source code: https://github.com/cran/metagene
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 128

Run `revdep_details(, "metagene")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 10.1Mb
      sub-directories of 1Mb or more:
        doc       3.1Mb
        extdata   5.3Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Package listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘similaRpeak’
    A package should be listed in only one of these fields.
    ```

# metamicrobiomeR

<details>

* Version: 1.2
* GitHub: https://github.com/nhanhocu/metamicrobiomeR
* Source code: https://github.com/cran/metamicrobiomeR
* Date/Publication: 2020-11-09 11:20:05 UTC
* Number of recursive dependencies: 149

Run `revdep_details(, "metamicrobiomeR")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘lmerTest’
      All declared Imports should be used.
    ```

# MetaNeighbor

<details>

* Version: 1.14.0
* GitHub: NA
* Source code: https://github.com/cran/MetaNeighbor
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 94

Run `revdep_details(, "MetaNeighbor")` for more info

</details>

## In both

*   checking LazyData ... WARNING
    ```
      LazyData DB of 7.1 MB without LazyDataCompression set
      See §1.1.6 of 'Writing R Extensions'
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.7Mb
      sub-directories of 1Mb or more:
        data   7.1Mb
    ```

# MethReg

<details>

* Version: 1.4.0
* GitHub: https://github.com/TransBioInfoLab/MethReg
* Source code: https://github.com/cran/MethReg
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 230

Run `revdep_details(, "MethReg")` for more info

</details>

## In both

*   checking tests ...
    ```
      Running ‘testthat.R’
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 50 lines of output:
<<<<<<< HEAD
      |====================================================|100%                      Completed after 0 s 
      
      |                                                    |  0%                      
      |====================================================|100% ~0 s remaining       
      |====================================================|100%                      Completed after 1 s 
      
    ...
||||||| cc87037
      Error: values must be length 1,
       but FUN(X[[2]]) result is length 0
      Backtrace:
          █
       1. └─MethReg::get_promoter_avg(dnam = dnam, genome = "hg38", arrayType = "450k") test-get_promoter_avg.R:37:4
=======
      |====================================================|100%                      Completed after 0 s 
      
      |                                                    |  0%                      
      |====================================================|100% ~0 s remaining       
      |====================================================|100%                      Completed after 2 s 
      
    ...
>>>>>>> develop
       2.   ├─base::cbind(rownames(se), rbind(unique.promoter.genes, non.unique.promoter.genes))
       3.   └─base::rbind(unique.promoter.genes, non.unique.promoter.genes)
       4.     └─S4Vectors::rbind(deparse.level, ...)
       5.       ├─S4Vectors::bindROWS(objects[[1L]], objects = objects[-1L])
       6.       └─S4Vectors::bindROWS(objects[[1L]], objects = objects[-1L])
       7.         └─base::vapply(all_objects, nrow, integer(1L), USE.NAMES = FALSE)
      
      [ FAIL 2 | WARN 4 | SKIP 4 | PASS 158 ]
      Error: Test failures
      Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.1Mb
      sub-directories of 1Mb or more:
        R     1.6Mb
        doc   2.5Mb
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘TCGAbiolinks’
    ```

# methrix

<details>

* Version: 1.8.0
* GitHub: https://github.com/CompEpigen/methrix
* Source code: https://github.com/cran/methrix
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 160

Run `revdep_details(, "methrix")` for more info

</details>

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
    --- re-building ‘methrix.Rmd’ using rmarkdown
    Loading required package: data.table
    Loading required package: SummarizedExperiment
    Loading required package: MatrixGenerics
    Loading required package: matrixStats
    
    Attaching package: 'MatrixGenerics'
    
    The following objects are masked from 'package:matrixStats':
    ...
    Quitting from lines 108-110 (methrix.Rmd) 
    Error: processing vignette 'methrix.Rmd' failed with diagnostics:
    
    --- failed re-building ‘methrix.Rmd’
    
    SUMMARY: processing the following file failed:
      ‘methrix.Rmd’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

# MethylAid

<details>

* Version: 1.28.0
* GitHub: NA
* Source code: https://github.com/cran/MethylAid
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 174

Run `revdep_details(, "MethylAid")` for more info

</details>

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
    --- re-building ‘MethylAid.Rnw’ using knitr
    Setting options('download.file.method.GEOquery'='auto')
    Setting options('GEOquery.inmemory.gpl'=FALSE)
    Loading required package: minfi
    Loading required package: BiocGenerics
    
    Attaching package: 'BiocGenerics'
    
    The following objects are masked from 'package:stats':
    ...
    l.189 \RequirePackage
                         {parnotes}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    --- failed re-building ‘MethylAid.Rnw’
    
    SUMMARY: processing the following file failed:
      ‘MethylAid.Rnw’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

# methylationArrayAnalysis

<details>

* Version: 1.18.0
* GitHub: NA
* Source code: https://github.com/cran/methylationArrayAnalysis
* Date/Publication: 2021-10-29
* Number of recursive dependencies: 217

Run `revdep_details(, "methylationArrayAnalysis")` for more info

</details>

## In both

*   checking whether package ‘methylationArrayAnalysis’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import 'rmarkdown::pdf_document' by 'BiocStyle::pdf_document' when loading 'methylationArrayAnalysis'
      Warning: replacing previous import 'rmarkdown::html_document' by 'BiocStyle::html_document' when loading 'methylationArrayAnalysis'
      Warning: replacing previous import 'rmarkdown::md_document' by 'BiocStyle::md_document' when loading 'methylationArrayAnalysis'
    See ‘/c4/home/henrik/repositories/matrixStats/revdep/checks/methylationArrayAnalysis/new/methylationArrayAnalysis.Rcheck/00install.out’ for details.
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      'knitr', 'rmarkdown', 'BiocStyle', 'limma', 'minfi',
      'IlluminaHumanMethylation450kanno.ilmn12.hg19',
      'IlluminaHumanMethylation450kmanifest', 'RColorBrewer', 'missMethyl',
      'matrixStats', 'minfiData', 'Gviz', 'DMRcate', 'stringr',
      'FlowSorted.Blood.450k'
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 322.7Mb
      sub-directories of 1Mb or more:
        doc        2.6Mb
        extdata  320.2Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Title field: should not end in a period.
    ```

# methylumi

<details>

* Version: 2.40.1
* GitHub: https://github.com/seandavi/methylumi
* Source code: https://github.com/cran/methylumi
* Date/Publication: 2021-10-31
* Number of recursive dependencies: 166

Run `revdep_details(, "methylumi")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      'Biobase', 'scales', 'reshape2', 'ggplot2', 'matrixStats',
      'FDb.InfiniumMethylation.hg19', 'minfi'
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 11.8Mb
      sub-directories of 1Mb or more:
        R         1.3Mb
        data      2.7Mb
        doc       1.9Mb
        extdata   5.8Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Packages listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘Biobase’ ‘minfi’ ‘lattice’ ‘matrixStats’
    A package should be listed in only one of these fields.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls to packages already attached by Depends:
      ‘FDb.InfiniumMethylation.hg19’ ‘ggplot2’ ‘matrixStats’ ‘minfi’
      ‘reshape2’ ‘scales’
      Please remove these calls from your code.
    'library' or 'require' calls in package code:
      ‘Biostrings’ ‘MASS’ ‘lumi’ ‘parallel’ ‘rtracklayer’
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    Namespace in Imports field not imported from: ‘graphics’
      All declared Imports should be used.
    ...
    Packages in Depends field not imported from:
      ‘FDb.InfiniumMethylation.hg19’ ‘ggplot2’ ‘matrixStats’ ‘methods’
      ‘reshape2’ ‘scales’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ':::' call which should be '::': ‘lumi:::produceMethylationGEOSubmissionFile’
      See the note in ?`:::` about the use of this operator.
    Unexported objects imported by ':::' calls:
      ‘Biobase:::unsafeSetSlot’ ‘genefilter:::.findCentralMap’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    getBarcodes: warning in list.files(path = path, patt = "idat"): partial
      argument match of 'patt' to 'pattern'
    coerce,RangedSummarizedExperiment-GenomicMethylSet: warning in
      assays(from, withDim = F): partial argument match of 'withDim' to
      'withDimnames'
    .getFinalReportBlock: no visible global function definition for
      ‘read.table’
    .mclapply: no visible global function definition for ‘mclapply’
    .parallel : <anonymous>: no visible global function definition for
      ‘ecdf’
    ...
      subsetCommonProbes text theme_bw title value variable weighted.mean
    Consider adding
      importFrom("grDevices", "colorRampPalette")
      importFrom("graphics", "abline", "axis", "box", "lines", "par",
                 "points", "rect", "text", "title")
      importFrom("stats", "as.dist", "dnorm", "ecdf", "hclust", "offset",
                 "p.adjust", "pnorm", "weighted.mean")
      importFrom("utils", "capture.output", "data", "packageDescription",
                 "packageVersion", "read.csv", "read.delim", "read.table")
    to your NAMESPACE file.
    ```

*   checking Rd files ... NOTE
    ```
    prepare_Rd: estimateM.Rd:34-36: Dropping empty section \seealso
    prepare_Rd: estimateM.Rd:37-39: Dropping empty section \examples
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    'library' or 'require' call not declared from: ‘TxDb.Hsapiens.UCSC.hg19.knownGene’
    ```

# MFHD

<details>

* Version: 0.0.1
* GitHub: NA
* Source code: https://github.com/cran/MFHD
* Date/Publication: 2013-10-23 23:50:20
* Number of recursive dependencies: 78

Run `revdep_details(, "MFHD")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    MFHD: no visible global function definition for ‘complete.cases’
    MFHD: no visible global function definition for ‘quantile’
    PIT: no visible global function definition for ‘xy.coords’
    PIT: no visible global function definition for ‘identify’
    PIT: no visible global function definition for ‘points’
    mfhd_compute.bagplot : cut.z.pg: no visible global function definition
      for ‘segments’
    mfhd_compute.bagplot : cut.z.pg: no visible global function definition
      for ‘points’
    mfhd_compute.bagplot : find.cut.z.pg: no visible global function
    ...
    plot.bagplot: no visible global function definition for ‘polygon’
    Undefined global functions or variables:
      boxplot chull complete.cases identify lines points polygon prcomp
      quantile sd segments text xy.coords
    Consider adding
      importFrom("grDevices", "chull", "xy.coords")
      importFrom("graphics", "boxplot", "identify", "lines", "points",
                 "polygon", "segments", "text")
      importFrom("stats", "complete.cases", "prcomp", "quantile", "sd")
    to your NAMESPACE file.
    ```

# mgcViz

<details>

* Version: 0.1.9
* GitHub: https://github.com/mfasiolo/mgcViz
* Source code: https://github.com/cran/mgcViz
* Date/Publication: 2021-10-05 07:10:12 UTC
* Number of recursive dependencies: 105

Run `revdep_details(, "mgcViz")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘webshot2’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.2Mb
      sub-directories of 1Mb or more:
        doc   4.6Mb
    ```

# microbiomeExplorer

<details>

* Version: 1.4.0
* GitHub: NA
* Source code: https://github.com/cran/microbiomeExplorer
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 197

Run `revdep_details(, "microbiomeExplorer")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 12.3Mb
      sub-directories of 1Mb or more:
        doc       4.9Mb
        extdata   4.1Mb
        shiny     2.8Mb
    ```

# MicrobiomeStat

<details>

* Version: 1.1
* GitHub: NA
* Source code: https://github.com/cran/MicrobiomeStat
* Date/Publication: 2022-01-24 00:02:41 UTC
* Number of recursive dependencies: 103

Run `revdep_details(, "MicrobiomeStat")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘MASS’ ‘Matrix’ ‘utils’
      All declared Imports should be used.
    ```

# migest

<details>

* Version: 2.0.2
* GitHub: https://github.com/guyabel/migest
* Source code: https://github.com/cran/migest
* Date/Publication: 2021-12-02 05:30:02 UTC
* Number of recursive dependencies: 111

Run `revdep_details(, "migest")` for more info

</details>

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 4 marked UTF-8 strings
    ```

# miloR

<details>

* Version: 1.2.0
* GitHub: https://github.com/MarioniLab/miloR
* Source code: https://github.com/cran/miloR
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 194

Run `revdep_details(, "miloR")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.7Mb
      sub-directories of 1Mb or more:
        data   1.8Mb
        doc    3.4Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported object imported by a ':::' call: ‘S4Vectors:::disableValidity’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking Rd files ... NOTE
    ```
    prepare_Rd: findNhoodGroupMarkers.Rd:79-81: Dropping empty section \examples
    ```

# MiMIR

<details>

* Version: 1.2
* GitHub: NA
* Source code: https://github.com/cran/MiMIR
* Date/Publication: 2022-04-15 10:32:32 UTC
* Number of recursive dependencies: 184

Run `revdep_details(, "MiMIR")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘glmnet’ ‘pkgload’ ‘plyr’ ‘shinyFiles’ ‘shinyWidgets’
      ‘shinycssloaders’ ‘shinydashboard’ ‘shinyjs’
      All declared Imports should be used.
    ```

# MinimumDistance

<details>

* Version: 1.38.0
* GitHub: NA
* Source code: https://github.com/cran/MinimumDistance
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 78

Run `revdep_details(, "MinimumDistance")` for more info

</details>

## In both

*   checking whether package ‘MinimumDistance’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘ff::pattern’ by ‘grid::pattern’ when loading ‘MinimumDistance’
    See ‘/c4/home/henrik/repositories/matrixStats/revdep/checks/MinimumDistance/new/MinimumDistance.Rcheck/00install.out’ for details.
    ```

*   checking Rd files ... WARNING
    ```
    checkRd: (5) MinimumDistance.Rd:0-7: Must have a \description
    ```

*   checking for code/documentation mismatches ... WARNING
    ```
    Functions or methods with usage in documentation object 'coerce' but not in code:
      ‘as’
    ```

*   checking Rd \usage sections ... WARNING
    ```
    Objects in \usage without \alias in documentation object 'coerce':
      ‘as’
    
    Functions with \usage entries need to have the appropriate \alias
    entries, and all their arguments documented.
    The \usage entries must correspond to syntactically valid R code.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking contents of ‘data’ directory ... WARNING
    ```
    Output for data("trioSetListExample", package = "MinimumDistance"):
      Warning message:
      replacing previous import ‘ff::pattern’ by ‘grid::pattern’ when loading ‘MinimumDistance’ 
    ```

*   checking package dependencies ... NOTE
    ```
    Package which this enhances but not available for checking: ‘doSNOW’
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘MatrixGenerics’
      All declared Imports should be used.
    Unexported object imported by a ':::' call: ‘VanillaICE:::plotIdiogram’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    callDenovoSegments: no visible global function definition for
      ‘read.bsfiles’
    combineRanges: no visible global function definition for ‘RangedData’
    getRefGene: no visible global function definition for ‘RangedData’
    overlapsCentromere: no visible global function definition for
      ‘RangedData’
    pruneMD: no visible global function definition for ‘RangedDataCBS’
    pruneTrioSet: no visible global function definition for
      ‘RangedDataList’
    read.bsfiles2: no visible binding for global variable ‘read.bsfiles’
    GenomeAnnotatedDataFrameFrom,character: no visible global function
      definition for ‘read.bsfiles’
    calculateMindist,list: no visible binding for global variable ‘elt’
    Undefined global functions or variables:
      RangedData RangedDataCBS RangedDataList elt read.bsfiles
    ```

# mixOmics

<details>

* Version: 6.18.1
* GitHub: https://github.com/mixOmicsTeam/mixOmics
* Source code: https://github.com/cran/mixOmics
* Date/Publication: 2021-11-18
* Number of recursive dependencies: 92

Run `revdep_details(, "mixOmics")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 12.2Mb
      sub-directories of 1Mb or more:
        R      1.3Mb
        data   3.3Mb
        doc    6.1Mb
    ```

# mnem

<details>

* Version: 1.10.0
* GitHub: https://github.com/cbg-ethz/mnem
* Source code: https://github.com/cran/mnem
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 159

Run `revdep_details(, "mnem")` for more info

</details>

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
      ...
    --- re-building ‘mnem.Rmd’ using rmarkdown
    Error: processing vignette 'mnem.Rmd' failed with diagnostics:
    there is no package called ‘BiocStyle’
    --- failed re-building ‘mnem.Rmd’
    
    SUMMARY: processing the following file failed:
      ‘mnem.Rmd’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
<<<<<<< HEAD
      installed size is  8.5Mb
||||||| cc87037
      installed size is  8.8Mb
=======
      installed size is  8.6Mb
>>>>>>> develop
      sub-directories of 1Mb or more:
        data   2.3Mb
        doc    3.8Mb
        libs   2.2Mb
    ```

# moanin

<details>

* Version: 1.2.0
* GitHub: NA
* Source code: https://github.com/cran/moanin
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 121

Run `revdep_details(, "moanin")` for more info

</details>

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
    --- re-building ‘documentation.Rmd’ using knitr
    Loading required package: SummarizedExperiment
    Loading required package: MatrixGenerics
    Loading required package: matrixStats
    
    Attaching package: 'MatrixGenerics'
    
    The following objects are masked from 'package:matrixStats':
    
    ...
    Quitting from lines 45-49 (documentation.Rmd) 
    Error: processing vignette 'documentation.Rmd' failed with diagnostics:
    there is no package called 'codetools'
    --- failed re-building ‘documentation.Rmd’
    
    SUMMARY: processing the following file failed:
      ‘documentation.Rmd’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘edge’
    Unknown package ‘KEGGprofile’ in Rd xrefs
    ```

# monocle

<details>

* Version: 2.22.0
* GitHub: NA
* Source code: https://github.com/cran/monocle
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 244

Run `revdep_details(, "monocle")` for more info

</details>

## In both

*   checking dependencies in R code ... WARNING
    ```
    Namespaces in Imports field not imported from:
      ‘Rcpp’ ‘biocViews’
      All declared Imports should be used.
    Missing or unexported object: ‘scater::newSCESet’
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
      ...
    --- re-building ‘monocle-vignette.Rnw’ using knitr
    Error: processing vignette 'monocle-vignette.Rnw' failed with diagnostics:
    Running 'texi2dvi' on 'monocle-vignette.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `sectsty.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    ...
    l.36 \sectionfont
                     {\sffamily\bfseries\color{RoyalBlue}\sectionrule{0pt}{0pt}{...
    !  ==> Fatal error occurred, no output PDF file produced!
    --- failed re-building ‘monocle-vignette.Rnw’
    
    SUMMARY: processing the following file failed:
      ‘monocle-vignette.Rnw’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

*   checking R code for possible problems ... NOTE
    ```
    assign_cell_lineage: no visible global function definition for ‘nei’
    buildBranchCellDataSet: no visible global function definition for ‘nei’
    clusterCells: no visible global function definition for ‘quantile’
    count_leaf_descendents: no visible global function definition for ‘nei’
    cth_classifier_cds: no visible global function definition for ‘nei’
    cth_classifier_cell: no visible global function definition for ‘nei’
    diff_test_helper: no visible binding for global variable ‘Size_Factor’
    exportCDS: no visible binding for global variable ‘use_for_ordering’
    extract_good_ordering: no visible global function definition for ‘nei’
    fit_model_helper: no visible binding for global variable ‘Size_Factor’
    ...
      variable ‘Branch’
    project2MST: no visible global function definition for ‘nei’
    reverseEmbeddingCDS : <anonymous>: no visible global function
      definition for ‘quantile’
    Undefined global functions or variables:
      Branch Size_Factor nei next_node pseudocount quantile
      use_for_ordering
    Consider adding
      importFrom("stats", "quantile")
    to your NAMESPACE file.
    ```

*   checking files in ‘vignettes’ ... NOTE
    ```
    The following directory looks like a leftover from 'knitr':
      ‘figure’
    Please remove from your package.
    ```

# MOSim

<details>

* Version: 1.8.0
* GitHub: https://github.com/Neurergus/MOSim
* Source code: https://github.com/cran/MOSim
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 79

Run `revdep_details(, "MOSim")` for more info

</details>

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
    --- re-building ‘MOSim.Rnw’ using knitr
    Generating simulation settings for RNA-seq.
    Finishing generation of configuration settings.
    Configuration generated.
    
    Starting simulation of RNA-seq.
    - Simulating count values for group 1.
    	- Making replicates for group 1 on time 0.
    	- Making replicates for group 1 on time 2.
    ...
    Quitting from lines 223-249 (MOSim.Rnw) 
    Error: processing vignette 'MOSim.Rnw' failed with diagnostics:
    there is no package called 'codetools'
    --- failed re-building ‘MOSim.Rnw’
    
    SUMMARY: processing the following file failed:
      ‘MOSim.Rnw’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

# motifbreakR

<details>

* Version: 2.8.0
* GitHub: https://github.com/Simon-Coetzee/motifbreakR
* Source code: https://github.com/cran/motifbreakR
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 153

Run `revdep_details(, "motifbreakR")` for more info

</details>

## In both

*   checking whether package ‘motifbreakR’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘S4Vectors::as.data.frame’ by ‘motifStack::as.data.frame’ when loading ‘motifbreakR’
    See ‘/c4/home/henrik/repositories/matrixStats/revdep/checks/motifbreakR/new/motifbreakR.Rcheck/00install.out’ for details.
    ```

*   checking for missing documentation entries ... WARNING
    ```
    Undocumented code objects:
      'rs1006140' 'variants.from.file'
    Undocumented data sets:
      'rs1006140'
    All user-level objects in a package should have documentation entries.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking Rd \usage sections ... WARNING
    ```
    Undocumented arguments in documentation object 'motifbreakR'
      ‘legacy.score’
    
    Undocumented arguments in documentation object 'scoreAllWindows'
      ‘snp.seq’ ‘snp.seq.rc’ ‘pwm’ ‘from’ ‘to’ ‘pwm.range’ ‘calcp’
    
    Undocumented arguments in documentation object 'snps.from.file'
      ‘indels’
    
    Functions with \usage entries need to have the appropriate \alias
    entries, and all their arguments documented.
    The \usage entries must correspond to syntactically valid R code.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
    --- re-building ‘motifbreakR-vignette.Rmd’ using rmarkdown
    Loading required package: grid
    Loading required package: MotifDb
    Loading required package: BiocGenerics
    
    Attaching package: 'BiocGenerics'
    
    The following objects are masked from 'package:stats':
    
    ...
    Quitting from lines 68-71 (motifbreakR-vignette.Rmd) 
    Error: processing vignette 'motifbreakR-vignette.Rmd' failed with diagnostics:
    there is no package called 'codetools'
    --- failed re-building ‘motifbreakR-vignette.Rmd’
    
    SUMMARY: processing the following file failed:
      ‘motifbreakR-vignette.Rmd’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      'compiler' 'grImport'
      All declared Imports should be used.
    Package in Depends field not imported from: 'MotifDb'
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

# mrfDepth

<details>

* Version: 1.0.13
* GitHub: https://github.com/PSegaert/mrfDepth
* Source code: https://github.com/cran/mrfDepth
* Date/Publication: 2020-08-26 16:10:33 UTC
* Number of recursive dependencies: 47

Run `revdep_details(, "mrfDepth")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 20.9Mb
      sub-directories of 1Mb or more:
        data   1.6Mb
        libs  19.0Mb
    ```

# MultiBD

<details>

* Version: 0.2.0
* GitHub: NA
* Source code: https://github.com/cran/MultiBD
* Date/Publication: 2016-12-05 18:28:46
* Number of recursive dependencies: 73

Run `revdep_details(, "MultiBD")` for more info

</details>

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
    --- re-building ‘SIR-MCMC.Rmd’ using rmarkdown
    Warning: It is deprecated to specify `guide = FALSE` to remove a guide. Please use `guide = "none"` instead.
    --- finished re-building ‘SIR-MCMC.Rmd’
    
    --- re-building ‘SIRtrans.Rnw’ using knitr
    Error: processing vignette 'SIRtrans.Rnw' failed with diagnostics:
    Running 'texi2dvi' on 'SIRtrans.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `bbm.sty' not found.
    ...
    l.86 \usepackage
                    {amsmath}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    --- failed re-building ‘SIRtrans.Rnw’
    
    SUMMARY: processing the following file failed:
      ‘SIRtrans.Rnw’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  9.4Mb
      sub-directories of 1Mb or more:
        libs   7.9Mb
    ```

# muscat

<details>

<<<<<<< HEAD
* Version: 1.8.0
||||||| cc87037
* Version: 1.6.0
=======
* Version: 1.8.2
>>>>>>> develop
* GitHub: https://github.com/HelenaLC/muscat
* Source code: https://github.com/cran/muscat
<<<<<<< HEAD
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 214
||||||| cc87037
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 213
=======
* Date/Publication: 2022-03-10
* Number of recursive dependencies: 218
>>>>>>> develop

Run `revdep_details(, "muscat")` for more info

</details>

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
    --- re-building ‘analysis.Rmd’ using rmarkdown
    
    Attaching package: 'dplyr'
    
    The following objects are masked from 'package:stats':
    
        filter, lag
    
    The following objects are masked from 'package:base':
    ...
        ident, sql
    
    Testing for internet connectivity via https_proxy... success!
    snapshotDate(): 2021-10-19
    Bioconductor version 3.14 (BiocManager 1.30.16), R 4.1.3 (2022-03-10)
    Installing package(s) 'muscData'
    trying URL 'https://bioconductor.org/packages/3.14/data/experiment/src/contrib/muscData_1.8.0.tar.gz'
    Content type 'application/x-gzip' length 235882 bytes (230 KB)
    ==================================================
    downloaded 230 KB
    ```

*   checking installed package size ... NOTE
    ```
<<<<<<< HEAD
      installed size is  8.0Mb
||||||| cc87037
      installed size is  9.0Mb
=======
      installed size is  7.6Mb
>>>>>>> develop
      sub-directories of 1Mb or more:
<<<<<<< HEAD
        data   1.4Mb
        doc    5.8Mb
||||||| cc87037
        data   2.3Mb
        doc    5.8Mb
=======
        doc   5.8Mb
>>>>>>> develop
    ```

# muscData

<details>

* Version: 1.8.0
* GitHub: https://github.com/HelenaLC/muscData
* Source code: https://github.com/cran/muscData
<<<<<<< HEAD
* Date/Publication: 2021-10-30
* Number of recursive dependencies: 263
||||||| cc87037
* Date/Publication: 2021-05-20
* Number of recursive dependencies: 250
=======
* Date/Publication: 2021-10-30
* Number of recursive dependencies: 267
>>>>>>> develop

Run `revdep_details(, "muscData")` for more info

</details>

## In both

*   checking for missing documentation entries ... WARNING
    ```
    Undocumented code objects:
      ‘Crowell19_4vs4’
    All user-level objects in a package should have documentation entries.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .BBSoptions
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Package in Depends field not imported from: ‘SingleCellExperiment’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    Testing for internet connectivity via https_proxy... success!
    ```

# NanoStringDiff

<details>

* Version: 1.24.0
* GitHub: NA
* Source code: https://github.com/cran/NanoStringDiff
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 54

Run `revdep_details(, "NanoStringDiff")` for more info

</details>

## In both

*   checking for missing documentation entries ... WARNING
    ```
    Undocumented code objects:
      ‘compute.baseSigma’ ‘est.dispersion’ ‘fun5’ ‘glmfit.OneGroup’
      ‘glmfit.full’ ‘glmfit.reduce’ ‘rnegbinom’
    All user-level objects in a package should have documentation entries.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking installed files from ‘inst/doc’ ... NOTE
    ```
    The following files look like leftovers/mistakes:
      ‘NanoStringDiff.log’
    Please remove them from your package.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error(s) in re-building vignettes:
    --- re-building ‘NanoStringDiff.Rnw’ using Sweave
    Loading required package: Biobase
    Loading required package: BiocGenerics
    
    Attaching package: ‘BiocGenerics’
    
    The following objects are masked from ‘package:stats’:
    
        IQR, mad, sd, var, xtabs
    ...
    l.37 \sectionfont
                     {\sffamily\bfseries\color{BiocBlue}\sectionrule{0pt}{0pt}{-...
    !  ==> Fatal error occurred, no output PDF file produced!
    --- failed re-building ‘NanoStringDiff.Rnw’
    
    SUMMARY: processing the following file failed:
      ‘NanoStringDiff.Rnw’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

# nearBynding

<details>

* Version: 1.4.0
* GitHub: NA
* Source code: https://github.com/cran/nearBynding
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 125

Run `revdep_details(, "nearBynding")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.7Mb
      sub-directories of 1Mb or more:
        extdata   6.9Mb
    ```

# nempi

<details>

* Version: 1.2.0
* GitHub: https://github.com/cbg-ethz/nempi
* Source code: https://github.com/cran/nempi
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 117

Run `revdep_details(, "nempi")` for more info

</details>

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
      ...
    --- re-building ‘nempi.Rmd’ using rmarkdown
    Error: processing vignette 'nempi.Rmd' failed with diagnostics:
    there is no package called ‘BiocStyle’
    --- failed re-building ‘nempi.Rmd’
    
    SUMMARY: processing the following file failed:
      ‘nempi.Rmd’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

# NetLogoR

<details>

* Version: 0.3.10
* GitHub: https://github.com/PredictiveEcology/NetLogoR
* Source code: https://github.com/cran/NetLogoR
* Date/Publication: 2022-02-18 08:20:11 UTC
* Number of recursive dependencies: 111

Run `revdep_details(, "NetLogoR")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘fastshp’
    ```

# NormalyzerDE

<details>

* Version: 1.12.0
* GitHub: https://github.com/ComputationalProteomics/NormalyzerDE
* Source code: https://github.com/cran/NormalyzerDE
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 158

Run `revdep_details(, "NormalyzerDE")` for more info

</details>

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
      ...
    --- re-building ‘vignette.Rmd’ using rmarkdown
    A new version of TeX Live has been released. If you need to install or update any LaTeX packages, you have to upgrade TinyTeX with tinytex::reinstall_tinytex(). If it fails to upgrade, you might be using a default random CTAN mirror that has not been fully synced to the main CTAN repository, and you need to wait for a few more days or use a CTAN mirror that is known to be up-to-date (see the "repository" argument on the help page ?tinytex::install_tinytex).
    ! LaTeX Error: Missing \begin{document}.
    
    Error: processing vignette 'vignette.Rmd' failed with diagnostics:
    LaTeX failed to compile /c4/home/henrik/repositories/matrixStats/revdep/checks/NormalyzerDE/new/NormalyzerDE.Rcheck/vign_test/NormalyzerDE/vignettes/vignette.tex. See https://yihui.org/tinytex/r/#debugging for debugging tips. See vignette.log for more info.
    --- failed re-building ‘vignette.Rmd’
    
    SUMMARY: processing the following file failed:
      ‘vignette.Rmd’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

# obfuscatoR

<details>

* Version: 0.2.1
* GitHub: https://github.com/edsandorf/obfuscatoR
* Source code: https://github.com/cran/obfuscatoR
* Date/Publication: 2020-11-19 08:20:09 UTC
* Number of recursive dependencies: 65

Run `revdep_details(, "obfuscatoR")` for more info

</details>

## In both

*   checking LazyData ... NOTE
    ```
      'LazyData' is specified without a 'data' directory
    ```

# omicplotR

<details>

* Version: 1.14.0
* GitHub: NA
* Source code: https://github.com/cran/omicplotR
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 87

Run `revdep_details(, "omicplotR")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Package in Depends field not imported from: ‘ALDEx2’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

# omicsPrint

<details>

* Version: 1.14.0
* GitHub: NA
* Source code: https://github.com/cran/omicsPrint
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 137

Run `revdep_details(, "omicsPrint")` for more info

</details>

## In both

*   checking LazyData ... WARNING
    ```
      LazyData DB of 7.3 MB without LazyDataCompression set
      See §1.1.6 of 'Writing R Extensions'
    ```

*   checking installed package size ... NOTE
    ```
<<<<<<< HEAD
      installed size is  8.6Mb
      sub-directories of 1Mb or more:
        data   7.3Mb
        doc    1.2Mb
    ```

||||||| cc87037
      installed size is 16.6Mb
      sub-directories of 1Mb or more:
        data  15.3Mb
        doc    1.2Mb
    ```

# omicwas

<details>

* Version: 0.8.0
* GitHub: https://github.com/fumi-github/omicwas
* Source code: https://github.com/cran/omicwas
* Date/Publication: 2020-10-08 12:50:03 UTC
* Number of recursive dependencies: 64

Run `revdep_details(, "omicwas")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.0Mb
      sub-directories of 1Mb or more:
        data   6.8Mb
    ```

=======
      installed size is  8.6Mb
      sub-directories of 1Mb or more:
        data   7.3Mb
        doc    1.2Mb
    ```

>>>>>>> develop
# OptimalDesign

<details>

* Version: 1.0.1
* GitHub: NA
* Source code: https://github.com/cran/OptimalDesign
* Date/Publication: 2019-12-02 08:50:07 UTC
* Number of recursive dependencies: 26

Run `revdep_details(, "OptimalDesign")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Packages which this enhances but not available for checking:
      'gurobi', 'slam'
    ```

*   checking LazyData ... NOTE
    ```
      'LazyData' is specified without a 'data' directory
    ```

# OUTRIDER

<details>

* Version: 1.12.0
* GitHub: https://github.com/gagneurlab/OUTRIDER
* Source code: https://github.com/cran/OUTRIDER
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 175

Run `revdep_details(, "OUTRIDER")` for more info

</details>

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
    --- re-building ‘OUTRIDER.Rnw’ using knitr
    The magick package is required to crop "figure/deVsOutlier-1.png" but not available.
    229 genes did not pass the filter due to zero counts. This is 22.9% of the genes.
<<<<<<< HEAD
    Mon Mar 28 16:51:23 2022: SizeFactor estimation ...
    Mon Mar 28 16:51:24 2022: Controlling for confounders ...
||||||| cc87037
    Wed Sep 15 06:49:01 2021: SizeFactor estimation ...
    Wed Sep 15 06:49:01 2021: Controlling for confounders ...
=======
    Sat Apr 16 20:54:32 2022: SizeFactor estimation ...
    Sat Apr 16 20:54:32 2022: Controlling for confounders ...
>>>>>>> develop
    Using estimated q with: 23
<<<<<<< HEAD
    Mon Mar 28 16:51:24 2022: Using the autoencoder implementation for controlling.
    Mon Mar 28 16:54:31 2022: Used the autoencoder implementation for controlling.
    Mon Mar 28 16:54:31 2022: P-value calculation ...
||||||| cc87037
    Wed Sep 15 06:49:01 2021: Using the autoencoder implementation for controlling.
    Wed Sep 15 06:54:46 2021: Used the autoencoder implementation for controlling.
    Wed Sep 15 06:54:46 2021: P-value calculation ...
=======
    Sat Apr 16 20:54:32 2022: Using the autoencoder implementation for controlling.
    Sat Apr 16 20:57:23 2022: Used the autoencoder implementation for controlling.
    Sat Apr 16 20:57:23 2022: P-value calculation ...
>>>>>>> develop
    ...
    l.189 \RequirePackage
                         {parnotes}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    --- failed re-building ‘OUTRIDER.Rnw’
    
    SUMMARY: processing the following file failed:
      ‘OUTRIDER.Rnw’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.1Mb
      sub-directories of 1Mb or more:
        doc    2.2Mb
        libs   2.8Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘Rcpp’
      All declared Imports should be used.
    ```

# pandaR

<details>

* Version: 1.26.0
* GitHub: NA
* Source code: https://github.com/cran/pandaR
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 58

Run `revdep_details(, "pandaR")` for more info

</details>

## In both

*   checking package subdirectories ... WARNING
    ```
    Invalid citation information in ‘inst/CITATION’:
      Error in tools:::.parse_CITATION_file(file, meta$Encoding): non-ASCII input in a CITATION file without a declared encoding
    ```

*   checking LazyData ... WARNING
    ```
      LazyData DB of 9.0 MB without LazyDataCompression set
      See §1.1.6 of 'Writing R Extensions'
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  9.3Mb
      sub-directories of 1Mb or more:
        data   9.0Mb
    ```

*   checking S3 generic/method consistency ... NOTE
    ```
    Found the following apparent S3 methods exported but not registered:
      summary.panda
    See section ‘Registering S3 methods’ in the ‘Writing R Extensions’
    manual.
    ```

*   checking Rd \usage sections ... NOTE
    ```
    S3 methods shown with full name in documentation object 'summary.panda':
      ‘summary.panda’
    
    The \usage entries for S3 methods should use the \method markup and not
    their full name.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

# PathoStat

<details>

* Version: 1.20.0
* GitHub: https://github.com/mani2012/PathoStat
* Source code: https://github.com/cran/PathoStat
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 196

Run `revdep_details(, "PathoStat")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘ComplexHeatmap’ ‘RColorBrewer’
      All declared Imports should be used.
    ```

# pcaMethods

<details>

* Version: 1.86.0
* GitHub: https://github.com/hredestig/pcamethods
* Source code: https://github.com/cran/pcaMethods
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 35

Run `revdep_details(, "pcaMethods")` for more info

</details>

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘ade4’
    ```

# PDATK

<details>

* Version: 1.2.0
* GitHub: https://github.com/bhklab/PDATK
* Source code: https://github.com/cran/PDATK
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 261

Run `revdep_details(, "PDATK")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.3Mb
      sub-directories of 1Mb or more:
        doc   3.3Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    .calculateMSMthresholds: no visible binding for global variable
      ‘funContext’
    .plotNetwork: no visible global function definition for ‘legend’
    NCSModel: no visible binding for global variable ‘metric’
    NCSModel: no visible binding for global variable ‘comparison’
    NCSModel: no visible binding for global variable ‘centroid_K’
    NCSModel: no visible binding for global variable ‘assay_K’
    NetworkCommunitySearchModel: no visible binding for global variable
      ‘metric’
    NetworkCommunitySearchModel: no visible binding for global variable
    ...
    trainModel,NCSModel: no visible binding for global variable
      ‘cor_threshold’
    Undefined global functions or variables:
      .error assay_K assay_cohort centroid_K centroid_cohort cluster_label
      comparison cor_threshold dev.off feature feature_rank feature_score
      funContext ingroup_proportion legend metric missignAssays pdf tmp
    Consider adding
      importFrom("grDevices", "dev.off", "pdf")
      importFrom("graphics", "legend")
    to your NAMESPACE file.
    ```

# peakPick

<details>

* Version: 0.11
* GitHub: NA
* Source code: https://github.com/cran/peakPick
* Date/Publication: 2015-12-04 15:40:34
* Number of recursive dependencies: 24

Run `revdep_details(, "peakPick")` for more info

</details>

## In both

*   checking LazyData ... NOTE
    ```
      'LazyData' is specified without a 'data' directory
    ```

# phosphonormalizer

<details>

* Version: 1.18.0
* GitHub: NA
* Source code: https://github.com/cran/phosphonormalizer
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 50

Run `revdep_details(, "phosphonormalizer")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package which this enhances but not available for checking: ‘MSnbase’
    ```

# Pigengene

<details>

* Version: 1.20.0
* GitHub: NA
* Source code: https://github.com/cran/Pigengene
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 188

Run `revdep_details(, "Pigengene")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘Pigengene-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: gene.mapping
    > ### Title: Maps gene IDs
    > ### Aliases: gene.mapping
    > ### Keywords: misc
    > 
    > ### ** Examples
    > 
    ...
    +         outputDb=list(org.Hs.eg.db,org.Mm.eg.db), verbose=1)
    Mapping to:  org.Hs.eg.db-SYMBOL 
    'select()' returned 1:1 mapping between keys and columns
    Loading required package: biomaRt
    Warning: Ensembl will soon enforce the use of https.
    Ensure the 'host' argument includes "https://"
    Error in .readFromCache(bfc, hash) : Multiple cache results found.
    Please clear your cache by running biomartCacheClear()
    Calls: gene.mapping ... gene.mapping -> hu.mouse -> <Anonymous> -> .readFromCache
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
    --- re-building ‘Pigengene_inference.Rnw’ using knitr
    Warning in has_utility("pdfcrop") :
      pdfcrop not installed or not in PATH
    sh: pdfcrop: command not found
    Warning in system2("pdfcrop", shQuote(c(x, x)), stdout = if (quiet) FALSE else "") :
      error in running command
    sh: pdfcrop: command not found
    Warning in system2("pdfcrop", shQuote(c(x, x)), stdout = if (quiet) FALSE else "") :
      error in running command
    ...
    l.189 \RequirePackage
                         {parnotes}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    --- failed re-building ‘Pigengene_inference.Rnw’
    
    SUMMARY: processing the following file failed:
      ‘Pigengene_inference.Rnw’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls in package code:
      ‘AnnotationDbi’ ‘biomaRt’ ‘energy’ ‘org.Hs.eg.db’ ‘org.Mm.eg.db’
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    Package in Depends field not imported from: ‘BiocStyle’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    Found the following possibly unsafe calls:
    File ‘Pigengene/R/bn.calculation.R’:
      assignInNamespace("supported.clusters", fixArgs, "bnlearn")
    
    apply.filter: no visible binding for global variable ‘epsilon’
    get.enriched.pw: no visible global function definition for ‘enrichNCG’
    one.step.pigengene: no visible binding for global variable
      ‘org.Hs.eg.db’
    Undefined global functions or variables:
      enrichNCG epsilon org.Hs.eg.db
    ```

# pipeComp

<details>

* Version: 1.4.0
* GitHub: https://github.com/plger/pipeComp
* Source code: https://github.com/cran/pipeComp
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 199

Run `revdep_details(, "pipeComp")` for more info

</details>

## In both

*   checking contents of ‘data’ directory ... WARNING
    ```
    Output for data("exampleDEAresults", package = "pipeComp"):
      Warning: namespace ‘DBI’ is not available and has been replaced
      by .GlobalEnv when processing object ‘exampleDEAresults’
      Warning: namespace ‘DBI’ is not available and has been replaced
      by .GlobalEnv when processing object ‘exampleDEAresults’
    Output for data("exampleResults", package = "pipeComp"):
      Warning: namespace ‘DBI’ is not available and has been replaced
      by .GlobalEnv when processing object ‘exampleResults’
      Warning: namespace ‘DBI’ is not available and has been replaced
      by .GlobalEnv when processing object ‘exampleResults’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.2Mb
      sub-directories of 1Mb or more:
        doc   4.5Mb
    ```

*   checking top-level files ... NOTE
    ```
    File
      LICENSE
    is not mentioned in the DESCRIPTION file.
    ```

*   checking dependencies in R code ... NOTE
    ```
    There are ::: calls to the package's namespace in its code. A package
      almost never needs to use ::: for its own objects:
      ‘.compileExcludedCells’ ‘.getMM’ ‘.homogenizeDEA’ ‘.runf’
    ```

*   checking R code for possible problems ... NOTE
    ```
    scrna_describeDatasets: no visible binding for global variable
      ‘cluster’
    scrna_describeDatasets: no visible binding for global variable ‘nb’
    scrna_describeDatasets : pf: no visible binding for global variable
      ‘.x’
    scrna_describeDatasets : rd: no visible binding for global variable ‘y’
    scrna_describeDatasets : rd: no visible binding for global variable
      ‘cluster’
    scrna_evalPlot_filtering: no visible binding for global variable
      ‘max.lost’
    ...
    scrna_evalPlot_filtering: no visible binding for global variable
      ‘doubletmethod’
    scrna_evalPlot_filtering: no visible binding for global variable
      ‘method’
    scrna_evalPlot_overall: no visible binding for global variable
      ‘true.nbClusts’
    scrna_evalPlot_overall: no visible binding for global variable ‘n_clus’
    Undefined global functions or variables:
      .x cluster doubletmethod filt max.lost mean_F1 method n_clus nb
      true.nbClusts y
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    'library' or 'require' calls not declared from:
      ‘edgeR’ ‘sva’
    ```

# pmp

<details>

* Version: 1.6.0
* GitHub: NA
* Source code: https://github.com/cran/pmp
<<<<<<< HEAD
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 102
||||||| cc87037
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 98
=======
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 104
>>>>>>> develop

Run `revdep_details(, "pmp")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.5Mb
      sub-directories of 1Mb or more:
        data   1.8Mb
        doc    5.5Mb
    ```

# PrecisionTrialDrawer

<details>

* Version: 1.10.0
* GitHub: NA
* Source code: https://github.com/cran/PrecisionTrialDrawer
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 128

Run `revdep_details(, "PrecisionTrialDrawer")` for more info

</details>

## In both

<<<<<<< HEAD
*   checking examples ... ERROR
    ```
    Running examples in ‘PrecisionTrialDrawer-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: newCancerPanel
    > ### Title: CancerPanel object constructor
    > ### Aliases: newCancerPanel
    > 
    > ### ** Examples
    > 
    > # Load the panel example
    ...
    Warning: Ensembl will soon enforce the use of https.
    Ensure the 'host' argument includes "https://"
    Ensembl site unresponsive, trying uswest mirror
    Ensembl site unresponsive, trying asia mirror
    Warning: Ensembl will soon enforce the use of https.
    Ensure the 'host' argument includes "https://"
    Ensembl site unresponsive, trying www mirror
    Error: Your query has been redirected to http://status.ensembl.org indicating this Ensembl service is currently unavailable.
    Look at ?useEnsembl for details on how to try a mirror site.
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
      ...
    --- re-building ‘PrecisionTrialDrawer.Rmd’ using rmarkdown
    Creating a generic function for 'toJSON' from package 'jsonlite' in package 'googleVis'
    Warning: Ensembl will soon enforce the use of https.
    Ensure the 'host' argument includes "https://"
    Warning: Ensembl will soon enforce the use of https.
    Ensure the 'host' argument includes "https://"
    Warning in if (BioMartVersion == "\n" | BioMartVersion == "") { :
      the condition has length > 1 and only the first element will be used
    ...
    Quitting from lines 165-166 (PrecisionTrialDrawer.Rmd) 
    Error: processing vignette 'PrecisionTrialDrawer.Rmd' failed with diagnostics:
    invalid 'text' argument
    --- failed re-building ‘PrecisionTrialDrawer.Rmd’
    
    SUMMARY: processing the following file failed:
      ‘PrecisionTrialDrawer.Rmd’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

||||||| cc87037
=======
*   checking examples ... ERROR
    ```
    Running examples in ‘PrecisionTrialDrawer-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: newCancerPanel
    > ### Title: CancerPanel object constructor
    > ### Aliases: newCancerPanel
    > 
    > ### ** Examples
    > 
    > # Load the panel example
    ...
    Calculating panel size...
    Connecting to ensembl biomart...
    Warning: Ensembl will soon enforce the use of https.
    Ensure the 'host' argument includes "https://"
    Warning: Ensembl will soon enforce the use of https.
    Ensure the 'host' argument includes "https://"
    Error in .readFromCache(bfc, hash) : Multiple cache results found.
    Please clear your cache by running biomartCacheClear()
    Calls: newCancerPanel ... .annotateGeneLength -> <Anonymous> -> .readFromCache
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
      ...
    --- re-building ‘PrecisionTrialDrawer.Rmd’ using rmarkdown
    Creating a generic function for 'toJSON' from package 'jsonlite' in package 'googleVis'
    Warning: Ensembl will soon enforce the use of https.
    Ensure the 'host' argument includes "https://"
    Warning: Ensembl will soon enforce the use of https.
    Ensure the 'host' argument includes "https://"
    Quitting from lines 165-166 (PrecisionTrialDrawer.Rmd) 
    Error: processing vignette 'PrecisionTrialDrawer.Rmd' failed with diagnostics:
    Multiple cache results found.
    Please clear your cache by running biomartCacheClear()
    --- failed re-building ‘PrecisionTrialDrawer.Rmd’
    
    SUMMARY: processing the following file failed:
      ‘PrecisionTrialDrawer.Rmd’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

>>>>>>> develop
*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘LowMACA’
    ```

# ProteoDisco

<details>

* Version: 1.0.0
* GitHub: https://github.com/ErasmusMC-CCBC/ProteoDisco
* Source code: https://github.com/cran/ProteoDisco
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 169

Run `revdep_details(, "ProteoDisco")` for more info

</details>

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
    --- re-building ‘Overview_ProteoDisco.Rmd’ using rmarkdown
    Currently in a tryCatch or withCallingHandlers block, so unable to add global calling handlers. ParallelLogger will not capture R messages, errors, and warnings, only explicit calls to ParallelLogger.
    Currently in a tryCatch or withCallingHandlers block, so unable to add global calling handlers. ParallelLogger will not capture R messages, errors, and warnings, only explicit calls to ParallelLogger.
    Loading required package: VariantAnnotation
    Loading required package: MatrixGenerics
    Loading required package: matrixStats
    
    Attaching package: 'matrixStats'
    
    ...
    Quitting from lines 281-302 (Overview_ProteoDisco.Rmd) 
    Error: processing vignette 'Overview_ProteoDisco.Rmd' failed with diagnostics:
    there is no package called 'codetools'
    --- failed re-building ‘Overview_ProteoDisco.Rmd’
    
    SUMMARY: processing the following file failed:
      ‘Overview_ProteoDisco.Rmd’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Error loading dataset 'ProteoDiscographyExample.hg19':
       Error in h(simpleError(msg, call)) : 
        error in evaluating the argument 'x' in selecting a method for function 'seqlevels': UCSC library operation failed
      
    ```

# ProteoMM

<details>

* Version: 1.12.0
* GitHub: NA
* Source code: https://github.com/cran/ProteoMM
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 99

Run `revdep_details(, "ProteoMM")` for more info

</details>

## In both

*   checking DESCRIPTION meta-information ... NOTE
    ```
    License components which are templates and need '+ file LICENSE':
      MIT
    ```

# QDNAseq

<details>

* Version: 1.30.0
* GitHub: https://github.com/ccagc/QDNAseq
* Source code: https://github.com/cran/QDNAseq
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 80

Run `revdep_details(, "QDNAseq")` for more info

</details>

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    Error(s) in re-building vignettes:
    --- re-building ‘QDNAseq.Rnw’ using Sweave
    EM algorithm started ... 
    
    Warning in allprior/tot :
      Recycling array of length 1 in vector-array arithmetic is deprecated.
      Use c() or as.vector() instead.
    
    Warning in allprior/tot :
      Recycling array of length 1 in vector-array arithmetic is deprecated.
    ...
    l.189 \RequirePackage
                         {parnotes}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    --- failed re-building ‘QDNAseq.Rnw’
    
    SUMMARY: processing the following file failed:
      ‘QDNAseq.Rnw’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

# QFeatures

<details>

* Version: 1.4.0
* GitHub: https://github.com/RforMassSpectrometry/QFeatures
* Source code: https://github.com/cran/QFeatures
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 151

Run `revdep_details(, "QFeatures")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.1Mb
      sub-directories of 1Mb or more:
        doc   4.2Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      ‘BiocGenerics:::replaceSlots’
      ‘MultiAssayExperiment:::.sampleMapFromData’
      See the note in ?`:::` about the use of this operator.
    ```

# r2dRue

<details>

* Version: 1.0.4
* GitHub: NA
* Source code: https://github.com/cran/r2dRue
* Date/Publication: 2013-06-28 12:22:19
* Number of recursive dependencies: 4

Run `revdep_details(, "r2dRue")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' call to ‘rgdal’ which was already attached by Depends.
      Please remove these calls from your code.
    Packages in Depends field not imported from:
      ‘matrixStats’ ‘rgdal’ ‘sp’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

# RaceID

<details>

* Version: 0.2.3
* GitHub: NA
* Source code: https://github.com/cran/RaceID
* Date/Publication: 2021-04-08 06:10:08 UTC
* Number of recursive dependencies: 149

Run `revdep_details(, "RaceID")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 10.3Mb
      sub-directories of 1Mb or more:
        data   2.1Mb
        doc    5.8Mb
        libs   1.8Mb
    ```

# rADA

<details>

* Version: 1.1.9
* GitHub: NA
* Source code: https://github.com/cran/rADA
* Date/Publication: 2021-03-23 19:40:06 UTC
* Number of recursive dependencies: 107

Run `revdep_details(, "rADA")` for more info

</details>

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
      ...
    --- re-building ‘rada_vignette.Rmd’ using knitr
    Loading required package: grid
    Loading required package: magrittr
    Loading required package: checkmate
    Quitting from lines 62-70 (rada_vignette.Rmd) 
    Error: processing vignette 'rada_vignette.Rmd' failed with diagnostics:
    there is no package called 'codetools'
    --- failed re-building ‘rada_vignette.Rmd’
    ...
    Quitting from lines 62-70 (rada_vignette_supplemental.Rmd) 
    Error: processing vignette 'rada_vignette_supplemental.Rmd' failed with diagnostics:
    there is no package called 'codetools'
    --- failed re-building ‘rada_vignette_supplemental.Rmd’
    
    SUMMARY: processing the following files failed:
      ‘rada_vignette.Rmd’ ‘rada_vignette_supplemental.Rmd’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.6Mb
      sub-directories of 1Mb or more:
        doc   5.5Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘Hmisc’ ‘car’ ‘dplyr’ ‘grid’ ‘gridExtra’ ‘knitr’ ‘lmerTest’ ‘tidyr’
      All declared Imports should be used.
    ```

# RadioGx

<details>

* Version: 1.4.0
* GitHub: NA
* Source code: https://github.com/cran/RadioGx
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 136

Run `revdep_details(, "RadioGx")` for more info

</details>

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
    --- re-building ‘RadioGx.Rmd’ using knitr
    Loading required package: CoreGx
    Loading required package: BiocGenerics
    
    Attaching package: 'BiocGenerics'
    
    The following objects are masked from 'package:stats':
    
        IQR, mad, sd, var, xtabs
    ...
    Quitting from lines 86-87 (RadioGx.Rmd) 
    Error: processing vignette 'RadioGx.Rmd' failed with diagnostics:
    cannot open URL 'https://zenodo.org/record/3899568/files/availableRadioSets.csv?download=1'
    --- failed re-building ‘RadioGx.Rmd’
    
    SUMMARY: processing the following file failed:
      ‘RadioGx.Rmd’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.0Mb
      sub-directories of 1Mb or more:
        data   3.5Mb
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 14 marked UTF-8 strings
    ```

<<<<<<< HEAD
||||||| cc87037
# ramr

<details>

* Version: 1.0.3
* GitHub: https://github.com/BBCG/ramr
* Source code: https://github.com/cran/ramr
* Date/Publication: 2021-08-15
* Number of recursive dependencies: 154

Run `revdep_details(, "ramr")` for more info

</details>

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
      ...
    --- re-building ‘ramr.Rmd’ using rmarkdown
    Quitting from lines 236-244 (ramr.Rmd) 
    Error: processing vignette 'ramr.Rmd' failed with diagnostics:
    package or namespace load failed for 'annotatr' in loadNamespace(j <- i[[1L]], c(lib.loc, .libPaths()), versionCheck = vI[[j]]):
     there is no package called 'MatrixGenerics'
    --- failed re-building ‘ramr.Rmd’
    
    SUMMARY: processing the following file failed:
      ‘ramr.Rmd’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

=======
# ramr

<details>

* Version: 1.2.0
* GitHub: https://github.com/BBCG/ramr
* Source code: https://github.com/cran/ramr
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 155

Run `revdep_details(, "ramr")` for more info

</details>

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
      ...
    --- re-building ‘ramr.Rmd’ using rmarkdown
    Quitting from lines 236-244 (ramr.Rmd) 
    Error: processing vignette 'ramr.Rmd' failed with diagnostics:
    Corrupt Cache: sqlite file
      See AnnotationHub's TroubleshootingTheCache vignette section on corrupt cache
      cache: /c4/home/henrik/.cache/R/AnnotationHub
      filename: annotationhub.sqlite3
    --- failed re-building ‘ramr.Rmd’
    
    SUMMARY: processing the following file failed:
      ‘ramr.Rmd’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

>>>>>>> develop
# regsem

<details>

* Version: 1.8.0
* GitHub: NA
* Source code: https://github.com/cran/regsem
* Date/Publication: 2021-06-03 15:00:02 UTC
* Number of recursive dependencies: 161

Run `revdep_details(, "regsem")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.8Mb
      sub-directories of 1Mb or more:
        libs   5.5Mb
    ```

# rhierbaps

<details>

* Version: 1.1.3
* GitHub: https://github.com/gtonkinhill/rhierbaps
* Source code: https://github.com/cran/rhierbaps
* Date/Publication: 2019-12-11 14:20:05 UTC
* Number of recursive dependencies: 97

Run `revdep_details(, "rhierbaps")` for more info

</details>

## In both

*   checking LazyData ... NOTE
    ```
      'LazyData' is specified without a 'data' directory
    ```

# RJcluster

<details>

* Version: 3.2.4
* GitHub: NA
* Source code: https://github.com/cran/RJcluster
* Date/Publication: 2022-02-14 21:30:02 UTC
* Number of recursive dependencies: 57

Run `revdep_details(, "RJcluster")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.1Mb
      sub-directories of 1Mb or more:
        libs   6.0Mb
    ```

# RNAmodR

<details>

* Version: 1.8.0
* GitHub: https://github.com/FelixErnst/RNAmodR
* Source code: https://github.com/cran/RNAmodR
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 190

Run `revdep_details(, "RNAmodR")` for more info

</details>

## In both

<<<<<<< HEAD
*   checking examples ... ERROR
    ```
    Running examples in ‘RNAmodR-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: CoverageSequenceData-class
    > ### Title: CoverageSequenceData
    > ### Aliases: CoverageSequenceData-class CoverageSequenceData
    > ###   CoverageSequenceDataFrame CoverageSequenceDataFrame-class
    > ###   getData,CoverageSequenceData,BamFileList,GRangesList,XStringSet,ScanBamParam-method
    > ###   aggregateData,CoverageSequenceData-method
    > ###   getDataTrack,CoverageSequenceData-method
    ...
      hub path: 'https://experimenthub.bioconductor.org/fetch/2545'
      cache resource: 'EH2532 : 2545'
      reason: Corrupt Cache: resource id
      More than one entry in cache for: EH2532 : 2545
      See AnnotationHub's TroubleshootingTheCache vignette section on corrupt cache
    Error: failed to load resource
      name: EH2532
      title: RNAmodR.Data.example.wt.1
      reason: 1 resources failed to download
    Execution halted
    ```

*   checking tests ...
    ```
      Running ‘testthat.R’
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 50 lines of output:
      Error: failed to load resource
        name: EH2532
        title: RNAmodR.Data.example.wt.1
        reason: 1 resources failed to download
      Backtrace:
           ▆
    ...
       16. │                 └─value[[3L]](cond)
       17. │                   └─base::stop(...)
       18. ├─base::.handleSimpleError(...)
       19. │ └─base h(simpleError(msg, call))
       20. └─base::.handleSimpleError(...)
       21.   └─base h(simpleError(msg, call))
      
      [ FAIL 3 | WARN 9 | SKIP 0 | PASS 481 ]
      Error: Test failures
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
    --- re-building ‘RNAmodR.Rmd’ using rmarkdown
    Testing for internet connectivity via https_proxy... success!
    see ?RNAmodR.Data and browseVignettes('RNAmodR.Data') for documentation
    loading from cache
    see ?RNAmodR.Data and browseVignettes('RNAmodR.Data') for documentation
    loading from cache
    see ?RNAmodR.Data and browseVignettes('RNAmodR.Data') for documentation
    loading from cache
    see ?RNAmodR.Data and browseVignettes('RNAmodR.Data') for documentation
    ...
    --- failed re-building ‘RNAmodR.Rmd’
    
    --- re-building ‘RNAmodR.creation.Rmd’ using rmarkdown
    --- finished re-building ‘RNAmodR.creation.Rmd’
    
    SUMMARY: processing the following file failed:
      ‘RNAmodR.Rmd’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

*   checking dependencies in R code ... NOTE
||||||| cc87037
*   checking dependencies in R code ... NOTE
=======
*   checking examples ... ERROR
>>>>>>> develop
    ```
    Running examples in ‘RNAmodR-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: CoverageSequenceData-class
    > ### Title: CoverageSequenceData
    > ### Aliases: CoverageSequenceData-class CoverageSequenceData
    > ###   CoverageSequenceDataFrame CoverageSequenceDataFrame-class
    > ###   getData,CoverageSequenceData,BamFileList,GRangesList,XStringSet,ScanBamParam-method
    > ###   aggregateData,CoverageSequenceData-method
    > ###   getDataTrack,CoverageSequenceData-method
    ...
      hub path: 'https://experimenthub.bioconductor.org/fetch/2545'
      cache resource: 'EH2532 : 2545'
      reason: Corrupt Cache: resource id
      More than one entry in cache for: EH2532 : 2545
      See AnnotationHub's TroubleshootingTheCache vignette section on corrupt cache
    Error: failed to load resource
      name: EH2532
      title: RNAmodR.Data.example.wt.1
      reason: 1 resources failed to download
    Execution halted
    ```

<<<<<<< HEAD
# RNAseqQC

<details>

* Version: 0.1.1
* GitHub: NA
* Source code: https://github.com/cran/RNAseqQC
* Date/Publication: 2022-03-18 16:40:02 UTC
* Number of recursive dependencies: 167

Run `revdep_details(, "RNAseqQC")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.7Mb
      sub-directories of 1Mb or more:
        data   3.4Mb
        doc    2.2Mb
    ```

# RnBeads

<details>

* Version: 2.12.2
||||||| cc87037
# RnBeads

<details>

* Version: 2.10.0
=======
*   checking tests ...
    ```
      Running ‘testthat.R’
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 50 lines of output:
      Error: failed to load resource
        name: EH2532
        title: RNAmodR.Data.example.wt.1
        reason: 1 resources failed to download
      Backtrace:
           ▆
    ...
       16. │                 └─value[[3L]](cond)
       17. │                   └─base::stop(...)
       18. ├─base::.handleSimpleError(...)
       19. │ └─base h(simpleError(msg, call))
       20. └─base::.handleSimpleError(...)
       21.   └─base h(simpleError(msg, call))
      
      [ FAIL 3 | WARN 9 | SKIP 0 | PASS 481 ]
      Error: Test failures
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
    --- re-building ‘RNAmodR.Rmd’ using rmarkdown
    Testing for internet connectivity via https_proxy... success!
    see ?RNAmodR.Data and browseVignettes('RNAmodR.Data') for documentation
    loading from cache
    see ?RNAmodR.Data and browseVignettes('RNAmodR.Data') for documentation
    loading from cache
    see ?RNAmodR.Data and browseVignettes('RNAmodR.Data') for documentation
    loading from cache
    see ?RNAmodR.Data and browseVignettes('RNAmodR.Data') for documentation
    ...
    --- failed re-building ‘RNAmodR.Rmd’
    
    --- re-building ‘RNAmodR.creation.Rmd’ using rmarkdown
    --- finished re-building ‘RNAmodR.creation.Rmd’
    
    SUMMARY: processing the following file failed:
      ‘RNAmodR.Rmd’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      'BiocGenerics:::replaceSlots' 'Biostrings:::XString'
      'IRanges:::.valid.CompressedList' 'IRanges:::coerceToCompressedList'
      'S4Vectors:::.valid.DataFrame' 'S4Vectors:::labeledLine'
      'S4Vectors:::make_zero_col_DataFrame'
      'S4Vectors:::prepare_objects_to_bind'
      See the note in ?`:::` about the use of this operator.
    There are ::: calls to the package's namespace in its code. A package
      almost never needs to use ::: for its own objects:
      '.get_inosine_score'
    ```

# RNAseqQC

<details>

* Version: 0.1.1
* GitHub: NA
* Source code: https://github.com/cran/RNAseqQC
* Date/Publication: 2022-03-18 16:40:02 UTC
* Number of recursive dependencies: 167

Run `revdep_details(, "RNAseqQC")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.7Mb
      sub-directories of 1Mb or more:
        data   3.4Mb
        doc    2.2Mb
    ```

# RnBeads

<details>

* Version: 2.12.2
>>>>>>> develop
* GitHub: NA
* Source code: https://github.com/cran/RnBeads
* Date/Publication: 2021-11-05
* Number of recursive dependencies: 276

Run `revdep_details(, "RnBeads")` for more info

</details>

## In both

*   checking running R code from vignettes ...
    ```
      ‘RnBeads.Rnw’... failed
      ‘RnBeads_Annotations.Rnw’... OK
     ERROR
    Errors in running code in vignettes:
    when running code in ‘RnBeads.Rnw’
      ...
    cg24378253              0            0             0
    cg04794393              0            0             0
    cg17840536              0            0             0
    cg19758750              0            0             0
    
    > rnb.plot.control.boxplot(rnb.set.example)
    
      When sourcing ‘RnBeads.R’:
    Error: could not find function "grid.draw"
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘RefFreeEWAS’
    
    Depends: includes the non-default packages:
      'BiocGenerics', 'S4Vectors', 'GenomicRanges', 'MASS', 'cluster',
      'ff', 'fields', 'ggplot2', 'gplots', 'gridExtra', 'limma',
      'matrixStats', 'illuminaio', 'methylumi', 'plyr'
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  9.0Mb
      sub-directories of 1Mb or more:
        R     3.1Mb
        doc   3.7Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Package listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘qvalue’
    A package should be listed in only one of these fields.
    ```

*   checking whether the namespace can be loaded with stated dependencies ... NOTE
    ```
    Warning: no function found corresponding to methods exports from ‘RnBeads’ for: ‘samples’
    
    A namespace must be able to be loaded with just the base namespace
    loaded: otherwise if the namespace gets loaded by a saved object, the
    session will be unable to start.
    
    Probably some imports need to be declared in the NAMESPACE file.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      'Gviz:::.getBMFeatureMap' 'doParallel:::.options'
      'grDevices:::.smoothScatterCalcDensity'
      'minfi:::.default.450k.annotation' 'minfi:::.extractFromRGSet450k'
      'minfi:::.normalizeFunnorm450k'
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    .stopImplicitCluster: no visible global function definition for
      ‘stopCluster’
    MethyLumiSet2RnBeadSet: no visible global function definition for
      ‘phenoData’
    MethyLumiSet2RnBeadSet: no visible global function definition for
      ‘assayDataElementNames’
    MethyLumiSet2RnBeadSet: no visible global function definition for
      ‘featureNames’
    MethyLumiSet2RnBeadSet: no visible global function definition for
      ‘varLabels’
    ...
      preprocessSWAN pvalues qvalue readMethylome readSNPTable refText
      reg.type region.size registerDoParallel relative.coord removeSNPs
      report runLOLA samples segmentPMDs segmentUMRsLMRs seqlengths
      seqlevels<- setNames sigCategories sigset.l sites2ignore size
      solve.QP stopCluster sva target texthere tsne type types
      ucscTableQuery universeCounts useMart v var.diff varFit varLabels x
      x2 xmlValue y y2 yint
    Consider adding
      importFrom("stats", "setNames")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error(s) in re-building vignettes:
    --- re-building ‘RnBeads.Rnw’ using Sweave
    
    Error: processing vignette 'RnBeads.Rnw' failed with diagnostics:
     chunk 20 
    Error in grid.draw(grb) : could not find function "grid.draw"
    
    --- failed re-building ‘RnBeads.Rnw’
    
    --- re-building ‘RnBeads_Annotations.Rnw’ using Sweave
    ...
    l.10 \usepackage
                    [english,american]{babel}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    --- failed re-building ‘RnBeads_Annotations.Rnw’
    
    SUMMARY: processing the following files failed:
      ‘RnBeads.Rnw’ ‘RnBeads_Annotations.Rnw’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

# robustlm

<details>

* Version: 0.1.0
* GitHub: NA
* Source code: https://github.com/cran/robustlm
* Date/Publication: 2021-03-22 15:40:02 UTC
* Number of recursive dependencies: 25

Run `revdep_details(, "robustlm")` for more info

</details>

## In both

*   checking LazyData ... NOTE
    ```
      'LazyData' is specified without a 'data' directory
    ```

# rtrend

<details>

* Version: 0.1.3
* GitHub: https://github.com/rpkgs/rtrend
* Source code: https://github.com/cran/rtrend
* Date/Publication: 2022-04-03 19:20:02 UTC
* Number of recursive dependencies: 63

Run `revdep_details(, "rtrend")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘ggplot2’ ‘plyr’ ‘terra’
      All declared Imports should be used.
    ```

# scDataviz

<details>

* Version: 1.4.0
* GitHub: https://github.com/kevinblighe/scDataviz
* Source code: https://github.com/cran/scDataviz
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 186

Run `revdep_details(, "scDataviz")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.5Mb
      sub-directories of 1Mb or more:
        doc   5.6Mb
    ```

# scDHA

<details>

* Version: 1.1.2
* GitHub: https://github.com/duct317/scDHA
* Source code: https://github.com/cran/scDHA
* Date/Publication: 2021-09-15 20:20:02 UTC
* Number of recursive dependencies: 66

Run `revdep_details(, "scDHA")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.8Mb
      sub-directories of 1Mb or more:
        data   3.2Mb
        libs   3.4Mb
    ```

<<<<<<< HEAD
# SCFA

<details>

* Version: 1.4.0
* GitHub: https://github.com/duct317/SCFA
* Source code: https://github.com/cran/SCFA
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 72

Run `revdep_details(, "SCFA")` for more info

</details>

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
      ...
    --- re-building ‘Example.Rmd’ using rmarkdown
    Error: processing vignette 'Example.Rmd' failed with diagnostics:
    there is no package called ‘BiocStyle’
    --- failed re-building ‘Example.Rmd’
    
    SUMMARY: processing the following file failed:
      ‘Example.Rmd’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

||||||| cc87037
=======
# SCFA

<details>

* Version: 1.4.0
* GitHub: https://github.com/duct317/SCFA
* Source code: https://github.com/cran/SCFA
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 73

Run `revdep_details(, "SCFA")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘SCFA-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: SCFA
    > ### Title: SCFA
    > ### Aliases: SCFA
    > 
    > ### ** Examples
    > 
    > #Load example data (GBM dataset)
    ...
    > #Survival information
    > survival <- GBM$survival
    > library(survival)
    > #Generating subtyping result
    > set.seed(1)
    > subtype <- SCFA(dataList, seed = 1, ncores = 2L)
    Error in socketConnection(port = port, server = TRUE, blocking = TRUE,  : 
      cannot open the connection
    Calls: SCFA ... generating.latent -> suppressMessages -> withCallingHandlers
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
      ...
    --- re-building ‘Example.Rmd’ using rmarkdown
    Error: processing vignette 'Example.Rmd' failed with diagnostics:
    there is no package called ‘BiocStyle’
    --- failed re-building ‘Example.Rmd’
    
    SUMMARY: processing the following file failed:
      ‘Example.Rmd’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

>>>>>>> develop
# scmap

<details>

* Version: 1.16.0
* GitHub: https://github.com/hemberg-lab/scmap
* Source code: https://github.com/cran/scmap
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 78

Run `revdep_details(, "scmap")` for more info

</details>

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
      ...
    --- re-building ‘scmap.Rmd’ using rmarkdown
    Error: processing vignette 'scmap.Rmd' failed with diagnostics:
    there is no package called ‘BiocStyle’
    --- failed re-building ‘scmap.Rmd’
    
    SUMMARY: processing the following file failed:
      ‘scmap.Rmd’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.0Mb
      sub-directories of 1Mb or more:
        data   2.6Mb
        doc    1.1Mb
        libs   2.0Mb
    ```

*   checking top-level files ... NOTE
    ```
    File
      LICENSE
    is not mentioned in the DESCRIPTION file.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘Biobase’
      All declared Imports should be used.
    ```

# scone

<details>

* Version: 1.18.0
* GitHub: https://github.com/YosefLab/scone
* Source code: https://github.com/cran/scone
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 251

Run `revdep_details(, "scone")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    sconeReport: no visible global function definition for
      ‘visNetworkOutput’
    sconeReport: no visible global function definition for ‘plotlyOutput’
    sconeReport : server: no visible global function definition for
      ‘renderVisNetwork’
    sconeReport : server: no visible global function definition for ‘%>%’
    sconeReport : server: no visible global function definition for
      ‘visNetwork’
    sconeReport : server: no visible global function definition for
      ‘visHierarchicalLayout’
    ...
    sconeReport : server: no visible global function definition for
      ‘geom_point’
    sconeReport : server: no visible global function definition for
      ‘guides’
    Undefined global functions or variables:
      %>% aes coord_cartesian element_blank geom_bar geom_point geom_violin
      ggplot ggplotly guides labs plot_ly plotlyOutput renderVisNetwork
      scale_fill_manual theme visEdges visGroups visHierarchicalLayout
      visLegend visNetwork visNetworkOutput visNetworkProxy visOptions
      visSelectNodes ylim
    ```

# scp

<details>

* Version: 1.4.0
* GitHub: https://github.com/UCLouvain-CBIO/scp
* Source code: https://github.com/cran/scp
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 169

Run `revdep_details(, "scp")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.6Mb
      sub-directories of 1Mb or more:
        doc   5.1Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘rlang’
      All declared Imports should be used.
    Unexported objects imported by ':::' calls:
      ‘MultiAssayExperiment:::.sampleMapFromData’ ‘QFeatures:::.get_Hits’
      See the note in ?`:::` about the use of this operator.
    ```

# scPCA

<details>

* Version: 1.8.0
* GitHub: https://github.com/PhilBoileau/scPCA
* Source code: https://github.com/cran/scPCA
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 174

Run `revdep_details(, "scPCA")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DelayedArray’ ‘matrixStats’
      All declared Imports should be used.
    ```

# sctransform

<details>

* Version: 0.3.3
* GitHub: https://github.com/satijalab/sctransform
* Source code: https://github.com/cran/sctransform
* Date/Publication: 2022-01-13 08:20:02 UTC
* Number of recursive dependencies: 69

Run `revdep_details(, "sctransform")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package which this enhances but not available for checking: ‘glmGamPoi’
    ```

# seagull

<details>

* Version: 1.1.0
* GitHub: https://github.com/jklosa/seagull
* Source code: https://github.com/cran/seagull
* Date/Publication: 2021-04-20 12:10:02 UTC
* Number of recursive dependencies: 50

Run `revdep_details(, "seagull")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
<<<<<<< HEAD
      installed size is 10.7Mb
||||||| cc87037
      installed size is 11.0Mb
=======
      installed size is 10.6Mb
>>>>>>> develop
      sub-directories of 1Mb or more:
<<<<<<< HEAD
        libs  10.1Mb
    ```

||||||| cc87037
        libs  10.1Mb
    ```

# seeds

<details>

* Version: 0.9.1
* GitHub: https://github.com/Newmi1988/seeds
* Source code: https://github.com/cran/seeds
* Date/Publication: 2020-07-14 00:00:02 UTC
* Number of recursive dependencies: 87

Run `revdep_details(, "seeds")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘rsbml’
    ```

=======
        libs  10.0Mb
    ```

>>>>>>> develop
# sesame

<details>

<<<<<<< HEAD
* Version: 1.12.7
||||||| cc87037
* Version: 1.10.4
=======
* Version: 1.12.9
>>>>>>> develop
* GitHub: https://github.com/zwdzwd/sesame
* Source code: https://github.com/cran/sesame
<<<<<<< HEAD
* Date/Publication: 2021-12-14
* Number of recursive dependencies: 201
||||||| cc87037
* Date/Publication: 2021-06-06
* Number of recursive dependencies: 189
=======
* Date/Publication: 2022-02-08
* Number of recursive dependencies: 201
>>>>>>> develop

Run `revdep_details(, "sesame")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘sesame-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: qualityRank
    > ### Title: This function looks at public data of similar nature e.g.,
    > ###   tissue, FFPE vs non-FFPE, etc to evaluate the quality of the target
    > ###   data quality
    > ### Aliases: qualityRank
    > 
    > ### ** Examples
    ...
    [1] TRUE
    > sdf <- sesameDataGet('EPIC.1.SigDF')
    > ranks <- qualityRank(sdf)
    Warning: file ‘533eb2e2fc4_3720’ has magic number '<h1>I'
      Use of save versions prior to 2 is deprecated
    Error: failed to load resource
      name: EH3684
      title: detection.stats
      reason: error in evaluating the argument 'x' in selecting a method for function 'get': bad restore file magic number (file may be corrupted) -- no data loaded
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
    --- re-building ‘QC.Rmd’ using rmarkdown
    Loading required package: sesameData
    Loading required package: ExperimentHub
    Loading required package: BiocGenerics
    
    Attaching package: 'BiocGenerics'
    
    The following objects are masked from 'package:stats':
    
    ...
    Quitting from lines 413-415 (sesame.Rmd) 
    Error: processing vignette 'sesame.Rmd' failed with diagnostics:
    cannot load annotation package IlluminaHumanMethylation450kanno.ilmn12.hg19
    --- failed re-building ‘sesame.Rmd’
    
    SUMMARY: processing the following files failed:
      ‘QC.Rmd’ ‘sesame.Rmd’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
<<<<<<< HEAD
      installed size is 44.0Mb
||||||| cc87037
      installed size is 38.1Mb
=======
      installed size is 45.1Mb
>>>>>>> develop
      sub-directories of 1Mb or more:
<<<<<<< HEAD
        doc  43.5Mb
||||||| cc87037
        doc  36.5Mb
=======
        doc  44.6Mb
>>>>>>> develop
    ```

# Seurat

<details>

* Version: 4.1.0
* GitHub: https://github.com/satijalab/seurat
* Source code: https://github.com/cran/Seurat
* Date/Publication: 2022-01-14 18:32:42 UTC
* Number of recursive dependencies: 254

Run `revdep_details(, "Seurat")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 12.0Mb
      sub-directories of 1Mb or more:
        R      1.3Mb
        libs  10.1Mb
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘Signac’
    ```

# SGP

<details>

* Version: 1.9-5.0
* GitHub: https://github.com/CenterForAssessment/SGP
* Source code: https://github.com/cran/SGP
* Date/Publication: 2020-01-30 22:40:09 UTC
* Number of recursive dependencies: 111

Run `revdep_details(, "SGP")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.7Mb
      sub-directories of 1Mb or more:
        R     1.3Mb
        doc   3.4Mb
    ```

# shinyMethyl

<details>

* Version: 1.30.0
* GitHub: NA
* Source code: https://github.com/cran/shinyMethyl
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 159

Run `revdep_details(, "shinyMethyl")` for more info

</details>

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
    --- re-building ‘shinyMethyl.Rnw’ using knitr
    Loading required package: minfi
    Loading required package: BiocGenerics
    
    Attaching package: 'BiocGenerics'
    
    The following objects are masked from 'package:stats':
    
        IQR, mad, sd, var, xtabs
    ...
    l.189 \RequirePackage
                         {parnotes}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    --- failed re-building ‘shinyMethyl.Rnw’
    
    SUMMARY: processing the following file failed:
      ‘shinyMethyl.Rnw’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    addHoverDensity: no visible global function definition for ‘lines’
    addHoverPoints: no visible global function definition for ‘points’
    addHoverQC: no visible global function definition for ‘points’
    densitiesPlot: no visible global function definition for ‘lines’
    plotDesign450k: no visible global function definition for ‘par’
    plotDiscrepancyGenders: no visible global function definition for
      ‘legend’
    plotInternalControls: no visible global function definition for ‘grid’
    plotInternalControls: no visible global function definition for
      ‘abline’
    ...
      abline complete.cases current.control.type current.density.type
      current.probe.type genderCutoff grid legend lines lm
      mouse.click.indices palette par points prcomp rect write.csv
    Consider adding
      importFrom("grDevices", "palette")
      importFrom("graphics", "abline", "grid", "legend", "lines", "par",
                 "points", "rect")
      importFrom("stats", "complete.cases", "lm", "prcomp")
      importFrom("utils", "write.csv")
    to your NAMESPACE file.
    ```

# SIAMCAT

<details>

* Version: 1.14.0
* GitHub: NA
* Source code: https://github.com/cran/SIAMCAT
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 186

Run `revdep_details(, "SIAMCAT")` for more info

</details>

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 23.7Mb
      sub-directories of 1Mb or more:
        doc      18.4Mb
        extdata   2.1Mb
        help      1.9Mb
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘randomForest’
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 10 marked UTF-8 strings
    ```

# SICtools

<details>

* Version: 1.24.0
* GitHub: NA
* Source code: https://github.com/cran/SICtools
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 38

Run `revdep_details(, "SICtools")` for more info

</details>

## In both

*   checking whether package ‘SICtools’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      errmod.c:79:8: warning: self-comparison always evaluates to false [-Wtautological-compare]
      md5.c:155:26: warning: argument to ‘sizeof’ in ‘memset’ call is the same expression as the destination; did you mean to dereference it? [-Wsizeof-pointer-memaccess]
      Warning: replacing previous import ‘plyr::count’ by ‘matrixStats::count’ when loading ‘SICtools’
    See ‘/c4/home/henrik/repositories/matrixStats/revdep/checks/SICtools/new/SICtools.Rcheck/00install.out’ for details.
    ```

*   checking examples ... WARNING
    ```
    Found the following significant warnings:
    
      Warning: 'applyPileups' is deprecated.
    Deprecated functions may be defunct as soon as of the next release of
    R.
    See ?Deprecated.
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      'Rsamtools', 'doParallel', 'Biostrings', 'stringr', 'matrixStats',
      'plyr', 'GenomicRanges', 'IRanges'
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .BBSoptions
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    .indelDiffFunc: no visible global function definition for ‘fisher.test’
    .indelDiffFunc: no visible global function definition for ‘dist’
    indelDiff: no visible global function definition for ‘read.delim’
    snpDiff : calcInfoRange : <anonymous>: no visible global function
      definition for ‘fisher.test’
    snpDiff : calcInfoRange : <anonymous>: no visible global function
      definition for ‘dist’
    Undefined global functions or variables:
      dist fisher.test read.delim
    Consider adding
      importFrom("stats", "dist", "fisher.test")
      importFrom("utils", "read.delim")
    to your NAMESPACE file.
    ```

# sigminer

<details>

* Version: 2.1.3
* GitHub: https://github.com/ShixiangWang/sigminer
* Source code: https://github.com/cran/sigminer
* Date/Publication: 2022-03-10 11:00:02 UTC
* Number of recursive dependencies: 204

Run `revdep_details(, "sigminer")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.4Mb
      sub-directories of 1Mb or more:
        extdata   2.6Mb
        libs      1.2Mb
    ```

# sindyr

<details>

* Version: 0.2.3
* GitHub: NA
* Source code: https://github.com/cran/sindyr
* Date/Publication: 2020-06-09 05:20:03 UTC
* Number of recursive dependencies: 100

Run `revdep_details(, "sindyr")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘crqa’ ‘plot3D’
      All declared Imports should be used.
    ```

# Single.mTEC.Transcriptomes

<details>

* Version: 1.22.0
* GitHub: NA
* Source code: https://github.com/cran/Single.mTEC.Transcriptomes
* Date/Publication: 2021-10-30
* Number of recursive dependencies: 167

Run `revdep_details(, "Single.mTEC.Transcriptomes")` for more info

</details>

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
    --- re-building ‘mTECs.Rnw’ using knitr
    Loading required package: S4Vectors
    Loading required package: stats4
    Loading required package: BiocGenerics
    
    Attaching package: 'BiocGenerics'
    
    The following objects are masked from 'package:stats':
    
    ...
    l.189 \RequirePackage
                         {parnotes}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    --- failed re-building ‘mTECs.Rnw’
    
    SUMMARY: processing the following file failed:
      ‘mTECs.Rnw’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 904.4Mb
      sub-directories of 1Mb or more:
        data  895.1Mb
        doc     9.3Mb
    ```

# singleCellTK

<details>

* Version: 2.4.0
* GitHub: https://github.com/compbiomed/singleCellTK
* Source code: https://github.com/cran/singleCellTK
<<<<<<< HEAD
* Date/Publication: 2021-10-27
* Number of recursive dependencies: 379
||||||| cc87037
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 372
=======
* Date/Publication: 2021-10-27
* Number of recursive dependencies: 380
>>>>>>> develop

Run `revdep_details(, "singleCellTK")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.6Mb
      sub-directories of 1Mb or more:
        extdata   1.5Mb
<<<<<<< HEAD
        shiny     2.6Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      'AnnotationDbi' 'RColorBrewer'
      All declared Imports should be used.
||||||| cc87037
=======
        shiny     2.6Mb
>>>>>>> develop
    ```

<<<<<<< HEAD
# singscore

<details>

* Version: 1.14.0
||||||| cc87037
# singscore

<details>

* Version: 1.12.0
=======
*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      'AnnotationDbi' 'RColorBrewer'
      All declared Imports should be used.
    ```

# singscore

<details>

* Version: 1.14.0
>>>>>>> develop
* GitHub: https://github.com/DavisLaboratory/singscore
* Source code: https://github.com/cran/singscore
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 129

Run `revdep_details(, "singscore")` for more info

</details>

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
      ...
    --- re-building ‘singscore.Rmd’ using rmarkdown
    Error: processing vignette 'singscore.Rmd' failed with diagnostics:
    there is no package called ‘BiocStyle’
    --- failed re-building ‘singscore.Rmd’
    
    SUMMARY: processing the following file failed:
      ‘singscore.Rmd’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.2Mb
      sub-directories of 1Mb or more:
        R      1.2Mb
        data   1.9Mb
        doc    1.8Mb
    ```

# SkeweDF

<details>

* Version: 0.1.0
* GitHub: NA
* Source code: https://github.com/cran/SkeweDF
* Date/Publication: 2021-01-16 09:40:05 UTC
* Number of recursive dependencies: 29

Run `revdep_details(, "SkeweDF")` for more info

</details>

## In both

*   checking LazyData ... NOTE
    ```
      'LazyData' is specified without a 'data' directory
    ```

<<<<<<< HEAD
# slasso

<details>

* Version: 1.0.0
* GitHub: https://github.com/unina-sfere/slasso
* Source code: https://github.com/cran/slasso
* Date/Publication: 2021-10-15 07:40:02 UTC
* Number of recursive dependencies: 94

Run `revdep_details(, "slasso")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘RcppArmadillo’
      All declared Imports should be used.
    ```

# slingshot
||||||| cc87037
# slingshot
=======
# slasso
>>>>>>> develop

<details>

<<<<<<< HEAD
* Version: 2.2.0
* GitHub: https://github.com/kstreet13/slingshot
* Source code: https://github.com/cran/slingshot
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 186
||||||| cc87037
* Version: 2.0.0
* GitHub: https://github.com/kstreet13/slingshot
* Source code: https://github.com/cran/slingshot
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 183
=======
* Version: 1.0.0
* GitHub: https://github.com/unina-sfere/slasso
* Source code: https://github.com/cran/slasso
* Date/Publication: 2021-10-15 07:40:02 UTC
* Number of recursive dependencies: 94
>>>>>>> develop

Run `revdep_details(, "slasso")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘RcppArmadillo’
      All declared Imports should be used.
    ```

# sparrow

<details>

* Version: 1.0.3
* GitHub: https://github.com/lianos/sparrow
* Source code: https://github.com/cran/sparrow
* Date/Publication: 2022-03-24
* Number of recursive dependencies: 207

Run `revdep_details(, "sparrow")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
<<<<<<< HEAD
      Running ‘testthat.R’
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 50 lines of output:
      <packageNotFoundError/error/condition>
      Error in `loadNamespace(x)`: there is no package called 'DelayedMatrixStats'
      Backtrace:
           ▆
        1. ├─slingshot::slingshot(rd, cl) at test_slingshot.R:733:4
        2. ├─slingshot::slingshot(rd, cl)
    ...
       15. │                 └─TrajectoryUtils FUN(x, clusters)
       16. │                   └─TrajectoryUtils:::.rowstats_w(...)
       17. └─base::loadNamespace(x)
       18.   └─base::withRestarts(stop(cond), retry_loadNamespace = function() NULL)
       19.     └─base withOneRestart(expr, restarts[[1L]])
       20.       └─base doWithOneRestart(return(expr), restart)
      
      [ FAIL 13 | WARN 2 | SKIP 0 | PASS 0 ]
      Error: Test failures
      Execution halted
||||||| cc87037
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 50 lines of output:
       16.       └─base:::doWithOneRestart(return(expr), restart)
      ── Error (test_slingshot.R:733:5): branchID functions work as expected ─────────
      Error: there is no package called 'DelayedMatrixStats'
      Backtrace:
           █
        1. ├─slingshot::slingshot(rd, cl) test_slingshot.R:733:4
        2. ├─slingshot::slingshot(rd, cl)
    ...
       15. │                 └─TrajectoryUtils:::FUN(x, clusters)
       16. │                   └─TrajectoryUtils:::.rowstats_w(...)
       17. └─base::loadNamespace(x)
       18.   └─base::withRestarts(stop(cond), retry_loadNamespace = function() NULL)
       19.     └─base:::withOneRestart(expr, restarts[[1L]])
       20.       └─base:::doWithOneRestart(return(expr), restart)
      
      [ FAIL 13 | WARN 2 | SKIP 0 | PASS 0 ]
      Error: Test failures
      Execution halted
=======
      installed size is  5.6Mb
      sub-directories of 1Mb or more:
        R         1.1Mb
        doc       2.0Mb
        extdata   2.1Mb
>>>>>>> develop
    ```

*   checking R code for possible problems ... NOTE
    ```
<<<<<<< HEAD
    Error(s) in re-building vignettes:
    --- re-building ‘conditionsVignette.Rmd’ using rmarkdown
    The magick package is required to crop "/c4/home/henrik/repositories/matrixStats/revdep/checks/slingshot/new/slingshot.Rcheck/vign_test/slingshot/vignettes/conditionsVignette_files/figure-html/unnamed-chunk-1-1.png" but not available.
    Quitting from lines 68-69 (conditionsVignette.Rmd) 
    Error: processing vignette 'conditionsVignette.Rmd' failed with diagnostics:
    there is no package called 'DelayedMatrixStats'
    --- failed re-building ‘conditionsVignette.Rmd’
    
    --- re-building ‘vignette.Rmd’ using rmarkdown
    Loading required package: SummarizedExperiment
    ...
    Quitting from lines 276-277 (vignette.Rmd) 
    Error: processing vignette 'vignette.Rmd' failed with diagnostics:
    there is no package called 'DelayedMatrixStats'
    --- failed re-building ‘vignette.Rmd’
    
    SUMMARY: processing the following files failed:
      ‘conditionsVignette.Rmd’ ‘vignette.Rmd’
    
    Error: Vignette re-building failed.
    Execution halted
||||||| cc87037
    Error(s) in re-building vignettes:
    --- re-building ‘conditionsVignette.Rmd’ using rmarkdown
    The magick package is required to crop "/scratch/henrik/revdepcheck.extras/matrixStats/revdep/checks/slingshot/new/slingshot.Rcheck/vign_test/slingshot/vignettes/conditionsVignette_files/figure-html/unnamed-chunk-1-1.png" but not available.
    Quitting from lines 68-69 (conditionsVignette.Rmd) 
    Error: processing vignette 'conditionsVignette.Rmd' failed with diagnostics:
    there is no package called 'DelayedMatrixStats'
    --- failed re-building ‘conditionsVignette.Rmd’
    
    --- re-building ‘vignette.Rmd’ using rmarkdown
    Loading required package: SummarizedExperiment
    ...
    Quitting from lines 276-277 (vignette.Rmd) 
    Error: processing vignette 'vignette.Rmd' failed with diagnostics:
    there is no package called 'DelayedMatrixStats'
    --- failed re-building ‘vignette.Rmd’
    
    SUMMARY: processing the following files failed:
      ‘conditionsVignette.Rmd’ ‘vignette.Rmd’
    
    Error: Vignette re-building failed.
    Execution halted
=======
    geneSetsStats: no visible binding for global variable ‘direction’
    Undefined global functions or variables:
      direction
>>>>>>> develop
    ```

# sparrow

<details>

* Version: 1.0.1
* GitHub: https://github.com/lianos/sparrow
* Source code: https://github.com/cran/sparrow
* Date/Publication: 2021-11-04
* Number of recursive dependencies: 207

Run `revdep_details(, "sparrow")` for more info

</details>

## In both

*   checking tests ...
    ```
      Running ‘testthat.R’
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 50 lines of output:
      
      The following objects are masked from 'package:data.table':
      
          between, first, last
      
      The following object is masked from 'package:testthat':
    ...
      Component 1: Component 2: Lengths (2, 1) differ (string compare on first 1)
      Component 1: Component 2: 1 string mismatch
      Component 1: Component 3: Lengths (0, 1) differ (string compare on first 0)
      Component 1: Component 4: Length mismatch: comparison on first 0 components
      Component 1: Component 5: target is logical, current is character
      ...
      
      [ FAIL 1 | WARN 0 | SKIP 1 | PASS 1485 ]
      Error: Test failures
      Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.6Mb
      sub-directories of 1Mb or more:
        R         1.1Mb
        doc       2.0Mb
        extdata   2.1Mb
    ```

# sparseMatrixStats

<details>

* Version: 1.6.0
* GitHub: https://github.com/const-ae/sparseMatrixStats
* Source code: https://github.com/cran/sparseMatrixStats
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 57

Run `revdep_details(, "sparseMatrixStats")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.1Mb
      sub-directories of 1Mb or more:
        libs   3.5Mb
    ```

# spathial

<details>

* Version: 0.1.2
* GitHub: NA
* Source code: https://github.com/cran/spathial
* Date/Publication: 2020-04-10 18:20:02 UTC
* Number of recursive dependencies: 34

Run `revdep_details(, "spathial")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘irlba’ ‘knitr’ ‘rmarkdown’
      All declared Imports should be used.
    ```

*   checking LazyData ... NOTE
    ```
      'LazyData' is specified without a 'data' directory
    ```

<<<<<<< HEAD
# spatzie

<details>

* Version: 1.0.0
* GitHub: NA
* Source code: https://github.com/cran/spatzie
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 183

Run `revdep_details(, "spatzie")` for more info

</details>

## In both

*   checking files in ‘vignettes’ ... NOTE
    ```
    The following directory looks like a leftover from 'knitr':
      ‘figure’
    Please remove from your package.
    ```

||||||| cc87037
=======
# spatzie

<details>

* Version: 1.0.1
* GitHub: NA
* Source code: https://github.com/cran/spatzie
* Date/Publication: 2022-02-03
* Number of recursive dependencies: 183

Run `revdep_details(, "spatzie")` for more info

</details>

## In both

*   checking files in ‘vignettes’ ... NOTE
    ```
    The following directory looks like a leftover from 'knitr':
      ‘figure’
    Please remove from your package.
    ```

>>>>>>> develop
# splatter

<details>

* Version: 1.18.2
* GitHub: https://github.com/Oshlack/splatter
* Source code: https://github.com/cran/splatter
* Date/Publication: 2022-01-11
* Number of recursive dependencies: 238

Run `revdep_details(, "splatter")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  9.9Mb
      sub-directories of 1Mb or more:
        doc   9.0Mb
    ```

# spqn

<details>

* Version: 1.6.0
* GitHub: https://github.com/hansenlab/spqn
* Source code: https://github.com/cran/spqn
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 72

Run `revdep_details(, "spqn")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Package in Depends field not imported from: ‘BiocGenerics’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

# stapler

<details>

* Version: 0.7.1
* GitHub: https://github.com/muschellij2/stapler
* Source code: https://github.com/cran/stapler
* Date/Publication: 2020-01-09 18:50:06 UTC
* Number of recursive dependencies: 63

Run `revdep_details(, "stapler")` for more info

</details>

## In both

*   checking LazyData ... NOTE
    ```
      'LazyData' is specified without a 'data' directory
    ```

# stm

<details>

* Version: 1.3.6
* GitHub: https://github.com/bstewart/stm
* Source code: https://github.com/cran/stm
* Date/Publication: 2020-09-18 13:50:02 UTC
* Number of recursive dependencies: 85

Run `revdep_details(, "stm")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
<<<<<<< HEAD
      installed size is  5.5Mb
||||||| cc87037
      installed size is  7.7Mb
=======
      installed size is  5.6Mb
>>>>>>> develop
      sub-directories of 1Mb or more:
<<<<<<< HEAD
        data   1.7Mb
        libs   2.7Mb
||||||| cc87037
        data   3.5Mb
        libs   2.7Mb
=======
        data   1.7Mb
        libs   2.8Mb
>>>>>>> develop
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error(s) in re-building vignettes:
      ...
    --- re-building ‘stmVignette.Rnw’ using Sweave
    stm v1.3.6 successfully loaded. See ?stm for help. 
     Papers, resources, and other materials at structuraltopicmodel.com
    Error: processing vignette 'stmVignette.Rnw' failed with diagnostics:
    Running 'texi2dvi' on 'stmVignette.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ulem.sty' not found.
    
    ...
    l.57 ^^M
            
    !  ==> Fatal error occurred, no output PDF file produced!
    --- failed re-building 'stmVignette.Rnw'
    
    SUMMARY: processing the following file failed:
      'stmVignette.Rnw'
    
    Error: Vignette re-building failed.
    Execution halted
    ```

# STROMA4

<details>

* Version: 1.18.0
* GitHub: NA
* Source code: https://github.com/cran/STROMA4
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 12

Run `revdep_details(, "STROMA4")` for more info

</details>

## Newly fixed

*   checking examples ... ERROR
    ```
    Running examples in ‘STROMA4-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: assign.properties
    > ### Title: Function to assign properties to an expression matrix
    > ### Aliases: assign.properties
    > 
    > ### ** Examples
    > 
    > library(breastCancerMAINZ)
    ...
    --Assigning properties to expression data--
    --There are duplicated genes. Using most variable to collapse--
    ----206 out of 297 total genes matching for D.stroma.property----
    ----1064 out of 1105 total genes matching for MSL.property----
    Warning in socketConnection(port = port, server = TRUE, blocking = TRUE,  :
      port 11412 cannot be opened
    Error in socketConnection(port = port, server = TRUE, blocking = TRUE,  : 
      cannot open the connection
    Calls: assign.properties ... <Anonymous> -> <Anonymous> -> newSOCKnode -> socketConnection
    Execution halted
    ```

## In both

*   checking running R code from vignettes ...
    ```
      ‘STROMA4-vignette.Rnw’... failed
     ERROR
    Errors in running code in vignettes:
    when running code in ‘STROMA4-vignette.Rnw’
      ...
    
    > if (!requireNamespace("BiocManager", quietly = TRUE)) install.packages("BiocManager")
    Installing package into ‘/c4/home/henrik/repositories/matrixStats/revdep/checks/STROMA4/new/STROMA4.Rcheck’
    (as ‘lib’ is unspecified)
    
      When sourcing ‘STROMA4-vignette.R’:
    Error: trying to use CRAN without setting a mirror
    Execution halted
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    'library' or 'require' call not declared from: ‘BiocManager’
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error(s) in re-building vignettes:
      ...
    --- re-building ‘STROMA4-vignette.Rnw’ using Sweave
    Installing package into ‘/c4/home/henrik/repositories/matrixStats/revdep/checks/STROMA4/new/STROMA4.Rcheck’
    (as ‘lib’ is unspecified)
    
    Error: processing vignette 'STROMA4-vignette.Rnw' failed with diagnostics:
     chunk 1 
    Error in contrib.url(repos, type) : 
      trying to use CRAN without setting a mirror
    
    --- failed re-building ‘STROMA4-vignette.Rnw’
    
    SUMMARY: processing the following file failed:
      ‘STROMA4-vignette.Rnw’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

# summarytools

<details>

* Version: 1.0.0
* GitHub: https://github.com/dcomtois/summarytools
* Source code: https://github.com/cran/summarytools
* Date/Publication: 2021-07-28 05:00:02 UTC
* Number of recursive dependencies: 79

Run `revdep_details(, "summarytools")` for more info

</details>

## In both

*   checking whether package ‘summarytools’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: no DISPLAY variable so Tk is not available
    See ‘/c4/home/henrik/repositories/matrixStats/revdep/checks/summarytools/new/summarytools.Rcheck/00install.out’ for details.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 78 marked UTF-8 strings
    ```

# SuperPCA

<details>

* Version: 0.4.0
* GitHub: NA
* Source code: https://github.com/cran/SuperPCA
* Date/Publication: 2021-07-26 12:30:07 UTC
* Number of recursive dependencies: 37

Run `revdep_details(, "SuperPCA")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘R.matlab’ ‘fBasics’ ‘spls’
      All declared Imports should be used.
    ```

# susieR

<details>

* Version: 0.11.92
* GitHub: https://github.com/stephenslab/susieR
* Source code: https://github.com/cran/susieR
* Date/Publication: 2021-11-12 16:50:02 UTC
* Number of recursive dependencies: 78

Run `revdep_details(, "susieR")` for more info

</details>

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
    --- re-building ‘finemapping.Rmd’ using rmarkdown
    --- finished re-building ‘finemapping.Rmd’
    
    --- re-building ‘finemapping_summary_statistics.Rmd’ using rmarkdown
    --- finished re-building ‘finemapping_summary_statistics.Rmd’
    
    --- re-building ‘l0_initialization.Rmd’ using rmarkdown
    --- finished re-building ‘l0_initialization.Rmd’
    
    ...
    l.8 \usepackage
                   {xspace}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    --- failed re-building ‘trendfiltering_derivations.Rnw’
    
    SUMMARY: processing the following file failed:
      ‘trendfiltering_derivations.Rnw’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

# sva

<details>

* Version: 3.42.0
* GitHub: NA
* Source code: https://github.com/cran/sva
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 103

Run `revdep_details(, "sva")` for more info

</details>

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    Error(s) in re-building vignettes:
    --- re-building ‘sva.Rnw’ using Sweave
    Loading required package: mgcv
    Loading required package: nlme
<<<<<<< HEAD
    This is mgcv 1.8-39. For overview type 'help("mgcv-package")'.
||||||| cc87037
    This is mgcv 1.8-36. For overview type 'help("mgcv-package")'.
=======
    This is mgcv 1.8-40. For overview type 'help("mgcv-package")'.
>>>>>>> develop
    Loading required package: genefilter
    Loading required package: BiocParallel
    Loading required package: Biobase
    Loading required package: BiocGenerics
    
    ...
    l.189 \RequirePackage
                         {parnotes}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    --- failed re-building ‘sva.Rnw’
    
    SUMMARY: processing the following file failed:
      ‘sva.Rnw’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

# TCA

<details>

* Version: 1.2.1
* GitHub: https://github.com/cozygene/TCA
* Source code: https://github.com/cran/TCA
* Date/Publication: 2021-02-14 21:50:06 UTC
* Number of recursive dependencies: 66

Run `revdep_details(, "TCA")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘Matrix’ ‘quadprog’
      All declared Imports should be used.
    ```

*   checking LazyData ... NOTE
    ```
      'LazyData' is specified without a 'data' directory
    ```

# templateICAr

<details>

* Version: 0.3.1
* GitHub: https://github.com/mandymejia/templateICAr
* Source code: https://github.com/cran/templateICAr
* Date/Publication: 2022-03-09 21:10:02 UTC
* Number of recursive dependencies: 108

Run `revdep_details(, "templateICAr")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘INLA’
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘abind’ ‘ica’
      All declared Imports should be used.
    ```

# tenXplore

<details>

* Version: 1.16.0
* GitHub: NA
* Source code: https://github.com/cran/tenXplore
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 129

Run `revdep_details(, "tenXplore")` for more info

</details>

## In both

*   checking tests ...
    ```
      Running ‘test.R’
     ERROR
    Running the tests in ‘tests/test.R’ failed.
    Last 50 lines of output:
      
      Attaching package: 'Biobase'
      
      The following object is masked from 'package:MatrixGenerics':
      
          rowMedians
    ...
       1. └─ontoProc::getEFOOnto() at test_tenXplore.R:8:2
       2.   └─ontoProc:::get_onto("efoOnto")
       3.     └─AnnotationHub::AnnotationHub()
       4.       └─AnnotationHub::.Hub(...)
       5.         └─AnnotationHub:::.create_cache(...)
       6.           └─AnnotationHub:::.updateHubDB(hub_bfc, .class, url, proxy, localHub)
      
      [ FAIL 1 | WARN 0 | SKIP 0 | PASS 0 ]
      Error: Test failures
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error(s) in re-building vignettes:
      ...
    --- re-building ‘tenXplore.Rmd’ using rmarkdown
    Error: processing vignette 'tenXplore.Rmd' failed with diagnostics:
    there is no package called ‘BiocStyle’
    --- failed re-building ‘tenXplore.Rmd’
    
    SUMMARY: processing the following file failed:
      ‘tenXplore.Rmd’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

<<<<<<< HEAD
# TOAST

<details>

* Version: 1.7.1
* GitHub: https://github.com/ziyili20/TOAST
* Source code: https://github.com/cran/TOAST
* Date/Publication: 2021-05-31
* Number of recursive dependencies: 98

Run `revdep_details(, "TOAST")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘RefFreeEWAS’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

||||||| cc87037
# TOAST

<details>

* Version: 1.6.0
* GitHub: https://github.com/ziyili20/TOAST
* Source code: https://github.com/cran/TOAST
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 54

Run `revdep_details(, "TOAST")` for more info

</details>

## In both

*   checking package subdirectories ... WARNING
    ```
    Invalid citation information in ‘inst/CITATION’:
      Error in bibentry(bibtype = entry, textVersion = textVersion, header = header,     footer = footer, ...): object 'X' not found
    ```

=======
>>>>>>> develop
# topGO

<details>

* Version: 2.46.0
* GitHub: NA
* Source code: https://github.com/cran/topGO
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 57

Run `revdep_details(, "topGO")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      'BiocGenerics', 'graph', 'Biobase', 'GO.db', 'AnnotationDbi',
      'SparseM'
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls in package code:
      ‘Rgraphviz’ ‘multtest’
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    Missing object imported by a ':::' call: ‘globaltest:::globaltest’
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error(s) in re-building vignettes:
    --- re-building ‘topGO.Rnw’ using Sweave
    
    Loading required package: xtable
    Loading required package: BiocGenerics
    
    Attaching package: ‘BiocGenerics’
    
    The following objects are masked from ‘package:stats’:
    
    ...
    l.11 \usepackage
                    {caption}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    --- failed re-building ‘topGO.Rnw’
    
    SUMMARY: processing the following file failed:
      ‘topGO.Rnw’
    
    Error: Vignette re-building failed.
    Execution halted
    ```

# tradeSeq

<details>

* Version: 1.8.0
* GitHub: https://github.com/statOmics/tradeSeq
* Source code: https://github.com/cran/tradeSeq
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 177

Run `revdep_details(, "tradeSeq")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘tradeSeq-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: plotGeneCount
    > ### Title: Plot gene expression in reduced dimension.
    > ### Aliases: plotGeneCount plotGeneCount,SlingshotDataSet-method
    > ###   plotGeneCount,PseudotimeOrdering-method
    > ###   plotGeneCount,SingleCellExperiment-method
    > 
    > ### ** Examples
    ...
    
    > data(crv, package="tradeSeq")
    > data(countMatrix, package="tradeSeq")
    > rd <- slingReducedDim(crv)
    > cl <- kmeans(rd, centers = 7)$cluster
    > lin <- getLineages(rd, clusterLabels = cl, start.clus = 4)
    Error in loadNamespace(x) : 
      there is no package called ‘DelayedMatrixStats’
    Calls: getLineages ... loadNamespace -> withRestarts -> withOneRestart -> doWithOneRestart
    Execution halted
    ```

*   checking tests ...
    ```
      Running ‘testthat.R’
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 50 lines of output:
      
      Attaching package: 'Biobase'
      
      The following object is masked from 'package:MatrixGenerics':
      
          rowMedians
    ...
       4. │   └─TrajectoryUtils::rowmean(reducedDim(sds), slingClusterLabels(sds))
       5. │     └─TrajectoryUtils:::.rowstats_w(...)
       6. └─base::loadNamespace(x)
       7.   └─base::withRestarts(stop(cond), retry_loadNamespace = function() NULL)
       8.     └─base withOneRestart(expr, restarts[[1L]])
       9.       └─base doWithOneRestart(return(expr), restart)
      
      [ FAIL 1 | WARN 0 | SKIP 0 | PASS 136 ]
      Error: Test failures
      Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.4Mb
      sub-directories of 1Mb or more:
        doc   6.4Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘Biobase’ ‘igraph’
      All declared Imports should be used.
    ```

*   checking R code for possible problems ... NOTE
    ```
    .earlyDETest: no visible binding for global variable ‘X1’
    .earlyDETest: no visible binding for global variable ‘X2’
    .findKnots: no visible binding for global variable ‘t1’
    .findKnots: no visible binding for global variable ‘l1’
    .plotGeneCount: no visible binding for global variable ‘dim1’
    .plotGeneCount: no visible binding for global variable ‘dim2’
    .plotSmoothers: no visible binding for global variable ‘time’
    .plotSmoothers: no visible binding for global variable ‘gene_count’
    .plotSmoothers: no visible binding for global variable ‘lineage’
    .plotSmoothers_conditions: no visible binding for global variable
    ...
      ‘pCol’
    .plotSmoothers_sce: no visible binding for global variable ‘time’
    .plotSmoothers_sce: no visible binding for global variable ‘gene_count’
    .plotSmoothers_sce: no visible binding for global variable ‘lineage’
    .plotSmoothers_sce: no visible binding for global variable ‘pCol’
    Undefined global functions or variables:
      X1 X2 dim1 dim2 gene_count l1 lineage pCol t1 time
    Consider adding
      importFrom("stats", "time")
    to your NAMESPACE file.
    ```

# TraRe

<details>

* Version: 1.2.0
* GitHub: https://github.com/ubioinformat/TraRe
* Source code: https://github.com/cran/TraRe
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 96

Run `revdep_details(, "TraRe")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.9Mb
      sub-directories of 1Mb or more:
        doc       2.3Mb
        extdata   4.4Mb
    ```

# UPG

<details>

* Version: 0.2.2
* GitHub: NA
* Source code: https://github.com/cran/UPG
* Date/Publication: 2021-01-07 09:00:05 UTC
* Number of recursive dependencies: 48

Run `revdep_details(, "UPG")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
<<<<<<< HEAD
      installed size is  8.0Mb
||||||| cc87037
      installed size is  8.3Mb
=======
      installed size is  7.9Mb
>>>>>>> develop
      sub-directories of 1Mb or more:
        libs   7.3Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘RcppProgress’
      All declared Imports should be used.
    ```

# VanillaICE

<details>

* Version: 1.56.3
* GitHub: NA
* Source code: https://github.com/cran/VanillaICE
* Date/Publication: 2021-12-07
* Number of recursive dependencies: 75

Run `revdep_details(, "VanillaICE")` for more info

</details>

## In both

*   checking Rd files ... WARNING
    ```
    checkRd: (5) VanillaICE.Rd:0-7: Must have a \description
    ```

*   checking package dependencies ... NOTE
    ```
    Packages which this enhances but not available for checking:
      'doMC', 'doMPI', 'doSNOW', 'doParallel', 'doRedis'
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘BSgenome.Hsapiens.UCSC.hg18’ ‘MatrixGenerics’
      All declared Imports should be used.
    ```

# VaSP

<details>

* Version: 1.6.0
* GitHub: https://github.com/yuhuihui2011/VaSP
* Source code: https://github.com/cran/VaSP
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 113

Run `revdep_details(, "VaSP")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.5Mb
      sub-directories of 1Mb or more:
        doc       4.0Mb
        extdata   3.4Mb
    ```

# wateRmelon

<details>

* Version: 2.0.0
* GitHub: NA
* Source code: https://github.com/cran/wateRmelon
* Date/Publication: 2021-10-26
* Number of recursive dependencies: 192

Run `revdep_details(, "wateRmelon")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      'Biobase', 'limma', 'matrixStats', 'methylumi', 'lumi', 'ROC',
      'IlluminaHumanMethylation450kanno.ilmn12.hg19', 'illuminaio'
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Package listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘Biobase’
    A package should be listed in only one of these fields.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Packages in Depends field not imported from:
      ‘IlluminaHumanMethylation450kanno.ilmn12.hg19’ ‘matrixStats’
      ‘methylumi’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    Unexported objects imported by ':::' calls:
      ‘minfi:::.annoGet’ ‘minfi:::.availableAnnotation’
      ‘minfi:::.isRGOrStop’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    .adjusted_normalizeFunnorm450k : unbiased_normalizeQuantiles: no
      visible global function definition for ‘colQuantiles’
    .adjusted_normalizeFunnorm450k: no visible global function definition
      for ‘getMeth’
    .adjusted_normalizeFunnorm450k: no visible global function definition
      for ‘getUnmeth’
    .adjusted_normalizeFunnorm450k: no visible global function definition
      for ‘assay<-’
    .buildControlMatrix450k: no visible global function definition for
      ‘colMeans2’
    ...
      betas betas<- colCummaxs colData colMeans2 colMedians colQuantiles
      colSds detectionP ds epic.controls epic.ordering getAnnotation
      getAnnotationObject getBeta getCN getControlAddress getGreen
      getManifest getManifestInfo getMeth getNBeads getOOB getProbeInfo
      getProbeType getRed getSex getSnpBeta getUnmeth hm27.controls
      hm27.ordering hm450.controls hm450.ordering i intensitiesByChannel
      mapToGenome mclapply metadata methylated methylated<- methylumiR n.sd
      n_1 preprocessNoob preprocessRaw preprocessSWAN
      probeAnnotationsCategory projectName pval.detect<- pvals rowMeans2
      seqnames sex_coef subsetByLoci unmethylated unmethylated<- warn
    ```

# WGCNA

<details>

* Version: 1.70-3
* GitHub: NA
* Source code: https://github.com/cran/WGCNA
* Date/Publication: 2021-02-28 20:20:03 UTC
* Number of recursive dependencies: 103

Run `revdep_details(, "WGCNA")` for more info

</details>

## In both

*   checking Rd cross-references ... WARNING
    ```
    Missing link or links in documentation object 'empiricalBayesLM.Rd':
      ‘rlm’
    
    See section 'Cross-references' in the 'Writing R Extensions' manual.
    ```

