# abcrf

<details>

* Version: 1.8.1
* GitHub: NA
* Source code: https://github.com/cran/abcrf
* Date/Publication: 2019-11-05 14:40:02 UTC
* Number of recursive dependencies: 33

Run `revdep_details(, "abcrf")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.3Mb
      sub-directories of 1Mb or more:
        data   2.6Mb
        libs   4.4Mb
    ```

# airpart

<details>

* Version: 1.0.0
* GitHub: NA
* Source code: https://github.com/cran/airpart
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 148

Run `revdep_details(, "airpart")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'scater', 'smurf', 'apeglm', 'emdbook', 'ComplexHeatmap'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# AlpsNMR

<details>

* Version: 3.1.5
* GitHub: NA
* Source code: https://github.com/cran/AlpsNMR
* Date/Publication: 2021-03-31
* Number of recursive dependencies: 162

Run `revdep_details(, "AlpsNMR")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'mixOmics', 'speaq', 'baseline'
    
    Package suggested but not available for checking: ‘ChemoSpec’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# AMARETTO

<details>

* Version: 1.8.0
* GitHub: NA
* Source code: https://github.com/cran/AMARETTO
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 153

Run `revdep_details(, "AMARETTO")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'ComplexHeatmap', 'circlize', 'curatedTCGAData'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# amplican

<details>

* Version: 1.14.0
* GitHub: https://github.com/valenlab/amplican
* Source code: https://github.com/cran/amplican
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 105

Run `revdep_details(, "amplican")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'ShortRead', 'waffle'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# apollo

<details>

* Version: 0.2.4
* GitHub: NA
* Source code: https://github.com/cran/apollo
* Date/Publication: 2021-02-25 15:30:05 UTC
* Number of recursive dependencies: 69

Run `revdep_details(, "apollo")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'maxLik', 'randtoolbox'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# aroma.affymetrix

<details>

* Version: 3.2.0
* GitHub: https://github.com/HenrikBengtsson/aroma.affymetrix
* Source code: https://github.com/cran/aroma.affymetrix
* Date/Publication: 2019-06-23 06:00:14 UTC
* Number of recursive dependencies: 73

Run `revdep_details(, "aroma.affymetrix")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      'affy', 'affyPLM', 'gcrma', 'oligo', 'oligoClasses', 'pdInfoBuilder'
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  8.0Mb
      sub-directories of 1Mb or more:
        R             4.0Mb
        help          2.1Mb
        testScripts   1.1Mb
    ```

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘affy’, ‘gcrma’, ‘affyPLM’, ‘oligo’, ‘oligoClasses’
    ```

# aroma.cn

<details>

* Version: 1.6.1
* GitHub: https://github.com/HenrikBengtsson/aroma.cn
* Source code: https://github.com/cran/aroma.cn
* Date/Publication: 2015-10-28 00:08:16
* Number of recursive dependencies: 25

Run `revdep_details(, "aroma.cn")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘GLAD’
    ```

# aroma.core

<details>

* Version: 3.2.2
* GitHub: https://github.com/HenrikBengtsson/aroma.core
* Source code: https://github.com/cran/aroma.core
* Date/Publication: 2021-01-05 05:10:12 UTC
* Number of recursive dependencies: 47

Run `revdep_details(, "aroma.core")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      'EBImage', 'GLAD', 'sfit', 'expectile', 'HaarSeg', 'mpcbs'
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘GLAD’
    ```

# aroma.light

<details>

* Version: 3.22.0
* GitHub: https://github.com/HenrikBengtsson/aroma.light
* Source code: https://github.com/cran/aroma.light
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 6

Run `revdep_details(, "aroma.light")` for more info

</details>

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      inst/rsp/.rspPlugins
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

# autonomics

<details>

* Version: 1.0.1
* GitHub: https://github.com/bhagwataditya/autonomics
* Source code: https://github.com/cran/autonomics
* Date/Publication: 2021-06-06
* Number of recursive dependencies: 201

Run `revdep_details(, "autonomics")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘assertive’
    
    Packages suggested but not available for checking:
      'affy', 'hgu95av2.db', 'ICSNP', 'mixOmics', 'org.Hs.eg.db',
      'org.Mm.eg.db', 'ropls', 'seqinr'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# bahc

<details>

* Version: 0.3.0
* GitHub: NA
* Source code: https://github.com/cran/bahc
* Date/Publication: 2020-09-21 16:40:02 UTC
* Number of recursive dependencies: 2

Run `revdep_details(, "bahc")` for more info

</details>

## In both

*   checking LazyData ... NOTE
    ```
      'LazyData' is specified without a 'data' directory
    ```

# BASiCS

<details>

* Version: 2.4.0
* GitHub: https://github.com/catavallejos/BASiCS
* Source code: https://github.com/cran/BASiCS
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 136

Run `revdep_details(, "BASiCS")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'scran', 'scuttle'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# BatchQC

<details>

* Version: 1.20.0
* GitHub: https://github.com/mani2012/BatchQC
* Source code: https://github.com/cran/BatchQC
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 160

Run `revdep_details(, "BatchQC")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘sva’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# baystability

<details>

* Version: 0.1.0
* GitHub: NA
* Source code: https://github.com/cran/baystability
* Date/Publication: 2018-03-13 15:55:34 UTC
* Number of recursive dependencies: 106

Run `revdep_details(, "baystability")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘ggfortify’ ‘ggplot2’ ‘matrixStats’ ‘reshape2’ ‘scales’
      All declared Imports should be used.
    ```

# bdynsys

<details>

* Version: 1.3
* GitHub: NA
* Source code: https://github.com/cran/bdynsys
* Date/Publication: 2014-12-08 07:01:51
* Number of recursive dependencies: 75

Run `revdep_details(, "bdynsys")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘plm’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# bigPint

<details>

* Version: 1.8.0
* GitHub: https://github.com/lindsayrutter/bigPint
* Source code: https://github.com/cran/bigPint
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 167

Run `revdep_details(, "bigPint")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘EDASeq’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 11.7Mb
      sub-directories of 1Mb or more:
        data             6.0Mb
        doc              2.3Mb
        shiny-examples   3.0Mb
    ```

# bingat

<details>

* Version: 1.3
* GitHub: NA
* Source code: https://github.com/cran/bingat
* Date/Publication: 2017-07-05 18:30:37 UTC
* Number of recursive dependencies: 34

Run `revdep_details(, "bingat")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘network’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# BioNERO

<details>

* Version: 1.0.1
* GitHub: https://github.com/almeidasilvaf/BioNERO
* Source code: https://github.com/cran/BioNERO
* Date/Publication: 2021-07-04
* Number of recursive dependencies: 208

Run `revdep_details(, "BioNERO")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'WGCNA', 'DESeq2', 'sva', 'ComplexHeatmap', 'ggnetwork',
      'intergraph', 'ggpubr'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# biscuiteer

<details>

* Version: 1.6.0
* GitHub: https://github.com/trichelab/biscuiteer
* Source code: https://github.com/cran/biscuiteer
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 214

Run `revdep_details(, "biscuiteer")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'biscuiteerData', 'bsseq', 'HDF5Array', 'VariantAnnotation',
      'Mus.musculus', 'Homo.sapiens', 'dmrseq'
    
    Packages suggested but not available for checking:
      'DSS', 'scmeth', 'pkgdown'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# BloodGen3Module

<details>

* Version: 1.0.0
* GitHub: NA
* Source code: https://github.com/cran/BloodGen3Module
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 163

Run `revdep_details(, "BloodGen3Module")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'ExperimentHub', 'circlize', 'ComplexHeatmap'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# bnbc

<details>

* Version: 1.14.0
* GitHub: https://github.com/hansenlab/bnbc
* Source code: https://github.com/cran/bnbc
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 92

Run `revdep_details(, "bnbc")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'rhdf5', 'sva', 'EBImage'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# bnclassify

<details>

* Version: 0.4.5
* GitHub: https://github.com/bmihaljevic/bnclassify
* Source code: https://github.com/cran/bnclassify
* Date/Publication: 2020-03-12 17:40:02 UTC
* Number of recursive dependencies: 92

Run `revdep_details(, "bnclassify")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      'gRain', 'gRbase', 'Rgraphviz'
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  9.1Mb
      sub-directories of 1Mb or more:
        libs   8.2Mb
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘gRain’
    ```

# bnem

<details>

* Version: 1.0.0
* GitHub: https://github.com/MartinFXP/bnem
* Source code: https://github.com/cran/bnem
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 157

Run `revdep_details(, "bnem")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'CellNOptR', 'Rgraphviz', 'epiNEM', 'mnem', 'affy', 'sva', 'vsn'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# brms

<details>

* Version: 2.15.0
* GitHub: https://github.com/paul-buerkner/brms
* Source code: https://github.com/cran/brms
* Date/Publication: 2021-03-14 15:50:31 UTC
* Number of recursive dependencies: 223

Run `revdep_details(, "brms")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'shinystan', 'projpred'
    
    Packages suggested but not available for checking:
      'emmeans', 'cmdstanr', 'rtdists', 'MCMCglmm'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# BSgenome

<details>

* Version: 1.60.0
* GitHub: https://github.com/Bioconductor/BSgenome
* Source code: https://github.com/cran/BSgenome
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 101

Run `revdep_details(, "BSgenome")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘BSgenome-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: BSgenomeViews-class
    > ### Title: BSgenomeViews objects
    > ### Aliases: class:BSgenomeViews BSgenomeViews-class BSgenomeViews
    > ###   Views,BSgenome-method subject,BSgenomeViews-method
    > ###   granges,BSgenomeViews-method length,BSgenomeViews-method
    > ###   names,BSgenomeViews-method seqnames,BSgenomeViews-method
    > ###   start,BSgenomeViews-method end,BSgenomeViews-method
    ...
    > ### Keywords: methods classes
    > 
    > ### ** Examples
    > 
    > library(BSgenome.Mmusculus.UCSC.mm10)
    > genome <- BSgenome.Mmusculus.UCSC.mm10
    > library(TxDb.Mmusculus.UCSC.mm10.knownGene)
    Error in library(TxDb.Mmusculus.UCSC.mm10.knownGene) : 
      there is no package called ‘TxDb.Mmusculus.UCSC.mm10.knownGene’
    Execution halted
    ```

*   checking Rd cross-references ... WARNING
    ```
    Package unavailable to check Rd xrefs: ‘GenomicFeatures’
    Missing link or links in documentation object 'export-methods.Rd':
      ‘[rtracklayer]{export}’
    
    See section 'Cross-references' in the 'Writing R Extensions' manual.
    ```

*   checking for missing documentation entries ... WARNING
    ```
    Undocumented code objects:
      ‘MaskedBSgenome’ ‘OnDiskLongTable’ ‘OnDiskLongTable_old’
      ‘as.data.frame.BSgenomeViews’ ‘batchsizes’ ‘blocksizes’ ‘breakpoints’
      ‘forgeMaskedBSgenomeDataPkg’
      ‘getBatchesByOverlapsFromOnDiskLongTable’
      ‘getBatchesBySeqnameFromOnDiskLongTable’
      ‘getBatchesFromOnDiskLongTable’ ‘getBatchesFromOnDiskLongTable_old’
      ‘getRowsByIdFromOnDiskLongTable’ ‘getRowsByIdFromOnDiskLongTable_old’
      ‘getRowsByIndexFromOnDiskLongTable_old’ ‘getRowsFromOnDiskLongTable’
      ‘rowids’ ‘saveAsOnDiskLongTable_old’
    ...
      generic 'seqinfo' and siglist 'TwobitNamedSequences'
      generic 'seqnames' and siglist 'OnDiskNamedSequences'
      generic 'show' and siglist 'OnDiskLongTable'
      generic 'show' and siglist 'OnDiskLongTable_old'
      generic 'show' and siglist 'OnDiskNamedSequences'
      generic 'spatialIndex' and siglist 'OnDiskLongTable'
    All user-level objects in a package (including S4 classes and methods)
    should have documentation entries.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking running R code from vignettes ...
    ```
      ‘BSgenomeForge.Rnw’ using ‘UTF-8’... OK
      ‘GenomeSearching.Rnw’ using ‘UTF-8’... failed
     WARNING
    Errors in running code in vignettes:
    when running code in ‘GenomeSearching.Rnw’
      ...
    > plotGenomeHits <- function(bsgenome, seqnames, hits) {
    +     chrlengths <- seqlengths(bsgenome)[seqnames]
    +     XMAX <- max(chrlengths)
    +     YMAX < .... [TRUNCATED] 
    
    > library(hgu95av2probe)
    
      When sourcing ‘GenomeSearching.R’:
    Error: there is no package called ‘hgu95av2probe’
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      'BSgenome.Hsapiens.UCSC.hg38.masked',
      'TxDb.Hsapiens.UCSC.hg38.knownGene',
      'TxDb.Mmusculus.UCSC.mm10.knownGene', 'hgu95av2probe'
    
    Depends: includes the non-default packages:
      'BiocGenerics', 'S4Vectors', 'IRanges', 'GenomeInfoDb',
      'GenomicRanges', 'Biostrings', 'rtracklayer'
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking for portable file names ... NOTE
    ```
    Found the following non-portable file path:
      BSgenome/inst/extdata/GentlemanLab/BSgenome.Gmellonella.NCBI.ASM364042v2-tools/fasta_to_sorted_2bit.R
    
    Tarballs are only required to store paths of up to 100 bytes and cannot
    store those of more than 256 bytes, with restrictions including to 100
    bytes for the final component.
    See section ‘Package structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  8.5Mb
      sub-directories of 1Mb or more:
        R         2.0Mb
        extdata   5.8Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Packages listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘methods’ ‘BiocGenerics’ ‘S4Vectors’ ‘IRanges’ ‘GenomeInfoDb’ ‘GenomicRanges’ ‘Biostrings’ ‘rtracklayer’
    A package should be listed in only one of these fields.
    ```

*   checking package subdirectories ... NOTE
    ```
    Found the following CITATION file in a non-standard place:
      inst/extdata/GentlemanLab/BSgenome.Creinhardtii.JGI.v5.6-tools/CITATION
    Most likely ‘inst/CITATION’ should be used instead.
    ```

*   checking dependencies in R code ... NOTE
    ```
    ':::' calls which should be '::':
      ‘S4Vectors:::makeClassinfoRowForCompactPrinting’
      ‘S4Vectors:::makePrettyMatrixForCompactPrinting’
      See the note in ?`:::` about the use of this operator.
    Unexported objects imported by ':::' calls:
      ‘GenomeInfoDb:::compactPrintNamedAtomicVector’
      ‘GenomicRanges:::get_out_of_bound_index’ ‘IRanges:::.normargSEW’
      ‘IRanges:::unlist_as_integer’ ‘S4Vectors:::anyMissing’
      ‘S4Vectors:::anyMissingOrOutside’ ‘S4Vectors:::decodeRle’
      ‘S4Vectors:::diffWithInitialZero’
      ‘S4Vectors:::make_zero_col_DataFrame’ ‘S4Vectors:::quick_unlist’
      ‘S4Vectors:::quick_unsplit’ ‘S4Vectors:::recycleVector’
      ‘rtracklayer:::.DNAString_to_twoBit’ ‘rtracklayer:::.TwoBits_export’
      ‘rtracklayer:::checkArgFormat’ ‘rtracklayer:::twoBitPath’
      See the note in ?`:::` about the use of this operator.
    ```

# BSW

<details>

* Version: 0.1.1
* GitHub: https://github.com/adam-bec/BSW
* Source code: https://github.com/cran/BSW
* Date/Publication: 2021-03-22 16:20:09 UTC
* Number of recursive dependencies: 47

Run `revdep_details(, "BSW")` for more info

</details>

## In both

*   checking LazyData ... NOTE
    ```
      'LazyData' is specified without a 'data' directory
    ```

# bumphunter

<details>

* Version: 1.34.0
* GitHub: https://github.com/rafalab/bumphunter
* Source code: https://github.com/cran/bumphunter
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 112

Run `revdep_details(, "bumphunter")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘GenomicFeatures’
    
    Packages suggested but not available for checking:
      'org.Hs.eg.db', 'TxDb.Hsapiens.UCSC.hg19.knownGene'
    
    Depends: includes the non-default packages:
      'S4Vectors', 'IRanges', 'GenomeInfoDb', 'GenomicRanges', 'foreach',
      'iterators', 'parallel', 'locfit'
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# CARBayesST

<details>

* Version: 3.2.1
* GitHub: https://github.com/duncanplee/CARBayesST
* Source code: https://github.com/cran/CARBayesST
* Date/Publication: 2021-05-31 07:30:09 UTC
* Number of recursive dependencies: 99

Run `revdep_details(, "CARBayesST")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.9Mb
      sub-directories of 1Mb or more:
        libs   6.1Mb
    ```

# carx

<details>

* Version: 0.7.1
* GitHub: NA
* Source code: https://github.com/cran/carx
* Date/Publication: 2017-11-20 04:24:20 UTC
* Number of recursive dependencies: 10

Run `revdep_details(, "carx")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘tmvtnorm’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# CDSeq

<details>

* Version: 1.0.8
* GitHub: https://github.com/kkang7/CDSeq_R_Package
* Source code: https://github.com/cran/CDSeq
* Date/Publication: 2021-02-10 16:10:02 UTC
* Number of recursive dependencies: 198

Run `revdep_details(, "CDSeq")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'qlcMatrix', 'Seurat', 'ggpubr'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# celda

<details>

* Version: 1.8.1
* GitHub: https://github.com/campbio/celda
* Source code: https://github.com/cran/celda
* Date/Publication: 2021-05-30
* Number of recursive dependencies: 356

Run `revdep_details(, "celda")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'MCMCprecision', 'scater', 'scran', 'ComplexHeatmap',
      'multipanelfigure', 'circlize'
    
    Packages suggested but not available for checking:
      'biomaRt', 'TENxPBMCData', 'singleCellTK'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# CelliD

<details>

* Version: 1.0.0
* GitHub: NA
* Source code: https://github.com/cran/CelliD
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 245

Run `revdep_details(, "CelliD")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'Seurat', 'scater'
    
    Packages suggested but not available for checking: 'ggpubr', 'destiny'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# cellmigRation

<details>

* Version: 1.0.0
* GitHub: https://github.com/ocbe-uio/cellmigRation
* Source code: https://github.com/cran/cellmigRation
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 166

Run `revdep_details(, "cellmigRation")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'FME', 'SpatialTools', 'vioplot', 'FactoMineR'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# cellWise

<details>

* Version: 2.2.5
* GitHub: NA
* Source code: https://github.com/cran/cellWise
* Date/Publication: 2021-03-09 11:00:06 UTC
* Number of recursive dependencies: 72

Run `revdep_details(, "cellWise")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      'robustHD', 'rospca', 'GSE'
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 14.9Mb
      sub-directories of 1Mb or more:
        data   3.1Mb
        doc    2.1Mb
        libs   9.2Mb
    ```

# CEMiTool

<details>

* Version: 1.16.0
* GitHub: NA
* Source code: https://github.com/cran/CEMiTool
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 183

Run `revdep_details(, "CEMiTool")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'WGCNA', 'ggpmisc', 'sna', 'clusterProfiler', 'intergraph', 'network'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# ChAMP

<details>

* Version: 2.22.0
* GitHub: NA
* Source code: https://github.com/cran/ChAMP
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 240

Run `revdep_details(, "ChAMP")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'minfi', 'DMRcate', 'IlluminaHumanMethylationEPICmanifest',
      'globaltest', 'sva', 'IlluminaHumanMethylation450kmanifest',
      'IlluminaHumanMethylationEPICanno.ilm10b4.hg19', 'wateRmelon',
      'goseq', 'missMethyl', 'isva', 'bumphunter'
    
    Depends: includes the non-default packages:
      'minfi', 'ChAMPdata', 'DMRcate', 'Illumina450ProbeVariants.db',
      'IlluminaHumanMethylationEPICmanifest', 'DT', 'RPMM'
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# ChemoSpec2D

<details>

* Version: 0.4.187
* GitHub: https://github.com/bryanhanson/ChemoSpec2D
* Source code: https://github.com/cran/ChemoSpec2D
* Date/Publication: 2021-03-16 06:40:02 UTC
* Number of recursive dependencies: 99

Run `revdep_details(, "ChemoSpec2D")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘ChemoSpec2D-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: pfacSpectra2D
    > ### Title: PARAFAC Analysis of a Spectra2D Object
    > ### Aliases: pfacSpectra2D
    > ### Keywords: multivariate
    > 
    > ### ** Examples
    > 
    > 
    > data(MUD1)
    > set.seed(123)
    > res <- pfacSpectra2D(MUD1, parallel = FALSE, nfac = 2)
    Error in pfacSpectra2D(MUD1, parallel = FALSE, nfac = 2) : 
      You must install package multiway to use this function
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      'multiway', 'mlrMBO', 'smoof'
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘multiway’
    ```

# Chicago

<details>

* Version: 1.20.0
* GitHub: NA
* Source code: https://github.com/cran/Chicago
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 172

Run `revdep_details(, "Chicago")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘Chicago-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: exportResults
    > ### Title: Export Results
    > ### Aliases: exportResults exportToGI
    > 
    > ### ** Examples
    > 
    > data(cdUnitTest)
    ...
    Writing out text file for WashU browser upload...
    > 
    > GI <- exportToGI(cdUnitTest)
    Reading the restriction map file...
    Reading the bait map file...
    Preparing the output table...
    Error in loadNamespace(x) : 
      there is no package called ‘GenomicInteractions’
    Calls: exportToGI ... loadNamespace -> withRestarts -> withOneRestart -> doWithOneRestart
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      'PCHiCdata', 'GenomicInteractions', 'AnnotationHub'
    ```

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘PCHiCdata’, ‘GenomicInteractions’
    ```

# ChIPpeakAnno

<details>

* Version: 3.26.0
* GitHub: NA
* Source code: https://github.com/cran/ChIPpeakAnno
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 207

Run `revdep_details(, "ChIPpeakAnno")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'ensembldb', 'GenomicFeatures', 'RBGL', 'biomaRt'
    
    Packages suggested but not available for checking:
      'AnnotationHub', 'reactome.db', 'org.Ce.eg.db', 'org.Hs.eg.db',
      'seqinr', 'EnsDb.Hsapiens.v75', 'EnsDb.Hsapiens.v79',
      'TxDb.Hsapiens.UCSC.hg19.knownGene',
      'TxDb.Hsapiens.UCSC.hg38.knownGene', 'GO.db', 'trackViewer',
      'motifStack', 'OrganismDbi'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# chromswitch

<details>

* Version: 1.14.0
* GitHub: https://github.com/sjessa/chromswitch
* Source code: https://github.com/cran/chromswitch
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 162

Run `revdep_details(, "chromswitch")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘NMF’
    
    Package suggested but not available for checking: ‘DescTools’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# cifti

<details>

* Version: 0.4.5
* GitHub: NA
* Source code: https://github.com/cran/cifti
* Date/Publication: 2018-02-01 23:25:24 UTC
* Number of recursive dependencies: 67

Run `revdep_details(, "cifti")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘oro.nifti’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# cliqueMS

<details>

* Version: 1.6.0
* GitHub: https://github.com/osenan/cliqueMS
* Source code: https://github.com/cran/cliqueMS
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 136

Run `revdep_details(, "cliqueMS")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'xcms', 'MSnbase', 'qlcMatrix'
    
    Package suggested but not available for checking: ‘CAMERA’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# clusterExperiment

<details>

* Version: 2.12.0
* GitHub: https://github.com/epurdom/clusterExperiment
* Source code: https://github.com/cran/clusterExperiment
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 181

Run `revdep_details(, "clusterExperiment")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'NMF', 'HDF5Array', 'zinbwave', 'phylobase', 'mbkmeans'
    
    Packages suggested but not available for checking: 'MAST', 'scran'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# clustifyr

<details>

* Version: 1.4.0
* GitHub: https://github.com/rnabioco/clustifyr
* Source code: https://github.com/cran/clustifyr
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 143

Run `revdep_details(, "clustifyr")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘clustifyr-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: plot_cor_heatmap
    > ### Title: Plot similarity measures on heatmap
    > ### Aliases: plot_cor_heatmap
    > 
    > ### ** Examples
    > 
    > res <- clustify(
    ...
    +     cluster_col = "classified",
    +     per_cell = FALSE
    + )
    using # of genes: 599
    similarity computation completed, matrix of 9 x 13, preparing output
    > 
    > plot_cor_heatmap(res)
    Error in loadNamespace(x) : there is no package called ‘ComplexHeatmap’
    Calls: plot_cor_heatmap ... loadNamespace -> withRestarts -> withOneRestart -> doWithOneRestart
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Complete output:
      > library(testthat)
      > library(clustifyr)
      > 
      > test_check("clustifyr")
      ══ Skipped tests ═══════════════════════════════════════════════════════════════
      • Seurat cannot be loaded (1)
      
    ...
          █
       1. ├─clustifyr::plot_cor_heatmap(res) test_plots.R:172:4
       2. └─base::loadNamespace(x)
       3.   └─base::withRestarts(stop(cond), retry_loadNamespace = function() NULL)
       4.     └─base:::withOneRestart(expr, restarts[[1L]])
       5.       └─base:::doWithOneRestart(return(expr), restart)
      
      [ FAIL 3 | WARN 0 | SKIP 1 | PASS 155 ]
      Error: Test failures
      Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘ComplexHeatmap’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.0Mb
      sub-directories of 1Mb or more:
        data   2.6Mb
        doc    1.5Mb
        help   1.2Mb
    ```

# cmapR

<details>

* Version: 1.4.0
* GitHub: https://github.com/cmap/cmapR
* Source code: https://github.com/cran/cmapR
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 74

Run `revdep_details(, "cmapR")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'rhdf5', 'flowCore'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# cna

<details>

* Version: 3.2.0
* GitHub: NA
* Source code: https://github.com/cran/cna
* Date/Publication: 2021-06-14 15:00:02 UTC
* Number of recursive dependencies: 73

Run `revdep_details(, "cna")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.5Mb
      sub-directories of 1Mb or more:
        libs   5.8Mb
    ```

# CNVScope

<details>

* Version: 3.6.0
* GitHub: https://github.com/jamesdalg/CNVScope
* Source code: https://github.com/cran/CNVScope
* Date/Publication: 2021-05-24 11:10:03 UTC
* Number of recursive dependencies: 199

Run `revdep_details(, "CNVScope")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'GenomicInteractions', 'OpenImageR', 'biomaRt'
    
    Packages suggested but not available for checking:
      'ComplexHeatmap', 'circlize', 'GenomicFeatures'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# cola

<details>

* Version: 1.8.0
* GitHub: https://github.com/jokergoo/cola
* Source code: https://github.com/cran/cola
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 230

Run `revdep_details(, "cola")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'ComplexHeatmap', 'GetoptLong', 'circlize', 'skmeans'
    
    Packages suggested but not available for checking:
      'genefilter', 'samr', 'NMF', 'WGCNA', 'clusterProfiler',
      'ReactomePA', 'DOSE', 'hu6800.db', 'Polychrome', 'simplifyEnrichment'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# collapse

<details>

* Version: 1.6.2
* GitHub: https://github.com/SebKrantz/collapse
* Source code: https://github.com/cran/collapse
* Date/Publication: 2021-07-04 21:00:01 UTC
* Number of recursive dependencies: 131

Run `revdep_details(, "collapse")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      'plm', 'fixest', 'vars', 'weights'
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 21.9Mb
      sub-directories of 1Mb or more:
        data   1.0Mb
        libs  18.7Mb
    ```

# ComplexHeatmap

<details>

* Version: 2.8.0
* GitHub: https://github.com/jokergoo/ComplexHeatmap
* Source code: https://github.com/cran/ComplexHeatmap
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 100

Run `revdep_details(, "ComplexHeatmap")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'circlize', 'GetoptLong'
    
    Package suggested but not available for checking: ‘EnrichedHeatmap’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# conclus

<details>

* Version: 1.0.0
* GitHub: NA
* Source code: https://github.com/cran/conclus
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 248

Run `revdep_details(, "conclus")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'fpc', 'factoextra', 'biomaRt', 'scran', 'scater', 'clusterProfiler'
    
    Packages suggested but not available for checking:
      'org.Hs.eg.db', 'org.Mm.eg.db'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# condiments

<details>

* Version: 1.0.0
* GitHub: https://github.com/HectorRDB/condiments
* Source code: https://github.com/cran/condiments
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 167

Run `revdep_details(, "condiments")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'slingshot', 'Ecume', 'TrajectoryUtils'
    
    Package suggested but not available for checking: ‘TSCAN’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# consensusOV

<details>

* Version: 1.14.0
* GitHub: https://github.com/bhklab/consensusOV
* Source code: https://github.com/cran/consensusOV
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 151

Run `revdep_details(, "consensusOV")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'GSVA', 'genefu'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# CopywriteR

<details>

* Version: 2.24.0
* GitHub: https://github.com/PeeperLab/CopywriteR
* Source code: https://github.com/cran/CopywriteR
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 62

Run `revdep_details(, "CopywriteR")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘chipseq’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# corrcoverage

<details>

* Version: 1.2.1
* GitHub: https://github.com/annahutch/corrcoverage
* Source code: https://github.com/cran/corrcoverage
* Date/Publication: 2019-12-06 23:20:12 UTC
* Number of recursive dependencies: 72

Run `revdep_details(, "corrcoverage")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘pkgdown’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.4Mb
      sub-directories of 1Mb or more:
        extdata   3.8Mb
        libs      1.2Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘data.table’
      All declared Imports should be used.
    ```

*   checking LazyData ... NOTE
    ```
      'LazyData' is specified without a 'data' directory
    ```

# crlmm

<details>

* Version: 1.50.0
* GitHub: NA
* Source code: https://github.com/cran/crlmm
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 58

Run `revdep_details(, "crlmm")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘oligoClasses’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# crossmeta

<details>

* Version: 1.18.0
* GitHub: NA
* Source code: https://github.com/cran/crossmeta
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 169

Run `revdep_details(, "crossmeta")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'affy', 'DESeq2', 'metaMA', 'oligo', 'reader', 'sva'
    
    Package suggested but not available for checking: ‘org.Hs.eg.db’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# cSEM

<details>

* Version: 0.4.0
* GitHub: https://github.com/M-E-Rademaker/cSEM
* Source code: https://github.com/cran/cSEM
* Date/Publication: 2021-04-19 22:00:18 UTC
* Number of recursive dependencies: 121

Run `revdep_details(, "cSEM")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'Rdpack', 'symmoments'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# cvCovEst

<details>

* Version: 0.3.5
* GitHub: https://github.com/PhilBoileau/cvCovEst
* Source code: https://github.com/cran/cvCovEst
* Date/Publication: 2021-04-18 06:50:07 UTC
* Number of recursive dependencies: 135

Run `revdep_details(, "cvCovEst")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'Rdpack', 'ggpubr'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# cytomapper

<details>

* Version: 1.4.1
* GitHub: https://github.com/BodenmillerGroup/cytomapper
* Source code: https://github.com/cran/cytomapper
* Date/Publication: 2021-05-21
* Number of recursive dependencies: 136

Run `revdep_details(, "cytomapper")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'EBImage', 'HDF5Array', 'rhdf5'
    
    Package suggested but not available for checking: ‘shinytest’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# CytoTree

<details>

* Version: 1.2.0
* GitHub: https://github.com/JhuangLab/CytoTree
* Source code: https://github.com/cran/CytoTree
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 241

Run `revdep_details(, "CytoTree")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'FlowSOM', 'destiny', 'flowUtils', 'flowCore', 'sva', 'scatterpie'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# DAMOCLES

<details>

* Version: 2.3
* GitHub: NA
* Source code: https://github.com/cran/DAMOCLES
* Date/Publication: 2020-08-12 14:22:11 UTC
* Number of recursive dependencies: 100

Run `revdep_details(, "DAMOCLES")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘DDD’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# dearseq

<details>

* Version: 1.4.0
* GitHub: https://github.com/borishejblum/dearseq
* Source code: https://github.com/cran/dearseq
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 139

Run `revdep_details(, "dearseq")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking: 'BiocSet', 'DESeq2'
    ```

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘DESeq2’, ‘BiocSet’
    ```

# decompr

<details>

* Version: 6.0.0
* GitHub: https://github.com/bquast/decompr
* Source code: https://github.com/cran/decompr
* Date/Publication: 2021-05-10 00:12:10 UTC
* Number of recursive dependencies: 46

Run `revdep_details(, "decompr")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘gvc’
    ```

# DeepBlueR

<details>

* Version: 1.18.0
* GitHub: NA
* Source code: https://github.com/cran/DeepBlueR
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 153

Run `revdep_details(, "DeepBlueR")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘Gviz’
    ```

*   checking Rd files ... NOTE
    ```
    prepare_Rd: deepblue_enrich_regions_fast.Rd:35-38: Dropping empty section \examples
    ```

# DelayedMatrixStats

<details>

* Version: 1.14.0
* GitHub: https://github.com/PeteHaitch/DelayedMatrixStats
* Source code: https://github.com/cran/DelayedMatrixStats
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 70

Run `revdep_details(, "DelayedMatrixStats")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘DelayedMatrixStats-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: colAnyMissings
    > ### Title: Checks if there are any missing values in an object or not
    > ### Aliases: colAnyMissings rowAnyMissings
    > ###   colAnyMissings,DelayedMatrix-method colAnyNAs,DelayedMatrix-method
    > ###   rowAnyMissings,DelayedMatrix-method rowAnyNAs,DelayedMatrix-method
    > 
    > ### ** Examples
    ...
    > # A DelayedMatrix with a 'matrix' seed
    > dm_matrix <- DelayedArray(matrix(c(rep(1L, 5),
    +                                    as.integer((0:4) ^ 2),
    +                                    seq(-5L, -1L, 1L)),
    +                                  ncol = 3))
    > # A DelayedMatrix with a 'HDF5ArraySeed' seed
    > # NOTE: Requires that the HDF5Array package is installed
    > library(HDF5Array)
    Error in library(HDF5Array) : there is no package called ‘HDF5Array’
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 50 lines of output:
      The following objects are masked from 'package:parallel':
      
          clusterApply, clusterApplyLB, clusterCall, clusterEvalQ,
          clusterExport, clusterMap, parApply, parCapply, parLapply,
          parLapplyLB, parRapply, parSapply, parSapplyLB
      
      The following objects are masked from 'package:stats':
    ...
      The following objects are masked from 'package:matrixStats':
      
          colAnyMissings, rowAnyMissings
      
      > 
      > test_check("DelayedMatrixStats")
      Error in (function (cond)  : 
        error in evaluating the argument 'x' in selecting a method for function 'seed': there is no package called 'HDF5Array'
      Calls: test_check ... realize -> .local -> load_BACKEND_package -> library
      Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘HDF5Array’
    ```

*   checking dependencies in R code ... NOTE
    ```
    Missing object imported by a ':::' call: ‘DelayedArray:::.reduce_array_dimensions’
    Unexported objects imported by ':::' calls:
      ‘DelayedArray:::.get_ans_type’ ‘DelayedArray:::RleArraySeed’
      ‘DelayedArray:::get_Nindex_lengths’ ‘DelayedArray:::set_dim’
      ‘DelayedArray:::set_dimnames’ ‘DelayedArray:::subset_by_Nindex’
      ‘DelayedArray:::to_linear_index’
      See the note in ?`:::` about the use of this operator.
    ```

# DeMixT

<details>

* Version: 1.8.0
* GitHub: NA
* Source code: https://github.com/cran/DeMixT
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 61

Run `revdep_details(, "DeMixT")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      'parallel', 'Rcpp', 'SummarizedExperiment', 'knitr', 'KernSmooth',
      'matrixcalc'
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 10.3Mb
      sub-directories of 1Mb or more:
        data   6.7Mb
        libs   2.7Mb
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    'library' or 'require' call not declared from: ‘calibrate’
    ```

# DepecheR

<details>

* Version: 1.8.0
* GitHub: NA
* Source code: https://github.com/cran/DepecheR
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 110

Run `revdep_details(, "DepecheR")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘mixOmics’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# DEqMS

<details>

* Version: 1.10.0
* GitHub: https://github.com/yafeng/DEqMS
* Source code: https://github.com/cran/DEqMS
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 115

Run `revdep_details(, "DEqMS")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘DEqMS-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: Residualplot
    > ### Title: plot the residuals against the number of quantified
    > ###   peptides/PSMs.
    > ### Aliases: Residualplot
    > 
    > ### ** Examples
    > 
    > library(ExperimentHub)
    Error in library(ExperimentHub) : 
      there is no package called ‘ExperimentHub’
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking: 'farms', 'ExperimentHub'
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Title field: should not end in a period.
    ```

# detrendr

<details>

* Version: 0.6.14
* GitHub: https://github.com/rorynolan/detrendr
* Source code: https://github.com/cran/detrendr
* Date/Publication: 2021-05-16 05:00:05 UTC
* Number of recursive dependencies: 105

Run `revdep_details(, "detrendr")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'arrayhelpers', 'autothresholdr', 'filesstrings'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# DGCA

<details>

* Version: 1.0.2
* GitHub: NA
* Source code: https://github.com/cran/DGCA
* Date/Publication: 2019-12-09 10:30:16 UTC
* Number of recursive dependencies: 164

Run `revdep_details(, "DGCA")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘WGCNA’
    
    Packages suggested but not available for checking:
      'GOstats', 'org.Hs.eg.db', 'MEGENA'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# DHS.rates

<details>

* Version: 0.9.0
* GitHub: NA
* Source code: https://github.com/cran/DHS.rates
* Date/Publication: 2021-06-18 08:20:11 UTC
* Number of recursive dependencies: 52

Run `revdep_details(, "DHS.rates")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘survey’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# diffloop

<details>

* Version: 1.20.0
* GitHub: https://github.com/aryeelab/diffloop
* Source code: https://github.com/cran/diffloop
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 146

Run `revdep_details(, "diffloop")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'Sushi', 'biomaRt'
    
    Package suggested but not available for checking: ‘DESeq2’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# diffUTR

<details>

* Version: 1.0.0
* GitHub: https://github.com/ETHZ-INS/diffUTR
* Source code: https://github.com/cran/diffUTR
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 143

Run `revdep_details(, "diffUTR")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'DEXSeq', 'ComplexHeatmap', 'ensembldb'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# DiscoRhythm

<details>

* Version: 1.8.0
* GitHub: https://github.com/matthewcarlucci/DiscoRhythm
* Source code: https://github.com/cran/DiscoRhythm
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 155

Run `revdep_details(, "DiscoRhythm")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘MetaCycle’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# dmrseq

<details>

* Version: 1.12.0
* GitHub: NA
* Source code: https://github.com/cran/dmrseq
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 161

Run `revdep_details(, "dmrseq")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'bsseq', 'bumphunter', 'annotatr', 'AnnotationHub'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# Doscheda

<details>

* Version: 1.14.0
* GitHub: NA
* Source code: https://github.com/cran/Doscheda
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 156

Run `revdep_details(, "Doscheda")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'vsn', 'affy'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# dplR

<details>

* Version: 1.7.2
* GitHub: https://github.com/AndyBunn/dplR
* Source code: https://github.com/cran/dplR
* Date/Publication: 2021-01-31 17:20:02 UTC
* Number of recursive dependencies: 88

Run `revdep_details(, "dplR")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘tikzDevice’
    ```

# DSWE

<details>

* Version: 1.5.1
* GitHub: https://github.com/TAMU-AML/DSWE-Package
* Source code: https://github.com/cran/DSWE
* Date/Publication: 2021-01-11 10:30:07 UTC
* Number of recursive dependencies: 18

Run `revdep_details(, "DSWE")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘mixtools’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# dtree

<details>

* Version: 0.4.2
* GitHub: NA
* Source code: https://github.com/cran/dtree
* Date/Publication: 2018-01-18 23:57:29 UTC
* Number of recursive dependencies: 82

Run `revdep_details(, "dtree")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'party', 'evtree', 'caret'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# ecospat

<details>

* Version: 3.2
* GitHub: https://github.com/ecospat/ecospat
* Source code: https://github.com/cran/ecospat
* Date/Publication: 2021-02-19 09:10:03 UTC
* Number of recursive dependencies: 237

Run `revdep_details(, "ecospat")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'adehabitatHR', 'biomod2', 'rms'
    
    Package suggested but not available for checking: ‘alphahull’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# EMDomics

<details>

* Version: 2.22.0
* GitHub: NA
* Source code: https://github.com/cran/EMDomics
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 51

Run `revdep_details(, "EMDomics")` for more info

</details>

## In both

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Authors@R field gives more than one person with maintainer role:
      Sadhika Malladi <contact@sadhikamalladi.com> [aut, cre]
      Daniel Schmolze <emd@schmolze.com> [aut, cre]
    ```

# ENmix

<details>

* Version: 1.28.2
* GitHub: NA
* Source code: https://github.com/cran/ENmix
* Date/Publication: 2021-06-01
* Number of recursive dependencies: 164

Run `revdep_details(, "ENmix")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'geneplotter', 'minfi', 'ExperimentHub', 'AnnotationHub',
      'genefilter'
    
    Package suggested but not available for checking: ‘minfiData’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# EnrichedHeatmap

<details>

* Version: 1.22.0
* GitHub: https://github.com/jokergoo/EnrichedHeatmap
* Source code: https://github.com/cran/EnrichedHeatmap
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 93

Run `revdep_details(, "EnrichedHeatmap")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'ComplexHeatmap', 'GetoptLong', 'circlize'
    
    Package suggested but not available for checking: ‘genefilter’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# EpiDISH

<details>

* Version: 2.8.0
* GitHub: https://github.com/sjczheng/EpiDISH
* Source code: https://github.com/cran/EpiDISH
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 79

Run `revdep_details(, "EpiDISH")` for more info

</details>

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

# epiGWAS

<details>

* Version: 1.0.2
* GitHub: NA
* Source code: https://github.com/cran/epiGWAS
* Date/Publication: 2019-09-08 16:50:02 UTC
* Number of recursive dependencies: 105

Run `revdep_details(, "epiGWAS")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'DescTools', 'SNPknock'
    
    Package suggested but not available for checking: ‘biglasso’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# epivizrData

<details>

* Version: 1.20.0
* GitHub: https://github.com/epiviz/epivizrData
* Source code: https://github.com/cran/epivizrData
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 155

Run `revdep_details(, "epivizrData")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'OrganismDbi', 'GenomicFeatures', 'ensembldb'
    
    Packages suggested but not available for checking:
      'bumphunter', 'hgu133plus2.db', 'Mus.musculus',
      'TxDb.Mmusculus.UCSC.mm10.knownGene', 'EnsDb.Mmusculus.v79',
      'AnnotationHub'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# EstimationTools

<details>

* Version: 2.1.0
* GitHub: https://github.com/Jaimemosg/EstimationTools
* Source code: https://github.com/cran/EstimationTools
* Date/Publication: 2021-03-10 18:40:05 UTC
* Number of recursive dependencies: 98

Run `revdep_details(, "EstimationTools")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘Rdpack’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# evaluomeR

<details>

* Version: 1.8.0
* GitHub: https://github.com/neobernad/evaluomeR
* Source code: https://github.com/cran/evaluomeR
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 107

Run `revdep_details(, "evaluomeR")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'fpc', 'Rdpack'
    
    Depends: includes the non-default packages:
      'SummarizedExperiment', 'MultiAssayExperiment', 'cluster', 'fpc',
      'randomForest', 'flexmix'
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# EventPointer

<details>

* Version: 3.0.0
* GitHub: https://github.com/jpromeror/EventPointer
* Source code: https://github.com/cran/EventPointer
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 161

Run `revdep_details(, "EventPointer")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'SGSeq', 'GenomicFeatures', 'RBGL', 'rhdf5'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# ExCluster

<details>

* Version: 1.10.0
* GitHub: NA
* Source code: https://github.com/cran/ExCluster
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 36

Run `revdep_details(, "ExCluster")` for more info

</details>

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .BBSoptions
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

# ExpHunterSuite

<details>

* Version: 1.0.0
* GitHub: NA
* Source code: https://github.com/cran/ExpHunterSuite
* Date/Publication: 2021-05-21
* Number of recursive dependencies: 215

Run `revdep_details(, "ExpHunterSuite")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'ReactomePA', 'biomaRt', 'topGO', 'clusterProfiler', 'diffcoexp',
      'WGCNA', 'enrichplot', 'DESeq2', 'MKinfer'
    
    Packages suggested but not available for checking:
      'optparse', 'TCC', 'org.Hs.eg.db', 'org.Mm.eg.db'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# ExpressionNormalizationWorkflow

<details>

* Version: 1.18.0
* GitHub: https://github.com/NA/NA
* Source code: https://github.com/cran/ExpressionNormalizationWorkflow
* Date/Publication: 2021-05-21
* Number of recursive dependencies: 106

Run `revdep_details(, "ExpressionNormalizationWorkflow")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'pvca', 'snm', 'sva', 'vsn'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# FADA

<details>

* Version: 1.3.5
* GitHub: NA
* Source code: https://github.com/cran/FADA
* Date/Publication: 2019-12-10 15:30:05 UTC
* Number of recursive dependencies: 22

Run `revdep_details(, "FADA")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'elasticnet', 'sparseLDA', 'sda'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# FastHCS

<details>

* Version: 0.0.7
* GitHub: NA
* Source code: https://github.com/cran/FastHCS
* Date/Publication: 2020-05-10 12:50:03 UTC
* Number of recursive dependencies: 5

Run `revdep_details(, "FastHCS")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 19.9Mb
      sub-directories of 1Mb or more:
        data   1.3Mb
        libs  18.5Mb
    ```

# FastPCS

<details>

* Version: 0.1.3
* GitHub: NA
* Source code: https://github.com/cran/FastPCS
* Date/Publication: 2018-05-22 16:49:12 UTC
* Number of recursive dependencies: 6

Run `revdep_details(, "FastPCS")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 10.2Mb
      sub-directories of 1Mb or more:
        libs  10.1Mb
    ```

# FastRCS

<details>

* Version: 0.0.8
* GitHub: NA
* Source code: https://github.com/cran/FastRCS
* Date/Publication: 2018-05-13 19:59:51 UTC
* Number of recursive dependencies: 6

Run `revdep_details(, "FastRCS")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 21.9Mb
      sub-directories of 1Mb or more:
        libs  21.8Mb
    ```

# fastshap

<details>

* Version: 0.0.5
* GitHub: https://github.com/bgreenwell/fastshap
* Source code: https://github.com/cran/fastshap
* Date/Publication: 2020-02-02 05:10:02 UTC
* Number of recursive dependencies: 79

Run `revdep_details(, "fastshap")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘earth’
    ```

# FateID

<details>

* Version: 0.2.0
* GitHub: NA
* Source code: https://github.com/cran/FateID
* Date/Publication: 2021-04-07 13:20:13 UTC
* Number of recursive dependencies: 108

Run `revdep_details(, "FateID")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘lle’
    
    Package suggested but not available for checking: ‘DESeq2’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# FDb.FANTOM4.promoters.hg19

<details>

* Version: 1.0.0
* GitHub: NA
* Source code: https://github.com/cran/FDb.FANTOM4.promoters.hg19
* Number of recursive dependencies: 89

Run `revdep_details(, "FDb.FANTOM4.promoters.hg19")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘GenomicFeatures’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# FEAST

<details>

* Version: 1.0.0
* GitHub: https://github.com/suke18/FEAST
* Source code: https://github.com/cran/FEAST
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 230

Run `revdep_details(, "FEAST")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'TSCAN', 'SC3'
    
    Packages suggested but not available for checking: 'Seurat', 'ggpubr'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# fergm

<details>

* Version: 1.1.4
* GitHub: https://github.com/benjamin-w-campbell/fergm
* Source code: https://github.com/cran/fergm
* Date/Publication: 2018-10-17 22:20:11 UTC
* Number of recursive dependencies: 92

Run `revdep_details(, "fergm")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'ergm', 'network'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# fipp

<details>

* Version: 1.0.0
* GitHub: NA
* Source code: https://github.com/cran/fipp
* Date/Publication: 2021-02-11 10:30:02 UTC
* Number of recursive dependencies: 21

Run `revdep_details(, "fipp")` for more info

</details>

## In both

*   checking LazyData ... NOTE
    ```
      'LazyData' is specified without a 'data' directory
    ```

# fishpond

<details>

* Version: 1.8.0
* GitHub: https://github.com/mikelove/fishpond
* Source code: https://github.com/cran/fishpond
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 179

Run `revdep_details(, "fishpond")` for more info

</details>

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Complete output:
      > library(testthat)
      > library(fishpond)
      > 
      > test_check("fishpond")
      ══ Skipped tests ═══════════════════════════════════════════════════════════════
      • empty test (4)
      
    ...
       5.       └─base:::doWithOneRestart(return(expr), restart)
      ── Error (test_swish.R:82:3): basic swish analyses ─────────────────────────────
      Error: first install the 'samr' package
      Backtrace:
          █
       1. └─fishpond::swish(...) test_swish.R:82:2
      
      [ FAIL 2 | WARN 0 | SKIP 4 | PASS 17 ]
      Error: Test failures
      Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      'tximeta', 'org.Hs.eg.db', 'samr', 'DESeq2', 'apeglm'
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unavailable namespace imported from by a ':::' call: ‘samr’
      See the note in ?`:::` about the use of this operator.
    ```

# flowCore

<details>

* Version: 2.4.0
* GitHub: NA
* Source code: https://github.com/cran/flowCore
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 134

Run `revdep_details(, "flowCore")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘cytolib’
    
    Packages suggested but not available for checking:
      'Rgraphviz', 'flowViz', 'flowStats', 'flowWorkspace', 'openCyto',
      'ggcyto'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# flowGraph

<details>

* Version: 1.0.0
* GitHub: https://github.com/aya49/flowGraph
* Source code: https://github.com/cran/flowGraph
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 83

Run `revdep_details(, "flowGraph")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘Rdpack’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# flowWorkspace

<details>

* Version: 4.4.0
* GitHub: NA
* Source code: https://github.com/cran/flowWorkspace
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 136

Run `revdep_details(, "flowWorkspace")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'cytolib', 'RBGL', 'Rgraphviz', 'flowCore', 'ncdfFlow'
    
    Packages suggested but not available for checking:
      'ggcyto', 'CytoML', 'openCyto'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# FRASER

<details>

* Version: 1.4.0
* GitHub: https://github.com/gagneurlab/FRASER
* Source code: https://github.com/cran/FRASER
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 185

Run `revdep_details(, "FRASER")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'biomaRt', 'GenomicFeatures', 'HDF5Array', 'OUTRIDER', 'rhdf5'
    
    Packages suggested but not available for checking:
      'TxDb.Hsapiens.UCSC.hg19.knownGene', 'org.Hs.eg.db'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# funtooNorm

<details>

* Version: 1.16.0
* GitHub: NA
* Source code: https://github.com/cran/funtooNorm
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 141

Run `revdep_details(, "funtooNorm")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'minfi', 'IlluminaHumanMethylation450kmanifest',
      'IlluminaHumanMethylation450kanno.ilmn12.hg19'
    
    Package suggested but not available for checking: ‘minfiData’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# gap

<details>

* Version: 1.2.3-1
* GitHub: https://github.com/jinghuazhao/R
* Source code: https://github.com/cran/gap
* Date/Publication: 2021-04-21 09:40:09 UTC
* Number of recursive dependencies: 147

Run `revdep_details(, "gap")` for more info

</details>

## In both

*   checking running R code from vignettes ...
    ```
      ‘gap.Rnw’... failed
      ‘jss.Rnw’... OK
     WARNING
    Errors in running code in vignettes:
    when running code in ‘gap.Rnw’
      ...
    > d <- gk1$kin.matrix - kk1
    
    > sum(abs(d))
    [1] 2.443634
    
    > library(pedigree)
    
      When sourcing ‘gap.R’:
    Error: there is no package called ‘pedigree’
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      'BradleyTerry2', 'MCMCglmm', 'R2jags', 'circlize', 'genetics',
      'haplo.stats', 'meta', 'metafor', 'mets', 'pedigree', 'rms'
    ```

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘haplo.stats’, ‘genetics’
    ```

# GAPGOM

<details>

* Version: 1.8.0
* GitHub: https://github.com/Berghopper/GAPGOM
* Source code: https://github.com/cran/GAPGOM
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 145

Run `revdep_details(, "GAPGOM")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'RBGL', 'GO.db', 'org.Hs.eg.db', 'org.Mm.eg.db', 'GOSemSim'
    
    Packages suggested but not available for checking:
      'org.Dm.eg.db', 'org.Rn.eg.db', 'org.Sc.sgd.db', 'org.Dr.eg.db',
      'org.Ce.eg.db', 'org.At.tair.db', 'org.EcK12.eg.db', 'org.Bt.eg.db',
      'org.Cf.eg.db', 'org.Ag.eg.db', 'org.EcSakai.eg.db', 'org.Gg.eg.db',
      'org.Pt.eg.db', 'org.Pf.plasmo.db', 'org.Mmu.eg.db', 'org.Ss.eg.db',
      'org.Xl.eg.db'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# GenEst

<details>

* Version: 1.4.6
* GitHub: NA
* Source code: https://github.com/cran/GenEst
* Date/Publication: 2021-06-17 07:10:08 UTC
* Number of recursive dependencies: 89

Run `revdep_details(, "GenEst")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘shinytest’
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘htmlwidgets’
      All declared Imports should be used.
    ```

# GeneTonic

<details>

* Version: 1.4.1
* GitHub: https://github.com/federicomarini/GeneTonic
* Source code: https://github.com/cran/GeneTonic
* Date/Publication: 2021-06-06
* Number of recursive dependencies: 200

Run `revdep_details(, "GeneTonic")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'backbone', 'circlize', 'ComplexHeatmap', 'DESeq2', 'GO.db'
    
    Packages suggested but not available for checking:
      'clusterProfiler', 'org.Hs.eg.db'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# genomation

<details>

* Version: 1.24.0
* GitHub: https://github.com/BIMSBbioinfo/genomation
* Source code: https://github.com/cran/genomation
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 92

Run `revdep_details(, "genomation")` for more info

</details>

## In both

*   checking whether package ‘genomation’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘Biostrings::pattern’ by ‘grid::pattern’ when loading ‘genomation’
    See ‘/scratch/henrik/revdepcheck.extras/matrixStats/revdep/checks/genomation/new/genomation.Rcheck/00install.out’ for details.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.9Mb
      sub-directories of 1Mb or more:
        doc       3.5Mb
        extdata   1.2Mb
        libs      1.7Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported object imported by a ':::' call: ‘BiocGenerics:::testPackage’
      See the note in ?`:::` about the use of this operator.
    ```

# geva

<details>

* Version: 1.0.0
* GitHub: https://github.com/sbcblab/geva
* Source code: https://github.com/cran/geva
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 105

Run `revdep_details(, "geva")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘topGO’
    ```

# ggdmc

<details>

* Version: 0.2.6.0
* GitHub: https://github.com/yxlin/ggdmc
* Source code: https://github.com/cran/ggdmc
* Date/Publication: 2019-04-29 05:10:03 UTC
* Number of recursive dependencies: 51

Run `revdep_details(, "ggdmc")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.6Mb
      sub-directories of 1Mb or more:
        libs   7.1Mb
    ```

*   checking LazyData ... NOTE
    ```
      'LazyData' is specified without a 'data' directory
    ```

# GGPA

<details>

* Version: 1.4.0
* GitHub: https://github.com/dongjunchung/GGPA
* Source code: https://github.com/cran/GGPA
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 68

Run `revdep_details(, "GGPA")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'network', 'sna'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# GJRM

<details>

* Version: 0.2-4
* GitHub: NA
* Source code: https://github.com/cran/GJRM
* Date/Publication: 2021-04-12 18:00:11 UTC
* Number of recursive dependencies: 67

Run `revdep_details(, "GJRM")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'survey', 'Rmpfr', 'copula', 'distrEx'
    
    Package which this enhances but not available for checking: ‘sp’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# glmGamPoi

<details>

* Version: 1.4.0
* GitHub: https://github.com/const-ae/glmGamPoi
* Source code: https://github.com/cran/glmGamPoi
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 168

Run `revdep_details(, "glmGamPoi")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘HDF5Array’
    
    Packages suggested but not available for checking:
      'DESeq2', 'TENxPBMCData', 'muscData', 'scran'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# GLMMadaptive

<details>

* Version: 0.8-2
* GitHub: https://github.com/drizopoulos/GLMMadaptive
* Source code: https://github.com/cran/GLMMadaptive
* Date/Publication: 2021-07-05 19:20:06 UTC
* Number of recursive dependencies: 106

Run `revdep_details(, "GLMMadaptive")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      'pkgdown', 'emmeans', 'effects'
    ```

# graper

<details>

* Version: 1.8.0
* GitHub: NA
* Source code: https://github.com/cran/graper
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 66

Run `revdep_details(, "graper")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.6Mb
      sub-directories of 1Mb or more:
        doc    1.8Mb
        libs   5.6Mb
    ```

# GUIDEseq

<details>

* Version: 1.22.0
* GitHub: NA
* Source code: https://github.com/cran/GUIDEseq
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 142

Run `revdep_details(, "GUIDEseq")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'CRISPRseek', 'ChIPpeakAnno'
    
    Packages suggested but not available for checking:
      'TxDb.Hsapiens.UCSC.hg19.knownGene', 'org.Hs.eg.db'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# Gviz

<details>

* Version: 1.36.2
* GitHub: https://github.com/ivanek/Gviz
* Source code: https://github.com/cran/Gviz
* Date/Publication: 2021-07-04
* Number of recursive dependencies: 150

Run `revdep_details(, "Gviz")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'biomaRt', 'GenomicFeatures', 'ensembldb', 'biovizBase'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# gwasurvivr

<details>

* Version: 1.10.0
* GitHub: https://github.com/suchestoncampbelllab/gwasurvivr
* Source code: https://github.com/cran/gwasurvivr
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 127

Run `revdep_details(, "gwasurvivr")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'GWASTools', 'VariantAnnotation', 'SNPRelate'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# GWENA

<details>

* Version: 1.2.0
* GitHub: https://github.com/Kumquatum/GWENA
* Source code: https://github.com/cran/GWENA
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 143

Run `revdep_details(, "GWENA")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘WGCNA’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# haldensify

<details>

* Version: 0.0.6
* GitHub: https://github.com/nhejazi/haldensify
* Source code: https://github.com/cran/haldensify
* Date/Publication: 2020-09-16 06:40:13 UTC
* Number of recursive dependencies: 83

Run `revdep_details(, "haldensify")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'hal9001', 'Rdpack'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# HDSpatialScan

<details>

* Version: 1.0.1
* GitHub: NA
* Source code: https://github.com/cran/HDSpatialScan
* Date/Publication: 2021-05-27 09:10:06 UTC
* Number of recursive dependencies: 88

Run `revdep_details(, "HDSpatialScan")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.2Mb
      sub-directories of 1Mb or more:
        libs   4.3Mb
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 42 marked UTF-8 strings
    ```

# hipathia

<details>

* Version: 2.8.0
* GitHub: NA
* Source code: https://github.com/cran/hipathia
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 127

Run `revdep_details(, "hipathia")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘AnnotationHub’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# IMIFA

<details>

* Version: 2.1.6
* GitHub: https://github.com/Keefe-Murphy/IMIFA
* Source code: https://github.com/cran/IMIFA
* Date/Publication: 2021-05-24 16:50:05 UTC
* Number of recursive dependencies: 33

Run `revdep_details(, "IMIFA")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘Rfast’
    
    Package suggested but not available for checking: ‘Rmpfr’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# incidental

<details>

* Version: 0.1
* GitHub: NA
* Source code: https://github.com/cran/incidental
* Date/Publication: 2020-09-16 09:50:03 UTC
* Number of recursive dependencies: 62

Run `revdep_details(, "incidental")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘dlnm’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# InfiniumPurify

<details>

* Version: 1.3.1
* GitHub: NA
* Source code: https://github.com/cran/InfiniumPurify
* Date/Publication: 2017-01-14 12:12:25
* Number of recursive dependencies: 1

Run `revdep_details(, "InfiniumPurify")` for more info

</details>

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘minfi’
    ```

# JMbayes2

<details>

* Version: 0.1-7
* GitHub: https://github.com/drizopoulos/JMbayes2
* Source code: https://github.com/cran/JMbayes2
* Date/Publication: 2021-07-07 10:50:06 UTC
* Number of recursive dependencies: 77

Run `revdep_details(, "JMbayes2")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘pkgdown’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  9.0Mb
      sub-directories of 1Mb or more:
        libs   8.1Mb
    ```

# kgschart

<details>

* Version: 1.3.5
* GitHub: https://github.com/kota7/kgschart
* Source code: https://github.com/cran/kgschart
* Date/Publication: 2017-07-02 22:16:15 UTC
* Number of recursive dependencies: 73

Run `revdep_details(, "kgschart")` for more info

</details>

## In both

*   checking LazyData ... NOTE
    ```
      'LazyData' is specified without a 'data' directory
    ```

# kissDE

<details>

* Version: 1.12.0
* GitHub: NA
* Source code: https://github.com/cran/kissDE
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 143

Run `revdep_details(, "kissDE")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'DESeq2', 'DSS'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# landmap

<details>

* Version: 0.0.11
* GitHub: https://github.com/envirometrix/landmap
* Source code: https://github.com/cran/landmap
* Date/Publication: 2021-05-27 06:40:07 UTC
* Number of recursive dependencies: 139

Run `revdep_details(, "landmap")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘spatstat’
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 23 marked UTF-8 strings
    ```

# latrend

<details>

* Version: 1.1.2
* GitHub: https://github.com/philips-software/latrend
* Source code: https://github.com/cran/latrend
* Date/Publication: 2021-04-14 13:50:10 UTC
* Number of recursive dependencies: 197

Run `revdep_details(, "latrend")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'Rdpack', 'longitudinalData', 'mclustcomp', 'caret'
    
    Packages suggested but not available for checking:
      'kml', 'mixtools', 'funFEM', 'akmedoids', 'dtwclust', 'mixAK'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# ldsep

<details>

* Version: 2.1.0
* GitHub: https://github.com/dcgerard/ldsep
* Source code: https://github.com/cran/ldsep
* Date/Publication: 2021-06-11 18:00:02 UTC
* Number of recursive dependencies: 159

Run `revdep_details(, "ldsep")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'ashr', 'modeest'
    
    Packages suggested but not available for checking:
      'updog', 'VariantAnnotation'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# liger

<details>

* Version: 2.0.1
* GitHub: https://github.com/JEFworks/liger
* Source code: https://github.com/cran/liger
* Date/Publication: 2021-01-25 05:50:09 UTC
* Number of recursive dependencies: 46

Run `revdep_details(, "liger")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.4Mb
      sub-directories of 1Mb or more:
        data   3.3Mb
        doc    2.1Mb
        libs   1.9Mb
    ```

# liminal

<details>

* Version: 0.1.2
* GitHub: https://github.com/sa-lee/liminal
* Source code: https://github.com/cran/liminal
* Date/Publication: 2021-05-28 07:30:05 UTC
* Number of recursive dependencies: 92

Run `revdep_details(, "liminal")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.8Mb
      sub-directories of 1Mb or more:
        data   7.1Mb
    ```

# loo

<details>

* Version: 2.4.1
* GitHub: https://github.com/stan-dev/loo
* Source code: https://github.com/cran/loo
* Date/Publication: 2020-12-09 22:50:02 UTC
* Number of recursive dependencies: 189

Run `revdep_details(, "loo")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking: 'brms', 'rstanarm'
    ```

# LS2Wstat

<details>

* Version: 2.1-2
* GitHub: NA
* Source code: https://github.com/cran/LS2Wstat
* Date/Publication: 2020-02-03 20:40:11 UTC
* Number of recursive dependencies: 34

Run `revdep_details(, "LS2Wstat")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘LS2W’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# LSAmitR

<details>

* Version: 1.0-2
* GitHub: NA
* Source code: https://github.com/cran/LSAmitR
* Date/Publication: 2018-06-08 13:23:05 UTC
* Number of recursive dependencies: 152

Run `revdep_details(, "LSAmitR")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘LSAmitR-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Encoding: UTF-8
    > 
    > ### Name: Kapitel  1
    > ### Title: Kapitel 1: Testkonstruktion
    > ### Aliases: 'Kapitel 1'
    > 
    > ### ** Examples
    > 
    > library(TAM)
    Error in library(TAM) : there is no package called ‘TAM’
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      'BIFIEsurvey', 'TAM', 'miceadds', 'sirt', 'difR', 'kerdiest', 'mirt',
      'mitml'
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘TAM’
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 55 marked UTF-8 strings
    ```

# lspartition

<details>

* Version: 0.4
* GitHub: NA
* Source code: https://github.com/cran/lspartition
* Date/Publication: 2019-08-08 22:40:06 UTC
* Number of recursive dependencies: 38

Run `revdep_details(, "lspartition")` for more info

</details>

## In both

*   checking LazyData ... NOTE
    ```
      'LazyData' is specified without a 'data' directory
    ```

# ltmle

<details>

* Version: 1.2-0
* GitHub: https://github.com/joshuaschwab/ltmle
* Source code: https://github.com/cran/ltmle
* Date/Publication: 2020-03-13 23:00:07 UTC
* Number of recursive dependencies: 102

Run `revdep_details(, "ltmle")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘tmle’
    ```

# Luminescence

<details>

* Version: 0.9.13
* GitHub: https://github.com/R-Lum/Luminescence
* Source code: https://github.com/cran/Luminescence
* Date/Publication: 2021-06-03 16:40:02 UTC
* Number of recursive dependencies: 152

Run `revdep_details(, "Luminescence")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘RLumShiny’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.9Mb
      sub-directories of 1Mb or more:
        R      1.9Mb
        help   1.3Mb
        libs   3.4Mb
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘RLumShiny’
    ```

# M3Drop

<details>

* Version: 1.18.0
* GitHub: https://github.com/tallulandrews/M3Drop
* Source code: https://github.com/cran/M3Drop
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 205

Run `revdep_details(, "M3Drop")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘M3Drop-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: M3DropConvertData
    > ### Title: Convert Data to be suitable for M3Drop
    > ### Aliases: M3DropConvertData
    > ### Keywords: normalization
    > 
    > ### ** Examples
    > 
    ...
    > 
    > 	# monocle Object
    > 	require("monocle")
    Loading required package: monocle
    Warning in library(package, lib.loc = lib.loc, character.only = TRUE, logical.return = TRUE,  :
      there is no package called ‘monocle’
    > 	obj <- suppressWarnings(newCellDataSet(as.matrix(lognorm)))
    Error in newCellDataSet(as.matrix(lognorm)) : 
      could not find function "newCellDataSet"
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      'scater', 'monocle', 'Seurat'
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 13.3Mb
      sub-directories of 1Mb or more:
        doc  12.6Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Package in Depends field not imported from: ‘numDeriv’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    M3DropConvertData: no visible global function definition for ‘assays’
    NBumiConvertData: no visible global function definition for ‘assays’
    bg__filter_cells: no visible binding for global variable ‘x’
    bg__get_extreme_residuals: no visible binding for global variable ‘x’
    Undefined global functions or variables:
      assays x
    ```

*   checking Rd files ... NOTE
    ```
    prepare_Rd: NBumi_FSOther.Rd:8-10: Dropping empty section \usage
    prepare_Rd: NBumi_TradDE.Rd:7-8: Dropping empty section \usage
    ```

# maEndToEnd

<details>

* Version: 2.12.0
* GitHub: NA
* Source code: https://github.com/cran/maEndToEnd
* Date/Publication: 2021-05-21
* Number of recursive dependencies: 227

Run `revdep_details(, "maEndToEnd")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'oligoClasses', 'ArrayExpress', 'pd.hugene.1.0.st.v1',
      'hugene10sttranscriptcluster.db', 'oligo', 'arrayQualityMetrics',
      'topGO', 'ReactomePA', 'clusterProfiler', 'geneplotter',
      'genefilter', 'Rgraphviz', 'enrichplot'
    
    Depends: includes the non-default packages:
      'Biobase', 'oligoClasses', 'ArrayExpress', 'pd.hugene.1.0.st.v1',
      'hugene10sttranscriptcluster.db', 'oligo', 'arrayQualityMetrics',
      'limma', 'topGO', 'ReactomePA', 'clusterProfiler', 'gplots',
      'ggplot2', 'geneplotter', 'pheatmap', 'RColorBrewer', 'dplyr',
      'tidyr', 'stringr', 'matrixStats', 'genefilter', 'openxlsx',
      'Rgraphviz', 'enrichplot'
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# maic

<details>

* Version: 0.1.3
* GitHub: https://github.com/heorltd/maic
* Source code: https://github.com/cran/maic
* Date/Publication: 2021-05-11 06:50:05 UTC
* Number of recursive dependencies: 91

Run `revdep_details(, "maic")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘weights’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# MatrixGenerics

<details>

* Version: 1.4.0
* GitHub: https://github.com/Bioconductor/MatrixGenerics
* Source code: https://github.com/cran/MatrixGenerics
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 50

Run `revdep_details(, "MatrixGenerics")` for more info

</details>

## Newly broken

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 50 lines of output:
      `matrixStats_formals` not identical to `MatrixGenerics_default_method_formals`.
      Length mismatch: comparison on first 6 components
      ── Failure (test-api_compatibility.R:1242:2): rowProds works  ──────────────────
      `matrixStats_formals` not identical to `MatrixGenerics_default_method_formals`.
      Length mismatch: comparison on first 6 components
      ── Failure (test-api_compatibility.R:1262:2): rowQuantiles works  ──────────────
      `matrixStats_formals` not identical to `MatrixGenerics_default_method_formals`.
    ...
      ── Failure (test-api_compatibility.R:1518:2): rowWeightedSds works  ────────────
      `matrixStats_formals` not identical to `MatrixGenerics_default_method_formals`.
      Length mismatch: comparison on first 6 components
      ── Failure (test-api_compatibility.R:1539:2): rowWeightedVars works  ───────────
      `matrixStats_formals` not identical to `MatrixGenerics_default_method_formals`.
      Length mismatch: comparison on first 6 components
      
      [ FAIL 68 | WARN 0 | SKIP 0 | PASS 242 ]
      Error: Test failures
      Execution halted
    ```

## In both

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      ‘methods:::fromNextMethod’ ‘methods:::getGenericFromCall’
      See the note in ?`:::` about the use of this operator.
    There are ::: calls to the package's namespace in its code. A package
      almost never needs to use ::: for its own objects:
      ‘.load_next_suggested_package_to_search’
    ```

# matrixTests

<details>

* Version: 0.1.9
* GitHub: https://github.com/KKPMW/matrixTests
* Source code: https://github.com/cran/matrixTests
* Date/Publication: 2020-05-01 06:50:12 UTC
* Number of recursive dependencies: 133

Run `revdep_details(, "matrixTests")` for more info

</details>

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Complete output:
      > library(testthat)
      > library(matrixTests)
      > 
      > test_check("matrixTests")
      ══ Failed tests ════════════════════════════════════════════════════════════════
      ── Error (test-correct-cosinor.R:45:3): monte-carlo random testing gives equal results ──
      Error: there is no package called 'cosinor2'
    ...
          █
       1. ├─matrixTests:::base_cosinor(X, T, 10) test-correct-cosinor.R:45:2
       2. └─base::loadNamespace(x) test-correct-cosinor.R:20:4
       3.   └─base::withRestarts(stop(cond), retry_loadNamespace = function() NULL)
       4.     └─base:::withOneRestart(expr, restarts[[1L]])
       5.       └─base:::doWithOneRestart(return(expr), restart)
      
      [ FAIL 1 | WARN 10 | SKIP 0 | PASS 2189 ]
      Error: Test failures
      Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘cosinor2’
    ```

*   checking LazyData ... NOTE
    ```
      'LazyData' is specified without a 'data' directory
    ```

# mcmcsae

<details>

* Version: 0.6.0
* GitHub: NA
* Source code: https://github.com/cran/mcmcsae
* Date/Publication: 2021-01-20 08:40:02 UTC
* Number of recursive dependencies: 109

Run `revdep_details(, "mcmcsae")` for more info

</details>

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Complete output:
      > library(testthat)
      > test_check("mcmcsae")
      Loading required package: mcmcsae
      ══ Failed tests ════════════════════════════════════════════════════════════════
      ── Error (test_modelmatrix.R:6:1): (code run outside of `test_that()`) ─────────
      Error: there is no package called 'survey'
      Backtrace:
    ...
       1. └─base::library(survey) test_modelmatrix.R:6:0
      ── Error (test_prior_sampling.R:14:3): prior specification of regression component works ──
      Error: there is no package called 'survey'
      Backtrace:
          █
       1. └─base::library(survey) test_prior_sampling.R:14:2
      
      [ FAIL 2 | WARN 0 | SKIP 0 | PASS 304 ]
      Error: Test failures
      Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘survey’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 16.2Mb
      sub-directories of 1Mb or more:
        libs  14.8Mb
    ```

# MEAL

<details>

* Version: 1.22.0
* GitHub: NA
* Source code: https://github.com/cran/MEAL
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 218

Run `revdep_details(, "MEAL")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'MultiDataSet', 'minfi', 'Gviz', 'missMethyl', 'isva', 'SmartSVA'
    
    Packages suggested but not available for checking:
      'IlluminaHumanMethylationEPICanno.ilm10b2.hg19',
      'IlluminaHumanMethylation450kanno.ilmn12.hg19', 'minfiData'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# MEDseq

<details>

* Version: 1.2.1
* GitHub: https://github.com/Keefe-Murphy/MEDseq
* Source code: https://github.com/cran/MEDseq
* Date/Publication: 2021-01-06 19:30:03 UTC
* Number of recursive dependencies: 125

Run `revdep_details(, "MEDseq")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'TraMineR', 'WeightedCluster'
    
    Package suggested but not available for checking: ‘klaR’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# memes

<details>

* Version: 1.0.0
* GitHub: https://github.com/snystrom/memes
* Source code: https://github.com/cran/memes
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 147

Run `revdep_details(, "memes")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘universalmotif’
    
    Packages suggested but not available for checking: 'MotifDb', 'PMCMRplus'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# metabCombiner

<details>

* Version: 1.2.0
* GitHub: NA
* Source code: https://github.com/cran/metabCombiner
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 94

Run `revdep_details(, "metabCombiner")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘caret’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# MetabolicSurv

<details>

* Version: 1.1.2
* GitHub: https://github.com/OlajumokeEvangelina/MetabolicSurv
* Source code: https://github.com/cran/MetabolicSurv
* Date/Publication: 2021-06-11 08:30:02 UTC
* Number of recursive dependencies: 140

Run `revdep_details(, "MetabolicSurv")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'survminer', 'rms', 'Rdpack'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# metagene

<details>

* Version: 2.24.0
* GitHub: https://github.com/CharlesJB/metagene
* Source code: https://github.com/cran/metagene
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 125

Run `revdep_details(, "metagene")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'GenomicFeatures', 'ensembldb', 'EnsDb.Hsapiens.v86'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# metagenomeSeq

<details>

* Version: 1.34.0
* GitHub: https://github.com/nosson/metagenomeSeq
* Source code: https://github.com/cran/metagenomeSeq
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 140

Run `revdep_details(, "metagenomeSeq")` for more info

</details>

## Newly broken

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 50 lines of output:
      Loading required package: parallel
      
      Attaching package: 'BiocGenerics'
      
      The following objects are masked from 'package:parallel':
      
          clusterApply, clusterApplyLB, clusterCall, clusterEvalQ,
    ...
      ── Failure (test-norm.R:28:3): `cumNormStat` returns the correct value ─────────
      cumNormStat(lungData) not equal to 0.7014946.
      names for target but not for current
      ── Failure (test-norm.R:34:3): `cumNormStatFast` returns the correct value ─────
      cumNormStatFast(lungData) not equal to 0.7014946.
      names for target but not for current
      
      [ FAIL 2 | WARN 6 | SKIP 0 | PASS 13 ]
      Error: Test failures
      Execution halted
    ```

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘metagenomeSeq-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: biom2MRexperiment
    > ### Title: Biom to MRexperiment objects
    > ### Aliases: biom2MRexperiment
    > 
    > ### ** Examples
    > 
    > 
    > library(biomformat)
    Error in library(biomformat) : there is no package called ‘biomformat’
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      'annotate', 'biomformat', 'interactiveDisplay', 'IHW'
    ```

# metamicrobiomeR

<details>

* Version: 1.2
* GitHub: https://github.com/nhanhocu/metamicrobiomeR
* Source code: https://github.com/cran/metamicrobiomeR
* Date/Publication: 2020-11-09 11:20:05 UTC
* Number of recursive dependencies: 135

Run `revdep_details(, "metamicrobiomeR")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'gamlss', 'meta', 'zCompositions', 'compositions'
    
    Package suggested but not available for checking: ‘caret’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# MetaNeighbor

<details>

* Version: 1.12.0
* GitHub: NA
* Source code: https://github.com/cran/MetaNeighbor
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 90

Run `revdep_details(, "MetaNeighbor")` for more info

</details>

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘MetaNeighbor-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: plotUpset
    > ### Title: Plot Upset plot showing how replicability depends on input
    > ###   dataset.
    > ### Aliases: plotUpset
    > 
    > ### ** Examples
    > 
    ...
    > var_genes = variableGenes(dat = mn_data, exp_labels = mn_data$study_id)
    > celltype_NV = MetaNeighborUS(var_genes = var_genes,
    +                              dat = mn_data,
    +                              study_id = mn_data$study_id,
    +                              cell_type = mn_data$cell_type,
    +                              fast_version = TRUE, one_vs_best = TRUE)
    Error in dimnames(y) <- list(rownames, colnames) : 
      length of 'dimnames' [1] not equal to array extent
    Calls: MetaNeighborUS ... MetaNeighborUSLowMem -> normalize_cols -> <Anonymous>
    Execution halted
    ```

## In both

*   checking LazyData ... WARNING
    ```
      LazyData DB of 7.1 MB without LazyDataCompression set
      See §1.1.6 of 'Writing R Extensions'
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 15.8Mb
      sub-directories of 1Mb or more:
        data  15.0Mb
    ```

# metavizr

<details>

* Version: 1.15.0
* GitHub: NA
* Source code: https://github.com/cran/metavizr
* Date/Publication: 2021-03-19
* Number of recursive dependencies: 199

Run `revdep_details(, "metavizr")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'metagenomeSeq', 'epivizr', 'epivizrData', 'epivizrStandalone',
      'phyloseq'
    
    Packages suggested but not available for checking:
      'msd16s', 'etec16s', 'curatedMetagenomicData'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# MethReg

<details>

* Version: 1.2.1
* GitHub: https://github.com/TransBioInfoLab/MethReg
* Source code: https://github.com/cran/MethReg
* Date/Publication: 2021-05-30
* Number of recursive dependencies: 234

Run `revdep_details(, "MethReg")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'ggpubr', 'sesameData'
    
    Packages suggested but not available for checking:
      'TFBSTools', 'motifmatchr', 'biomaRt', 'dorothea', 'viper', 'sesame'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# methrix

<details>

* Version: 1.6.0
* GitHub: https://github.com/CompEpigen/methrix
* Source code: https://github.com/cran/methrix
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 160

Run `revdep_details(, "methrix")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘HDF5Array’
    
    Packages suggested but not available for checking:
      'DSS', 'bsseq', 'MafDb.1Kgenomes.phase3.GRCh38',
      'MafDb.1Kgenomes.phase3.hs37d5', 'GenomicScores'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# MethylAid

<details>

* Version: 1.26.0
* GitHub: NA
* Source code: https://github.com/cran/MethylAid
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 169

Run `revdep_details(, "MethylAid")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘minfi’
    
    Packages suggested but not available for checking:
      'MethylAidData', 'minfiData', 'minfiDataEPIC'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# methylationArrayAnalysis

<details>

* Version: 1.16.0
* GitHub: NA
* Source code: https://github.com/cran/methylationArrayAnalysis
* Date/Publication: 2021-05-21
* Number of recursive dependencies: 215

Run `revdep_details(, "methylationArrayAnalysis")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'minfi', 'IlluminaHumanMethylation450kanno.ilmn12.hg19',
      'IlluminaHumanMethylation450kmanifest', 'missMethyl', 'minfiData',
      'Gviz', 'DMRcate', 'FlowSorted.Blood.450k'
    
    Depends: includes the non-default packages:
      'knitr', 'rmarkdown', 'BiocStyle', 'limma', 'minfi',
      'IlluminaHumanMethylation450kanno.ilmn12.hg19',
      'IlluminaHumanMethylation450kmanifest', 'RColorBrewer', 'missMethyl',
      'matrixStats', 'minfiData', 'Gviz', 'DMRcate', 'stringr',
      'FlowSorted.Blood.450k'
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# methylumi

<details>

* Version: 2.38.0
* GitHub: https://github.com/seandavi/methylumi
* Source code: https://github.com/cran/methylumi
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 191

Run `revdep_details(, "methylumi")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'FDb.InfiniumMethylation.hg19', 'minfi', 'annotate', 'genefilter'
    
    Packages suggested but not available for checking:
      'lumi', 'SQN', 'methyAnalysis',
      'IlluminaHumanMethylation450kanno.ilmn12.hg19',
      'FDb.InfiniumMethylation.hg18', 'Homo.sapiens'
    
    Depends: includes the non-default packages:
      'Biobase', 'scales', 'reshape2', 'ggplot2', 'matrixStats',
      'FDb.InfiniumMethylation.hg19', 'minfi'
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# MFHD

<details>

* Version: 0.0.1
* GitHub: NA
* Source code: https://github.com/cran/MFHD
* Date/Publication: 2013-10-23 23:50:20
* Number of recursive dependencies: 101

Run `revdep_details(, "MFHD")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘depth’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# mgcViz

<details>

* Version: 0.1.6
* GitHub: https://github.com/mfasiolo/mgcViz
* Source code: https://github.com/cran/mgcViz
* Date/Publication: 2020-03-04 15:10:02 UTC
* Number of recursive dependencies: 109

Run `revdep_details(, "mgcViz")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.4Mb
      sub-directories of 1Mb or more:
        doc   4.4Mb
    ```

*   checking LazyData ... NOTE
    ```
      'LazyData' is specified without a 'data' directory
    ```

# MHTcop

<details>

* Version: 0.1.1
* GitHub: NA
* Source code: https://github.com/cran/MHTcop
* Date/Publication: 2019-01-21 16:10:03 UTC
* Number of recursive dependencies: 41

Run `revdep_details(, "MHTcop")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘copula’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# miclust

<details>

* Version: 1.2.6
* GitHub: NA
* Source code: https://github.com/cran/miclust
* Date/Publication: 2021-03-10 19:30:02 UTC
* Number of recursive dependencies: 72

Run `revdep_details(, "miclust")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘doBy’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# microbiomeExplorer

<details>

* Version: 1.2.0
* GitHub: NA
* Source code: https://github.com/cran/microbiomeExplorer
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 203

Run `revdep_details(, "microbiomeExplorer")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'metagenomeSeq', 'DESeq2', 'biomformat'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# MIGSA

<details>

* Version: 1.16.0
* GitHub: https://github.com/jcrodriguez1989/MIGSA
* Source code: https://github.com/cran/MIGSA
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 120

Run `revdep_details(, "MIGSA")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'GO.db', 'GOstats', 'GSEABase', 'org.Hs.eg.db', 'RBGL', 'Rgraphviz'
    
    Package suggested but not available for checking: ‘mGSZ’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# miloR

<details>

* Version: 1.0.0
* GitHub: https://github.com/MarioniLab/miloR
* Source code: https://github.com/cran/miloR
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 194

Run `revdep_details(, "miloR")` for more info

</details>

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 50 lines of output:
      Backtrace:
          █
       1. └─base::library(scran) test_classes.R:6:0
      ── Error (test_countCells.R:6:1): (code run outside of `test_that()`) ──────────
      Error: there is no package called 'scran'
      Backtrace:
          █
    ...
       1. └─base::library(scran) test_testDiffExp.R:6:0
      ── Error (test_testNhoods.R:6:1): (code run outside of `test_that()`) ──────────
      Error: there is no package called 'scran'
      Backtrace:
          █
       1. └─base::library(scran) test_testNhoods.R:6:0
      
      [ FAIL 16 | WARN 0 | SKIP 0 | PASS 0 ]
      Error: Test failures
      Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      'scater', 'scran', 'MouseGastrulationData'
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.8Mb
      sub-directories of 1Mb or more:
        data   1.8Mb
        doc    3.3Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported object imported by a ':::' call: ‘S4Vectors:::disableValidity’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking Rd files ... NOTE
    ```
    prepare_Rd: findNhoodGroupMarkers.Rd:79-81: Dropping empty section \examples
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘scran’
    ```

# MinimumDistance

<details>

* Version: 1.36.0
* GitHub: NA
* Source code: https://github.com/cran/MinimumDistance
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 78

Run `revdep_details(, "MinimumDistance")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'VanillaICE', 'oligoClasses'
    
    Package which this enhances but not available for checking: ‘doSNOW’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# mixOmics

<details>

* Version: 6.16.1
* GitHub: https://github.com/mixOmicsTeam/mixOmics
* Source code: https://github.com/cran/mixOmics
* Date/Publication: 2021-06-22
* Number of recursive dependencies: 109

Run `revdep_details(, "mixOmics")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 12.5Mb
      sub-directories of 1Mb or more:
        R      1.9Mb
        data   3.3Mb
        doc    6.1Mb
    ```

# mnem

<details>

* Version: 1.8.0
* GitHub: https://github.com/cbg-ethz/mnem
* Source code: https://github.com/cran/mnem
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 156

Run `revdep_details(, "mnem")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'Rgraphviz', 'Linnorm'
    
    Package suggested but not available for checking: ‘epiNEM’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# moanin

<details>

* Version: 1.0.0
* GitHub: NA
* Source code: https://github.com/cran/moanin
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 116

Run `revdep_details(, "moanin")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'topGO', 'ClusterR'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# MoEClust

<details>

* Version: 1.4.0
* GitHub: https://github.com/Keefe-Murphy/MoEClust
* Source code: https://github.com/cran/MoEClust
* Date/Publication: 2021-06-22 04:00:05 UTC
* Number of recursive dependencies: 73

Run `revdep_details(, "MoEClust")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘MoEClust-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: noise_vol
    > ### Title: Approximate Hypervolume Estimate
    > ### Aliases: noise_vol
    > ### Keywords: control
    > 
    > ### ** Examples
    > 
    ...
    [1,] 5.295856 9.421848 48.08101 16.4601 120.8976
    
    attr(,"Method")
    [1] "hypvol"
    attr(,"class")
    [1] "NoiseVol"
    > 
    > noise_vol(ais[,3:7], reciprocal=FALSE, method="convexhull")
    Error: Use of the convexhull 'method' option requires loading the 'geometry'library
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘geometry’
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘geometry’
    ```

# monocle

<details>

* Version: 2.20.0
* GitHub: NA
* Source code: https://github.com/cran/monocle
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 210

Run `revdep_details(, "monocle")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'qlcMatrix', 'biocViews'
    
    Packages suggested but not available for checking:
      'destiny', 'Seurat', 'scater'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# MOSim

<details>

* Version: 1.6.0
* GitHub: https://github.com/Neurergus/MOSim
* Source code: https://github.com/cran/MOSim
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 75

Run `revdep_details(, "MOSim")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.0Mb
      sub-directories of 1Mb or more:
        data   4.2Mb
    ```

# motifbreakR

<details>

* Version: 2.6.0
* GitHub: https://github.com/Simon-Coetzee/motifbreakR
* Source code: https://github.com/cran/motifbreakR
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 150

Run `revdep_details(, "motifbreakR")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'MotifDb', 'VariantAnnotation', 'motifStack', 'Gviz'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# MPAgenomics

<details>

* Version: 1.2.3
* GitHub: NA
* Source code: https://github.com/cran/MPAgenomics
* Date/Publication: 2021-03-30 15:50:07 UTC
* Number of recursive dependencies: 51

Run `revdep_details(, "MPAgenomics")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'HDPenReg', 'spikeslab'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# mrfDepth

<details>

* Version: 1.0.13
* GitHub: https://github.com/PSegaert/mrfDepth
* Source code: https://github.com/cran/mrfDepth
* Date/Publication: 2020-08-26 16:10:33 UTC
* Number of recursive dependencies: 47

Run `revdep_details(, "mrfDepth")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘geometry’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# msImpute

<details>

* Version: 1.2.0
* GitHub: https://github.com/DavisLaboratory/msImpute
* Source code: https://github.com/cran/msImpute
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 103

Run `revdep_details(, "msImpute")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘scran’
    
    Packages suggested but not available for checking:
      'ComplexHeatmap', 'imputeLCMD'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# msqrob2

<details>

* Version: 1.0.0
* GitHub: https://github.com/statOmics/msqrob2
* Source code: https://github.com/cran/msqrob2
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 174

Run `revdep_details(, "msqrob2")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘QFeatures’
    
    Packages suggested but not available for checking: 'MSnbase', 'MsCoreUtils'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# MultiBD

<details>

* Version: 0.2.0
* GitHub: NA
* Source code: https://github.com/cran/MultiBD
* Date/Publication: 2016-12-05 18:28:46
* Number of recursive dependencies: 71

Run `revdep_details(, "MultiBD")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  9.5Mb
      sub-directories of 1Mb or more:
        libs   7.9Mb
    ```

# multiviewtest

<details>

* Version: 2.0.0
* GitHub: NA
* Source code: https://github.com/cran/multiviewtest
* Date/Publication: 2019-08-27 06:10:12 UTC
* Number of recursive dependencies: 18

Run `revdep_details(, "multiviewtest")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘randnet’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# muscat

<details>

* Version: 1.6.0
* GitHub: https://github.com/HelenaLC/muscat
* Source code: https://github.com/cran/muscat
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 214

Run `revdep_details(, "muscat")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'ComplexHeatmap', 'DESeq2', 'glmmTMB', 'scater', 'scuttle'
    
    Packages suggested but not available for checking:
      'countsimQC', 'ExperimentHub', 'iCOBRA'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# muscData

<details>

* Version: 1.6.0
* GitHub: https://github.com/HelenaLC/muscData
* Source code: https://github.com/cran/muscData
* Date/Publication: 2021-05-20
* Number of recursive dependencies: 247

Run `revdep_details(, "muscData")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘ExperimentHub’
    
    Packages suggested but not available for checking:
      'DropletUtils', 'muscat', 'scater', 'scds', 'Seurat'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# nandb

<details>

* Version: 2.1.0
* GitHub: https://github.com/rorynolan/nandb
* Source code: https://github.com/cran/nandb
* Date/Publication: 2021-05-16 22:40:06 UTC
* Number of recursive dependencies: 107

Run `revdep_details(, "nandb")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'autothresholdr', 'detrendr', 'filesstrings'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# NanoStringDiff

<details>

* Version: 1.22.0
* GitHub: NA
* Source code: https://github.com/cran/NanoStringDiff
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 50

Run `revdep_details(, "NanoStringDiff")` for more info

</details>

## In both

*   checking for missing documentation entries ... WARNING
    ```
    Undocumented code objects:
      ‘compute.baseSigma’ ‘est.dispersion’ ‘fun5’ ‘glmfit.OneGroup’
      ‘glmfit.full’ ‘glmfit.reduce’ ‘rnegbinom’
    All user-level objects in a package should have documentation entries.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking installed files from ‘inst/doc’ ... NOTE
    ```
    The following files look like leftovers/mistakes:
      ‘NanoStringDiff.log’
    Please remove them from your package.
    ```

# nearBynding

<details>

* Version: 1.2.0
* GitHub: NA
* Source code: https://github.com/cran/nearBynding
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 118

Run `revdep_details(, "nearBynding")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'TxDb.Hsapiens.UCSC.hg19.knownGene',
      'TxDb.Hsapiens.UCSC.hg38.knownGene'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# nempi

<details>

* Version: 1.0.0
* GitHub: https://github.com/cbg-ethz/nempi
* Source code: https://github.com/cran/nempi
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 112

Run `revdep_details(, "nempi")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'mnem', 'epiNEM'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# neurobase

<details>

* Version: 1.32.1
* GitHub: https://github.com/muschellij2/neurobase
* Source code: https://github.com/cran/neurobase
* Date/Publication: 2021-04-07 20:50:02 UTC
* Number of recursive dependencies: 105

Run `revdep_details(, "neurobase")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘oro.nifti’
    
    Package suggested but not available for checking: ‘brainR’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# NormalyzerDE

<details>

* Version: 1.10.0
* GitHub: https://github.com/ComputationalProteomics/NormalyzerDE
* Source code: https://github.com/cran/NormalyzerDE
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 156

Run `revdep_details(, "NormalyzerDE")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'vsn', 'RcmdrMisc'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# obfuscatoR

<details>

* Version: 0.2.1
* GitHub: https://github.com/edsandorf/obfuscatoR
* Source code: https://github.com/cran/obfuscatoR
* Date/Publication: 2020-11-19 08:20:09 UTC
* Number of recursive dependencies: 54

Run `revdep_details(, "obfuscatoR")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘Rfast’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# omicplotR

<details>

* Version: 1.12.0
* GitHub: NA
* Source code: https://github.com/cran/omicplotR
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 87

Run `revdep_details(, "omicplotR")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'ALDEx2', 'compositions', 'zCompositions'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# omicsPrint

<details>

* Version: 1.12.0
* GitHub: NA
* Source code: https://github.com/cran/omicsPrint
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 128

Run `revdep_details(, "omicsPrint")` for more info

</details>

## In both

*   checking LazyData ... WARNING
    ```
      LazyData DB of 7.3 MB without LazyDataCompression set
      See §1.1.6 of 'Writing R Extensions'
    ```

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      'VariantAnnotation', 'FDb.InfiniumMethylation.hg19'
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 16.6Mb
      sub-directories of 1Mb or more:
        data  15.3Mb
        doc    1.2Mb
    ```

# omicwas

<details>

* Version: 0.8.0
* GitHub: https://github.com/fumi-github/omicwas
* Source code: https://github.com/cran/omicwas
* Date/Publication: 2020-10-08 12:50:03 UTC
* Number of recursive dependencies: 64

Run `revdep_details(, "omicwas")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.0Mb
      sub-directories of 1Mb or more:
        data   6.8Mb
    ```

# OptimalDesign

<details>

* Version: 1.0.1
* GitHub: NA
* Source code: https://github.com/cran/OptimalDesign
* Date/Publication: 2019-12-02 08:50:07 UTC
* Number of recursive dependencies: 55

Run `revdep_details(, "OptimalDesign")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Packages which this enhances but not available for checking:
      'gurobi', 'slam'
    ```

*   checking LazyData ... NOTE
    ```
      'LazyData' is specified without a 'data' directory
    ```

# OUTRIDER

<details>

* Version: 1.10.0
* GitHub: https://github.com/gagneurlab/OUTRIDER
* Source code: https://github.com/cran/OUTRIDER
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 172

Run `revdep_details(, "OUTRIDER")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'GenomicFeatures', 'DESeq2'
    
    Packages suggested but not available for checking:
      'TxDb.Hsapiens.UCSC.hg19.knownGene', 'org.Hs.eg.db'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# pandaR

<details>

* Version: 1.24.0
* GitHub: NA
* Source code: https://github.com/cran/pandaR
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 49

Run `revdep_details(, "pandaR")` for more info

</details>

## In both

*   checking package subdirectories ... WARNING
    ```
    Invalid citation information in ‘inst/CITATION’:
      Error in tools:::.parse_CITATION_file(file, meta$Encoding): non-ASCII input in a CITATION file without a declared encoding
    ```

*   checking LazyData ... WARNING
    ```
      LazyData DB of 9.0 MB without LazyDataCompression set
      See §1.1.6 of 'Writing R Extensions'
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 17.0Mb
      sub-directories of 1Mb or more:
        data  16.6Mb
    ```

*   checking S3 generic/method consistency ... NOTE
    ```
    Found the following apparent S3 methods exported but not registered:
      summary.panda
    See section ‘Registering S3 methods’ in the ‘Writing R Extensions’
    manual.
    ```

*   checking Rd \usage sections ... NOTE
    ```
    S3 methods shown with full name in documentation object 'summary.panda':
      ‘summary.panda’
    
    The \usage entries for S3 methods should use the \method markup and not
    their full name.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

# PathoStat

<details>

* Version: 1.18.0
* GitHub: https://github.com/mani2012/PathoStat
* Source code: https://github.com/cran/PathoStat
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 196

Run `revdep_details(, "PathoStat")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'phyloseq', 'DESeq2', 'ComplexHeatmap'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# pcaMethods

<details>

* Version: 1.84.0
* GitHub: https://github.com/hredestig/pcamethods
* Source code: https://github.com/cran/pcaMethods
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 35

Run `revdep_details(, "pcaMethods")` for more info

</details>

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘ade4’
    ```

# PDATK

<details>

* Version: 1.0.2
* GitHub: https://github.com/bhklab/PDATK
* Source code: https://github.com/cran/PDATK
* Date/Publication: 2021-06-24
* Number of recursive dependencies: 263

Run `revdep_details(, "PDATK")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'ggplotify', 'CoreGx', 'caret', 'survminer', 'piano', 'survcomp',
      'genefu', 'verification'
    
    Package suggested but not available for checking: ‘HDF5Array’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# peakPick

<details>

* Version: 0.11
* GitHub: NA
* Source code: https://github.com/cran/peakPick
* Date/Publication: 2015-12-04 15:40:34
* Number of recursive dependencies: 19

Run `revdep_details(, "peakPick")` for more info

</details>

## In both

*   checking LazyData ... NOTE
    ```
      'LazyData' is specified without a 'data' directory
    ```

# phosphonormalizer

<details>

* Version: 1.16.0
* GitHub: NA
* Source code: https://github.com/cran/phosphonormalizer
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 46

Run `revdep_details(, "phosphonormalizer")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package which this enhances but not available for checking: ‘MSnbase’
    ```

# Pigengene

<details>

* Version: 1.18.0
* GitHub: NA
* Source code: https://github.com/cran/Pigengene
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 139

Run `revdep_details(, "Pigengene")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'C50', 'Rgraphviz', 'WGCNA', 'GO.db'
    
    Packages suggested but not available for checking:
      'org.Hs.eg.db', 'org.Mm.eg.db', 'biomaRt', 'energy'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# pipeComp

<details>

* Version: 1.2.0
* GitHub: https://github.com/plger/pipeComp
* Source code: https://github.com/cran/pipeComp
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 196

Run `revdep_details(, "pipeComp")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'ComplexHeatmap', 'Seurat', 'scran', 'intrinsicDimension', 'scater',
      'circlize'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# pmp

<details>

* Version: 1.4.0
* GitHub: NA
* Source code: https://github.com/cran/pmp
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 97

Run `revdep_details(, "pmp")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘missForest’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# PrecisionTrialDrawer

<details>

* Version: 1.8.0
* GitHub: NA
* Source code: https://github.com/cran/PrecisionTrialDrawer
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 128

Run `revdep_details(, "PrecisionTrialDrawer")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'biomaRt', 'brglm'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# ProteoMM

<details>

* Version: 1.10.0
* GitHub: NA
* Source code: https://github.com/cran/ProteoMM
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 96

Run `revdep_details(, "ProteoMM")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘biomaRt’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# QDNAseq

<details>

* Version: 1.28.0
* GitHub: https://github.com/ccagc/QDNAseq
* Source code: https://github.com/cran/QDNAseq
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 75

Run `revdep_details(, "QDNAseq")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘future’
      All declared Imports should be used.
    ```

# QFeatures

<details>

* Version: 1.2.0
* GitHub: https://github.com/RforMassSpectrometry/QFeatures
* Source code: https://github.com/cran/QFeatures
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 132

Run `revdep_details(, "QFeatures")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘MsCoreUtils’
    
    Packages suggested but not available for checking:
      'HDF5Array', 'vsn', 'imputeLCMD'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# r2dRue

<details>

* Version: 1.0.4
* GitHub: NA
* Source code: https://github.com/cran/r2dRue
* Date/Publication: 2013-06-28 12:22:19
* Number of recursive dependencies: 4

Run `revdep_details(, "r2dRue")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' call to ‘rgdal’ which was already attached by Depends.
      Please remove these calls from your code.
    Packages in Depends field not imported from:
      ‘matrixStats’ ‘rgdal’ ‘sp’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

# RaceID

<details>

* Version: 0.2.3
* GitHub: NA
* Source code: https://github.com/cran/RaceID
* Date/Publication: 2021-04-08 06:10:08 UTC
* Number of recursive dependencies: 143

Run `revdep_details(, "RaceID")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'FateID', 'fpc'
    
    Packages suggested but not available for checking: 'batchelor', 'DESeq2'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# rADA

<details>

* Version: 1.1.9
* GitHub: NA
* Source code: https://github.com/cran/rADA
* Date/Publication: 2021-03-23 19:40:06 UTC
* Number of recursive dependencies: 109

Run `revdep_details(, "rADA")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.7Mb
      sub-directories of 1Mb or more:
        doc   5.5Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘Hmisc’ ‘car’ ‘dplyr’ ‘grid’ ‘gridExtra’ ‘knitr’ ‘lmerTest’ ‘tidyr’
      All declared Imports should be used.
    ```

# RadioGx

<details>

* Version: 1.2.0
* GitHub: NA
* Source code: https://github.com/cran/RadioGx
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 132

Run `revdep_details(, "RadioGx")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'CoreGx', 'magicaxis'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# ramr

<details>

* Version: 1.0.2
* GitHub: https://github.com/BBCG/ramr
* Source code: https://github.com/cran/ramr
* Date/Publication: 2021-06-24
* Number of recursive dependencies: 153

Run `revdep_details(, "ramr")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      'annotatr', 'org.Hs.eg.db', 'TxDb.Hsapiens.UCSC.hg19.knownGene'
    ```

# RandomForestsGLS

<details>

* Version: 0.1.2
* GitHub: https://github.com/ArkajyotiSaha/RandomForestsGLS
* Source code: https://github.com/cran/RandomForestsGLS
* Date/Publication: 2021-01-31 00:50:09 UTC
* Number of recursive dependencies: 66

Run `revdep_details(, "RandomForestsGLS")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘BRISC’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# reconsi

<details>

* Version: 1.4.0
* GitHub: https://github.com/CenterForStatistics-UGent/reconsi
* Source code: https://github.com/cran/reconsi
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 93

Run `revdep_details(, "reconsi")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘phyloseq’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# regsem

<details>

* Version: 1.8.0
* GitHub: NA
* Source code: https://github.com/cran/regsem
* Date/Publication: 2021-06-03 15:00:02 UTC
* Number of recursive dependencies: 140

Run `revdep_details(, "regsem")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking: 'caret', 'semPlot'
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.6Mb
      sub-directories of 1Mb or more:
        libs   5.2Mb
    ```

# rhierbaps

<details>

* Version: 1.1.3
* GitHub: https://github.com/gtonkinhill/rhierbaps
* Source code: https://github.com/cran/rhierbaps
* Date/Publication: 2019-12-11 14:20:05 UTC
* Number of recursive dependencies: 91

Run `revdep_details(, "rhierbaps")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking: 'ggtree', 'phytools'
    ```

*   checking LazyData ... NOTE
    ```
      'LazyData' is specified without a 'data' directory
    ```

# RiboDiPA

<details>

* Version: 1.0.0
* GitHub: NA
* Source code: https://github.com/cran/RiboDiPA
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 142

Run `revdep_details(, "RiboDiPA")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'GenomicFeatures', 'DESeq2'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# RJcluster

<details>

* Version: 3.0.0
* GitHub: NA
* Source code: https://github.com/cran/RJcluster
* Date/Publication: 2021-07-07 21:10:02 UTC
* Number of recursive dependencies: 53

Run `revdep_details(, "RJcluster")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.2Mb
      sub-directories of 1Mb or more:
        libs   5.7Mb
    ```

# RNAmodR

<details>

* Version: 1.6.0
* GitHub: https://github.com/FelixErnst/RNAmodR
* Source code: https://github.com/cran/RNAmodR
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 190

Run `revdep_details(, "RNAmodR")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'GenomicFeatures', 'Gviz'
    
    Package suggested but not available for checking: ‘RNAmodR.Data’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# RnBeads

<details>

* Version: 2.10.0
* GitHub: NA
* Source code: https://github.com/cran/RnBeads
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 264

Run `revdep_details(, "RnBeads")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘methylumi’
    
    Packages suggested but not available for checking:
      'Category', 'GOstats', 'Gviz',
      'IlluminaHumanMethylation450kmanifest', 'annotate', 'biomaRt',
      'ggbio', 'isva', 'minfi', 'org.Hs.eg.db', 'org.Mm.eg.db',
      'org.Rn.eg.db', 'sva', 'wateRmelon', 'GLAD',
      'IlluminaHumanMethylation450kanno.ilmn12.hg19', 'missMethyl',
      'MethylSeekR', 'sesame'
    
    Depends: includes the non-default packages:
      'BiocGenerics', 'S4Vectors', 'GenomicRanges', 'MASS', 'cluster',
      'ff', 'fields', 'ggplot2', 'gplots', 'gridExtra', 'limma',
      'matrixStats', 'illuminaio', 'methylumi', 'plyr'
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# robustlm

<details>

* Version: 0.1.0
* GitHub: NA
* Source code: https://github.com/cran/robustlm
* Date/Publication: 2021-03-22 15:40:02 UTC
* Number of recursive dependencies: 20

Run `revdep_details(, "robustlm")` for more info

</details>

## In both

*   checking LazyData ... NOTE
    ```
      'LazyData' is specified without a 'data' directory
    ```

# scDataviz

<details>

* Version: 1.2.0
* GitHub: https://github.com/kevinblighe/scDataviz
* Source code: https://github.com/cran/scDataviz
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 182

Run `revdep_details(, "scDataviz")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'flowCore', 'Seurat'
    
    Package suggested but not available for checking: ‘PCAtools’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# scDHA

<details>

* Version: 1.1.1
* GitHub: https://github.com/duct317/scDHA
* Source code: https://github.com/cran/scDHA
* Date/Publication: 2021-06-10 22:00:02 UTC
* Number of recursive dependencies: 68

Run `revdep_details(, "scDHA")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘torch’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# scHOT

<details>

* Version: 1.4.0
* GitHub: NA
* Source code: https://github.com/cran/scHOT
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 97

Run `revdep_details(, "scHOT")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘scater’
    ```

# scmap

<details>

* Version: 1.14.0
* GitHub: https://github.com/hemberg-lab/scmap
* Source code: https://github.com/cran/scmap
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 74

Run `revdep_details(, "scmap")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.1Mb
      sub-directories of 1Mb or more:
        data   2.6Mb
        doc    1.1Mb
        libs   2.0Mb
    ```

*   checking top-level files ... NOTE
    ```
    File
      LICENSE
    is not mentioned in the DESCRIPTION file.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘Biobase’
      All declared Imports should be used.
    ```

# scone

<details>

* Version: 1.16.0
* GitHub: https://github.com/YosefLab/scone
* Source code: https://github.com/cran/scone
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 248

Run `revdep_details(, "scone")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'compositions', 'fpc', 'mixtools', 'rhdf5', 'RUVSeq'
    
    Packages suggested but not available for checking:
      'NMF', 'scran', 'scRNAseq', 'BatchJobs', 'splatter', 'scater'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# scp

<details>

* Version: 1.2.0
* GitHub: https://github.com/UCLouvain-CBIO/scp
* Source code: https://github.com/cran/scp
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 166

Run `revdep_details(, "scp")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'QFeatures', 'MsCoreUtils'
    
    Packages suggested but not available for checking:
      'BiocCheck', 'scater', 'sva', 'vsn'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# scPCA

<details>

* Version: 1.6.2
* GitHub: https://github.com/PhilBoileau/scPCA
* Source code: https://github.com/cran/scPCA
* Date/Publication: 2021-05-27
* Number of recursive dependencies: 184

Run `revdep_details(, "scPCA")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'Rdpack', 'elasticnet'
    
    Packages suggested but not available for checking: 'ggpubr', 'splatter'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# sctransform

<details>

* Version: 0.3.2
* GitHub: https://github.com/ChristophH/sctransform
* Source code: https://github.com/cran/sctransform
* Date/Publication: 2020-12-16 14:40:03 UTC
* Number of recursive dependencies: 67

Run `revdep_details(, "sctransform")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package which this enhances but not available for checking: ‘glmGamPoi’
    ```

# seagull

<details>

* Version: 1.1.0
* GitHub: https://github.com/jklosa/seagull
* Source code: https://github.com/cran/seagull
* Date/Publication: 2021-04-20 12:10:02 UTC
* Number of recursive dependencies: 46

Run `revdep_details(, "seagull")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 11.0Mb
      sub-directories of 1Mb or more:
        libs  10.1Mb
    ```

# seeds

<details>

* Version: 0.9.1
* GitHub: https://github.com/Newmi1988/seeds
* Source code: https://github.com/cran/seeds
* Date/Publication: 2020-07-14 00:00:02 UTC
* Number of recursive dependencies: 87

Run `revdep_details(, "seeds")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘rsbml’
    ```

# sensobol

<details>

* Version: 1.0.2
* GitHub: https://github.com/arnaldpuy/sensobol
* Source code: https://github.com/cran/sensobol
* Date/Publication: 2021-05-30 08:10:03 UTC
* Number of recursive dependencies: 80

Run `revdep_details(, "sensobol")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'randtoolbox', 'Rdpack', 'Rfast'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# sesame

<details>

* Version: 1.10.4
* GitHub: https://github.com/zwdzwd/sesame
* Source code: https://github.com/cran/sesame
* Date/Publication: 2021-06-06
* Number of recursive dependencies: 188

Run `revdep_details(, "sesame")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘sesameData’
    
    Packages suggested but not available for checking:
      'IlluminaHumanMethylation450kmanifest', 'minfi',
      'FlowSorted.CordBloodNorway.450k', 'FlowSorted.Blood.450k',
      'HDF5Array'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# Seurat

<details>

* Version: 4.0.3
* GitHub: https://github.com/satijalab/seurat
* Source code: https://github.com/cran/Seurat
* Date/Publication: 2021-06-10 21:20:02 UTC
* Number of recursive dependencies: 240

Run `revdep_details(, "Seurat")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      'MAST', 'DESeq2', 'Rfast2', 'monocle', 'metap', 'mixtools'
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 12.4Mb
      sub-directories of 1Mb or more:
        R      1.8Mb
        libs   9.9Mb
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘Signac’
    ```

# SGP

<details>

* Version: 1.9-5.0
* GitHub: https://github.com/CenterForAssessment/SGP
* Source code: https://github.com/cran/SGP
* Date/Publication: 2020-01-30 22:40:09 UTC
* Number of recursive dependencies: 109

Run `revdep_details(, "SGP")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘randomNames’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# shinyMethyl

<details>

* Version: 1.28.0
* GitHub: NA
* Source code: https://github.com/cran/shinyMethyl
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 154

Run `revdep_details(, "shinyMethyl")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'minfi', 'IlluminaHumanMethylation450kmanifest'
    
    Package suggested but not available for checking: ‘minfiData’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# SIAMCAT

<details>

* Version: 1.12.0
* GitHub: NA
* Source code: https://github.com/cran/SIAMCAT
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 185

Run `revdep_details(, "SIAMCAT")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘phyloseq’
    
    Packages suggested but not available for checking: 'optparse', 'ggpubr'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# SICtools

<details>

* Version: 1.22.0
* GitHub: NA
* Source code: https://github.com/cran/SICtools
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 40

Run `revdep_details(, "SICtools")` for more info

</details>

## In both

*   checking whether package ‘SICtools’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      errmod.c:79:8: warning: self-comparison always evaluates to false [-Wtautological-compare]
      md5.c:155:26: warning: argument to ‘sizeof’ in ‘memset’ call is the same expression as the destination; did you mean to dereference it? [-Wsizeof-pointer-memaccess]
      Warning: replacing previous import ‘plyr::count’ by ‘matrixStats::count’ when loading ‘SICtools’
    See ‘/scratch/henrik/revdepcheck.extras/matrixStats/revdep/checks/SICtools/new/SICtools.Rcheck/00install.out’ for details.
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      'Rsamtools', 'doParallel', 'Biostrings', 'stringr', 'matrixStats',
      'plyr', 'GenomicRanges', 'IRanges'
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .BBSoptions
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    .indelDiffFunc: no visible global function definition for ‘fisher.test’
    .indelDiffFunc: no visible global function definition for ‘dist’
    indelDiff: no visible global function definition for ‘read.delim’
    snpDiff : calcInfoRange : <anonymous>: no visible global function
      definition for ‘fisher.test’
    snpDiff : calcInfoRange : <anonymous>: no visible global function
      definition for ‘dist’
    Undefined global functions or variables:
      dist fisher.test read.delim
    Consider adding
      importFrom("stats", "dist", "fisher.test")
      importFrom("utils", "read.delim")
    to your NAMESPACE file.
    ```

# sigminer

<details>

* Version: 2.0.2
* GitHub: https://github.com/ShixiangWang/sigminer
* Source code: https://github.com/cran/sigminer
* Date/Publication: 2021-06-17 11:10:02 UTC
* Number of recursive dependencies: 203

Run `revdep_details(, "sigminer")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'ggpubr', 'NMF'
    
    Packages suggested but not available for checking: 'circlize', 'ggplotify'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# sindyr

<details>

* Version: 0.2.3
* GitHub: NA
* Source code: https://github.com/cran/sindyr
* Date/Publication: 2020-06-09 05:20:03 UTC
* Number of recursive dependencies: 95

Run `revdep_details(, "sindyr")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'arrangements', 'crqa'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# Single.mTEC.Transcriptomes

<details>

* Version: 1.20.0
* GitHub: NA
* Source code: https://github.com/cran/Single.mTEC.Transcriptomes
* Date/Publication: 2021-05-20
* Number of recursive dependencies: 164

Run `revdep_details(, "Single.mTEC.Transcriptomes")` for more info

</details>

## In both

*   checking contents of ‘data’ directory ... WARNING
    ```
    Output for data("fantom", package = "Single.mTEC.Transcriptomes"):
      Warning: namespace ‘DESeq2’ is not available and has been replaced
      by .GlobalEnv when processing object ‘dxdFANTOM’
    ```

*   checking data for non-ASCII characters ... WARNING
    ```
      Error loading dataset 'dxd':
       Error in .requirePackage(package) : 
        unable to find required package 'DESeq2'
      
      Error loading dataset 'dxdATAC':
       Error in .requirePackage(package) : 
        unable to find required package 'DESeq2'
      
      Error loading dataset 'dxdFANTOM':
       Error in .requirePackage(package) : 
        unable to find required package 'DESeq2'
      
      The dataset(s) may use package(s) not declared in the DESCRIPTION file.
    ```

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      'DESeq2', 'GenomicFeatures', 'genefilter', 'ggbio', 'Gviz',
      'geneplotter'
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .git_fetch_output.txt
      .git_merge_output.txt
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 904.5Mb
      sub-directories of 1Mb or more:
        data  895.1Mb
        doc     9.3Mb
    ```

# singleCellTK

<details>

* Version: 2.2.0
* GitHub: https://github.com/compbiomed/singleCellTK
* Source code: https://github.com/cran/singleCellTK
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 371

Run `revdep_details(, "singleCellTK")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'batchelor', 'celldex', 'ComplexHeatmap', 'DESeq2', 'ExperimentHub',
      'ggplotify', 'ggtree', 'GSVA', 'GSVAdata', 'MAST', 'scater',
      'scMerge', 'scran', 'Seurat', 'SingleR', 'sva', 'circlize', 'celda',
      'DropletUtils', 'scds', 'GSEABase', 'zinbwave', 'scRNAseq',
      'TENxPBMCData', 'scDblFinder', 'metap'
    
    Packages suggested but not available for checking: 'lintr', 'org.Mm.eg.db'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# singscore

<details>

* Version: 1.12.0
* GitHub: https://github.com/DavisLaboratory/singscore
* Source code: https://github.com/cran/singscore
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 124

Run `revdep_details(, "singscore")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘GSEABase’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# SkeweDF

<details>

* Version: 0.1.0
* GitHub: NA
* Source code: https://github.com/cran/SkeweDF
* Date/Publication: 2021-01-16 09:40:05 UTC
* Number of recursive dependencies: 29

Run `revdep_details(, "SkeweDF")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘optimr’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# slingshot

<details>

* Version: 2.0.0
* GitHub: https://github.com/kstreet13/slingshot
* Source code: https://github.com/cran/slingshot
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 198

Run `revdep_details(, "slingshot")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘TrajectoryUtils’
    
    Package suggested but not available for checking: ‘clusterExperiment’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# sparseMatrixStats

<details>

* Version: 1.4.0
* GitHub: https://github.com/const-ae/sparseMatrixStats
* Source code: https://github.com/cran/sparseMatrixStats
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 53

Run `revdep_details(, "sparseMatrixStats")` for more info

</details>

## Newly broken

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 50 lines of output:
          rowSdDiffs, rowSds, rowSums2, rowTabulates, rowVarDiffs, rowVars,
          rowWeightedMads, rowWeightedMeans, rowWeightedMedians,
          rowWeightedSds, rowWeightedVars
      
      > 
      > test_check("sparseMatrixStats")
      ══ Skipped tests ═══════════════════════════════════════════════════════════════
    ...
      ── Failure (test-functions.R:285:5): colRanks works ────────────────────────────
      colRanks(sp_mat, ties.method = "min") not equal to matrixStats::colRanks(mat, ties.method = "min").
      Attributes: < Length mismatch: comparison on first 1 components >
      ── Failure (test-functions.R:286:5): colRanks works ────────────────────────────
      colRanks(sp_mat, rows = row_subset, cols = col_subset) not equal to matrixStats::colRanks(mat, rows = row_subset, cols = col_subset).
      Attributes: < Length mismatch: comparison on first 1 components >
      
      [ FAIL 12 | WARN 0 | SKIP 21 | PASS 1731 ]
      Error: Test failures
      Execution halted
    ```

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.4Mb
      sub-directories of 1Mb or more:
        libs   3.5Mb
    ```

# spathial

<details>

* Version: 0.1.2
* GitHub: NA
* Source code: https://github.com/cran/spathial
* Date/Publication: 2020-04-10 18:20:02 UTC
* Number of recursive dependencies: 29

Run `revdep_details(, "spathial")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘irlba’ ‘knitr’ ‘rmarkdown’
      All declared Imports should be used.
    ```

*   checking LazyData ... NOTE
    ```
      'LazyData' is specified without a 'data' directory
    ```

# splatter

<details>

* Version: 1.16.1
* GitHub: https://github.com/Oshlack/splatter
* Source code: https://github.com/cran/splatter
* Date/Publication: 2021-05-20
* Number of recursive dependencies: 247

Run `revdep_details(, "splatter")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘scater’
    
    Packages suggested but not available for checking:
      'scDD', 'scran', 'mfa', 'BASiCS', 'zinbwave', 'scuttle',
      'VariantAnnotation'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# spqn

<details>

* Version: 1.4.0
* GitHub: https://github.com/hansenlab/spqn
* Source code: https://github.com/cran/spqn
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 68

Run `revdep_details(, "spqn")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Package in Depends field not imported from: ‘BiocGenerics’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

# stapler

<details>

* Version: 0.7.1
* GitHub: https://github.com/muschellij2/stapler
* Source code: https://github.com/cran/stapler
* Date/Publication: 2020-01-09 18:50:06 UTC
* Number of recursive dependencies: 58

Run `revdep_details(, "stapler")` for more info

</details>

## In both

*   checking LazyData ... NOTE
    ```
      'LazyData' is specified without a 'data' directory
    ```

# staRdom

<details>

* Version: 1.1.21
* GitHub: https://github.com/MatthiasPucher/staRdom
* Source code: https://github.com/cran/staRdom
* Date/Publication: 2021-04-20 11:00:03 UTC
* Number of recursive dependencies: 147

Run `revdep_details(, "staRdom")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘multiway’
    
    Package suggested but not available for checking: ‘xlsx’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# stm

<details>

* Version: 1.3.6
* GitHub: https://github.com/bstewart/stm
* Source code: https://github.com/cran/stm
* Date/Publication: 2020-09-18 13:50:02 UTC
* Number of recursive dependencies: 87

Run `revdep_details(, "stm")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘quanteda’
    
    Packages suggested but not available for checking: 'geometry', 'tm'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# STROMA4

<details>

* Version: 1.16.0
* GitHub: NA
* Source code: https://github.com/cran/STROMA4
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 12

Run `revdep_details(, "STROMA4")` for more info

</details>

## In both

*   checking running R code from vignettes ...
    ```
      ‘STROMA4-vignette.Rnw’... failed
     ERROR
    Errors in running code in vignettes:
    when running code in ‘STROMA4-vignette.Rnw’
      ...
    
    > if (!requireNamespace("BiocManager", quietly = TRUE)) install.packages("BiocManager")
    Installing package into ‘/scratch/henrik/revdepcheck.extras/matrixStats/revdep/checks/STROMA4/new/STROMA4.Rcheck’
    (as ‘lib’ is unspecified)
    
      When sourcing ‘STROMA4-vignette.R’:
    Error: trying to use CRAN without setting a mirror
    Execution halted
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    'library' or 'require' call not declared from: ‘BiocManager’
    ```

# summarytools

<details>

* Version: 0.9.9
* GitHub: https://github.com/dcomtois/summarytools
* Source code: https://github.com/cran/summarytools
* Date/Publication: 2021-03-19 08:50:02 UTC
* Number of recursive dependencies: 71

Run `revdep_details(, "summarytools")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘rapportools’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# sva

<details>

* Version: 3.40.0
* GitHub: NA
* Source code: https://github.com/cran/sva
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 99

Run `revdep_details(, "sva")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘genefilter’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# target

<details>

* Version: 1.6.0
* GitHub: https://github.com/MahShaaban/target
* Source code: https://github.com/cran/target
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 88

Run `revdep_details(, "target")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘shinytest’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.6Mb
      sub-directories of 1Mb or more:
        data      3.4Mb
        extdata   1.6Mb
    ```

# TCA

<details>

* Version: 1.2.1
* GitHub: https://github.com/cozygene/TCA
* Source code: https://github.com/cran/TCA
* Date/Publication: 2021-02-14 21:50:06 UTC
* Number of recursive dependencies: 62

Run `revdep_details(, "TCA")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘Matrix’ ‘quadprog’
      All declared Imports should be used.
    ```

*   checking LazyData ... NOTE
    ```
      'LazyData' is specified without a 'data' directory
    ```

# tenXplore

<details>

* Version: 1.14.0
* GitHub: NA
* Source code: https://github.com/cran/tenXplore
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 127

Run `revdep_details(, "tenXplore")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'restfulSE', 'ontoProc', 'org.Mm.eg.db'
    
    Package suggested but not available for checking: ‘org.Hs.eg.db’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# TOAST

<details>

* Version: 1.6.0
* GitHub: https://github.com/ziyili20/TOAST
* Source code: https://github.com/cran/TOAST
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 54

Run `revdep_details(, "TOAST")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘EpiDISH’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# topGO

<details>

* Version: 2.44.0
* GitHub: NA
* Source code: https://github.com/cran/topGO
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 57

Run `revdep_details(, "topGO")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘GO.db’
    
    Packages suggested but not available for checking:
      'hgu95av2.db', 'hgu133a.db', 'genefilter', 'Rgraphviz', 'globaltest'
    
    Depends: includes the non-default packages:
      'BiocGenerics', 'graph', 'Biobase', 'GO.db', 'AnnotationDbi',
      'SparseM'
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# tradeSeq

<details>

* Version: 1.6.0
* GitHub: https://github.com/statOmics/tradeSeq
* Source code: https://github.com/cran/tradeSeq
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 195

Run `revdep_details(, "tradeSeq")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'slingshot', 'monocle'
    
    Package suggested but not available for checking: ‘clusterExperiment’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# TraRe

<details>

* Version: 1.0.0
* GitHub: https://github.com/ubioinformat/TraRe
* Source code: https://github.com/cran/TraRe
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 91

Run `revdep_details(, "TraRe")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.8Mb
      sub-directories of 1Mb or more:
        doc       2.0Mb
        extdata   4.4Mb
    ```

# ttScreening

<details>

* Version: 1.6
* GitHub: NA
* Source code: https://github.com/cran/ttScreening
* Date/Publication: 2018-10-11 22:00:02 UTC
* Number of recursive dependencies: 66

Run `revdep_details(, "ttScreening")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'sva', 'simsalapar'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# tvgarch

<details>

* Version: 2.0
* GitHub: NA
* Source code: https://github.com/cran/tvgarch
* Date/Publication: 2021-04-16 15:50:03 UTC
* Number of recursive dependencies: 15

Run `revdep_details(, "tvgarch")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘hier.part’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# tximport

<details>

* Version: 1.20.0
* GitHub: https://github.com/mikelove/tximport
* Source code: https://github.com/cran/tximport
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 147

Run `revdep_details(, "tximport")` for more info

</details>

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Complete output:
      > library(testthat)
      > library(tximport)
      > library(tximportData)
      > library(readr)
      > test_check("tximport")
      ══ Failed tests ════════════════════════════════════════════════════════════════
      ── Error (test_h5.R:10:3): kallisto HDF5 import works ──────────────────────────
    ...
      ── Error (test_kallisto.R:12:3): import kallisto works ─────────────────────────
      Error: reading kallisto results from hdf5 files requires Bioconductor package `rhdf5`
      Backtrace:
          █
       1. └─tximport::tximport(...) test_kallisto.R:12:2
       2.   └─tximport:::infRepImporter(dirname(files[1]))
      
      [ FAIL 2 | WARN 0 | SKIP 0 | PASS 28 ]
      Error: Test failures
      Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      'TxDb.Hsapiens.UCSC.hg19.knownGene', 'DESeq2', 'rhdf5'
    ```

# UPG

<details>

* Version: 0.2.2
* GitHub: NA
* Source code: https://github.com/cran/UPG
* Date/Publication: 2021-01-07 09:00:05 UTC
* Number of recursive dependencies: 50

Run `revdep_details(, "UPG")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  8.3Mb
      sub-directories of 1Mb or more:
        libs   7.3Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘RcppProgress’
      All declared Imports should be used.
    ```

# utilities

<details>

* Version: 0.3.0
* GitHub: NA
* Source code: https://github.com/cran/utilities
* Date/Publication: 2021-06-23 04:20:02 UTC
* Number of recursive dependencies: 60

Run `revdep_details(, "utilities")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘ggdag’
    ```

# VanillaICE

<details>

* Version: 1.54.0
* GitHub: NA
* Source code: https://github.com/cran/VanillaICE
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 75

Run `revdep_details(, "VanillaICE")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'oligoClasses', 'crlmm'
    
    Packages which this enhances but not available for checking:
      'doMC', 'doMPI', 'doSNOW', 'doParallel', 'doRedis'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# VaSP

<details>

* Version: 1.4.0
* GitHub: https://github.com/yuhuihui2011/VaSP
* Source code: https://github.com/cran/VaSP
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 110

Run `revdep_details(, "VaSP")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'ballgown', 'Sushi'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# visualFields

<details>

* Version: 0.6.2
* GitHub: NA
* Source code: https://github.com/cran/visualFields
* Date/Publication: 2021-06-17 15:50:02 UTC
* Number of recursive dependencies: 79

Run `revdep_details(, "visualFields")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘flip’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# wateRmelon

<details>

* Version: 1.36.0
* GitHub: NA
* Source code: https://github.com/cran/wateRmelon
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 160

Run `revdep_details(, "wateRmelon")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'methylumi', 'lumi', 'IlluminaHumanMethylation450kanno.ilmn12.hg19'
    
    Packages suggested but not available for checking:
      'IlluminaHumanMethylationEPICanno.ilm10b2.hg19',
      'IlluminaHumanMethylationEPICmanifest'
    
    Package which this enhances but not available for checking: ‘minfi’
    
    Depends: includes the non-default packages:
      'Biobase', 'limma', 'matrixStats', 'methylumi', 'lumi', 'ROC',
      'IlluminaHumanMethylation450kanno.ilmn12.hg19', 'illuminaio'
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# WGCNA

<details>

* Version: 1.70-3
* GitHub: NA
* Source code: https://github.com/cran/WGCNA
* Date/Publication: 2021-02-28 20:20:03 UTC
* Number of recursive dependencies: 104

Run `revdep_details(, "WGCNA")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘GO.db’
    
    Packages suggested but not available for checking:
      'org.Hs.eg.db', 'org.Mm.eg.db'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# Wrench

<details>

* Version: 1.10.0
* GitHub: https://github.com/HCBravoLab/Wrench
* Source code: https://github.com/cran/Wrench
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 107

Run `revdep_details(, "Wrench")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘Wrench-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: wrench
    > ### Title: Normalization for sparse, under-sampled count data.
    > ### Aliases: wrench
    > 
    > ### ** Examples
    > 
    > #Obtain counts matrix and some group information
    > require(metagenomeSeq)
    Loading required package: metagenomeSeq
    Warning in library(package, lib.loc = lib.loc, character.only = TRUE, logical.return = TRUE,  :
      there is no package called ‘metagenomeSeq’
    > data(mouseData)
    Warning in data(mouseData) : data set ‘mouseData’ not found
    > cntsMatrix <- MRcounts(mouseData)
    Error in MRcounts(mouseData) : could not find function "MRcounts"
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      'metagenomeSeq', 'DESeq2'
    ```

# yarn

<details>

* Version: 1.18.0
* GitHub: NA
* Source code: https://github.com/cran/yarn
* Date/Publication: 2021-05-19
* Number of recursive dependencies: 167

Run `revdep_details(, "yarn")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'biomaRt', 'quantro'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# zinbwave

<details>

* Version: 1.14.1
* GitHub: https://github.com/drisso/zinbwave
* Source code: https://github.com/cran/zinbwave
* Date/Publication: 2021-05-25
* Number of recursive dependencies: 160

Run `revdep_details(, "zinbwave")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘genefilter’
    
    Packages suggested but not available for checking:
      'scRNAseq', 'biomaRt', 'DESeq2'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

